<!--
  ~ The Exomiser - A tool to annotate and prioritize genomic variants
  ~
  ~ Copyright (c) 2016-2021 Queen Mary University of London.
  ~ Copyright (c) 2012-2016 Charité Universitätsmedizin Berlin and Genome Research Ltd.
  ~
  ~ This program is free software: you can redistribute it and/or modify
  ~ it under the terms of the GNU Affero General Public License as
  ~ published by the Free Software Foundation, either version 3 of the
  ~ License, or (at your option) any later version.
  ~
  ~ This program is distributed in the hope that it will be useful,
  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  ~ GNU Affero General Public License for more details.
  ~
  ~ You should have received a copy of the GNU Affero General Public License
  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <title>The Exomiser - A Tool to Annotate and Prioritise Whole-Exome Sequencing Data</title>
    <meta charset="UTF-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        /*!
 * Bootstrap v3.3.7 (http://getbootstrap.com)
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:after,:before{color:#000!important;text-shadow:none!important;background:0 0!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}blockquote,pre{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}img{max-width:100%!important}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff2) format('woff2'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\002a"}.glyphicon-plus:before{content:"\002b"}.glyphicon-eur:before,.glyphicon-euro:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}.glyphicon-cd:before{content:"\e201"}.glyphicon-save-file:before{content:"\e202"}.glyphicon-open-file:before{content:"\e203"}.glyphicon-level-up:before{content:"\e204"}.glyphicon-copy:before{content:"\e205"}.glyphicon-paste:before{content:"\e206"}.glyphicon-alert:before{content:"\e209"}.glyphicon-equalizer:before{content:"\e210"}.glyphicon-king:before{content:"\e211"}.glyphicon-queen:before{content:"\e212"}.glyphicon-pawn:before{content:"\e213"}.glyphicon-bishop:before{content:"\e214"}.glyphicon-knight:before{content:"\e215"}.glyphicon-baby-formula:before{content:"\e216"}.glyphicon-tent:before{content:"\26fa"}.glyphicon-blackboard:before{content:"\e218"}.glyphicon-bed:before{content:"\e219"}.glyphicon-apple:before{content:"\f8ff"}.glyphicon-erase:before{content:"\e221"}.glyphicon-hourglass:before{content:"\231b"}.glyphicon-lamp:before{content:"\e223"}.glyphicon-duplicate:before{content:"\e224"}.glyphicon-piggy-bank:before{content:"\e225"}.glyphicon-scissors:before{content:"\e226"}.glyphicon-bitcoin:before{content:"\e227"}.glyphicon-btc:before{content:"\e227"}.glyphicon-xbt:before{content:"\e227"}.glyphicon-yen:before{content:"\00a5"}.glyphicon-jpy:before{content:"\00a5"}.glyphicon-ruble:before{content:"\20bd"}.glyphicon-rub:before{content:"\20bd"}.glyphicon-scale:before{content:"\e230"}.glyphicon-ice-lolly:before{content:"\e231"}.glyphicon-ice-lolly-tasted:before{content:"\e232"}.glyphicon-education:before{content:"\e233"}.glyphicon-option-horizontal:before{content:"\e234"}.glyphicon-option-vertical:before{content:"\e235"}.glyphicon-menu-hamburger:before{content:"\e236"}.glyphicon-modal-window:before{content:"\e237"}.glyphicon-oil:before{content:"\e238"}.glyphicon-grain:before{content:"\e239"}.glyphicon-sunglasses:before{content:"\e240"}.glyphicon-text-size:before{content:"\e241"}.glyphicon-text-color:before{content:"\e242"}.glyphicon-text-background:before{content:"\e243"}.glyphicon-object-align-top:before{content:"\e244"}.glyphicon-object-align-bottom:before{content:"\e245"}.glyphicon-object-align-horizontal:before{content:"\e246"}.glyphicon-object-align-left:before{content:"\e247"}.glyphicon-object-align-vertical:before{content:"\e248"}.glyphicon-object-align-right:before{content:"\e249"}.glyphicon-triangle-right:before{content:"\e250"}.glyphicon-triangle-left:before{content:"\e251"}.glyphicon-triangle-bottom:before{content:"\e252"}.glyphicon-triangle-top:before{content:"\e253"}.glyphicon-console:before{content:"\e254"}.glyphicon-superscript:before{content:"\e255"}.glyphicon-subscript:before{content:"\e256"}.glyphicon-menu-left:before{content:"\e257"}.glyphicon-menu-right:before{content:"\e258"}.glyphicon-menu-down:before{content:"\e259"}.glyphicon-menu-up:before{content:"\e260"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}button,input,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:focus,a:hover{color:#23527c;text-decoration:underline}a:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.carousel-inner>.item>a>img,.carousel-inner>.item>img,.img-responsive,.thumbnail a>img,.thumbnail>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}[role=button]{cursor:pointer}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-weight:400;line-height:1;color:#777}.h1,.h2,.h3,h1,h2,h3{margin-top:20px;margin-bottom:10px}.h1 .small,.h1 small,.h2 .small,.h2 small,.h3 .small,.h3 small,h1 .small,h1 small,h2 .small,h2 small,h3 .small,h3 small{font-size:65%}.h4,.h5,.h6,h4,h5,h6{margin-top:10px;margin-bottom:10px}.h4 .small,.h4 small,.h5 .small,.h5 small,.h6 .small,.h6 small,h4 .small,h4 small,h5 .small,h5 small,h6 .small,h6 small{font-size:75%}.h1,h1{font-size:36px}.h2,h2{font-size:30px}.h3,h3{font-size:24px}.h4,h4{font-size:18px}.h5,h5{font-size:14px}.h6,h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}.small,small{font-size:85%}.mark,mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:focus,a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:focus,a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:focus,a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:focus,a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:focus,a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:focus,a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:focus,a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:focus,a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:focus,a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:focus,a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ol,ul{margin-top:0;margin-bottom:10px}ol ol,ol ul,ul ol,ul ul{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dd,dt{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[data-original-title],abbr[title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote ol:last-child,blockquote p:last-child,blockquote ul:last-child{margin-bottom:0}blockquote .small,blockquote footer,blockquote small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote .small:before,blockquote footer:before,blockquote small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse .small:before,.blockquote-reverse footer:before,.blockquote-reverse small:before,blockquote.pull-right .small:before,blockquote.pull-right footer:before,blockquote.pull-right small:before{content:''}.blockquote-reverse .small:after,.blockquote-reverse footer:after,.blockquote-reverse small:after,blockquote.pull-right .small:after,blockquote.pull-right footer:after,blockquote.pull-right small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-10,.col-xs-11,.col-xs-12,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-10,.col-sm-11,.col-sm-12,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-10,.col-md-11,.col-md-12,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-10,.col-lg-11,.col-lg-12,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>tbody>tr>td,.table>tbody>tr>th,.table>tfoot>tr>td,.table>tfoot>tr>th,.table>thead>tr>td,.table>thead>tr>th{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>td,.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>td,.table>thead:first-child>tr:first-child>th{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>tbody>tr>td,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>td,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>thead>tr>th{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>tbody>tr>td,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>td,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border:1px solid #ddd}.table-bordered>thead>tr>td,.table-bordered>thead>tr>th{border-bottom-width:2px}.table-striped>tbody>tr:nth-of-type(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>tbody>tr.active>td,.table>tbody>tr.active>th,.table>tbody>tr>td.active,.table>tbody>tr>th.active,.table>tfoot>tr.active>td,.table>tfoot>tr.active>th,.table>tfoot>tr>td.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>thead>tr.active>th,.table>thead>tr>td.active,.table>thead>tr>th.active{background-color:#f5f5f5}.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr.active:hover>th,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover{background-color:#e8e8e8}.table>tbody>tr.success>td,.table>tbody>tr.success>th,.table>tbody>tr>td.success,.table>tbody>tr>th.success,.table>tfoot>tr.success>td,.table>tfoot>tr.success>th,.table>tfoot>tr>td.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>thead>tr.success>th,.table>thead>tr>td.success,.table>thead>tr>th.success{background-color:#dff0d8}.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr.success:hover>th,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover{background-color:#d0e9c6}.table>tbody>tr.info>td,.table>tbody>tr.info>th,.table>tbody>tr>td.info,.table>tbody>tr>th.info,.table>tfoot>tr.info>td,.table>tfoot>tr.info>th,.table>tfoot>tr>td.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>thead>tr.info>th,.table>thead>tr>td.info,.table>thead>tr>th.info{background-color:#d9edf7}.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr.info:hover>th,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover{background-color:#c4e3f3}.table>tbody>tr.warning>td,.table>tbody>tr.warning>th,.table>tbody>tr>td.warning,.table>tbody>tr>th.warning,.table>tfoot>tr.warning>td,.table>tfoot>tr.warning>th,.table>tfoot>tr>td.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>thead>tr.warning>th,.table>thead>tr>td.warning,.table>thead>tr>th.warning{background-color:#fcf8e3}.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr.warning:hover>th,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover{background-color:#faf2cc}.table>tbody>tr.danger>td,.table>tbody>tr.danger>th,.table>tbody>tr>td.danger,.table>tbody>tr>th.danger,.table>tfoot>tr.danger>td,.table>tfoot>tr.danger>th,.table>tfoot>tr>td.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>thead>tr.danger>th,.table>thead>tr>td.danger,.table>thead>tr>th.danger{background-color:#f2dede}.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr.danger:hover>th,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>td,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>thead>tr>th{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=checkbox],input[type=radio]{margin:4px 0 0;margin-top:1px\9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=checkbox]:focus,input[type=radio]:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control::-ms-expand{background-color:transparent;border:0}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{background-color:#eee;opacity:1}.form-control[disabled],fieldset[disabled] .form-control{cursor:not-allowed}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date].form-control,input[type=time].form-control,input[type=datetime-local].form-control,input[type=month].form-control{line-height:34px}.input-group-sm input[type=date],.input-group-sm input[type=time],.input-group-sm input[type=datetime-local],.input-group-sm input[type=month],input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}.input-group-lg input[type=date],.input-group-lg input[type=time],.input-group-lg input[type=datetime-local],.input-group-lg input[type=month],input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.checkbox,.radio{position:relative;display:block;margin-top:10px;margin-bottom:10px}.checkbox label,.radio label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{position:absolute;margin-top:4px\9;margin-left:-20px}.checkbox+.checkbox,.radio+.radio{margin-top:-5px}.checkbox-inline,.radio-inline{position:relative;display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.checkbox-inline+.checkbox-inline,.radio-inline+.radio-inline{margin-top:0;margin-left:10px}fieldset[disabled] input[type=checkbox],fieldset[disabled] input[type=radio],input[type=checkbox].disabled,input[type=checkbox][disabled],input[type=radio].disabled,input[type=radio][disabled]{cursor:not-allowed}.checkbox-inline.disabled,.radio-inline.disabled,fieldset[disabled] .checkbox-inline,fieldset[disabled] .radio-inline{cursor:not-allowed}.checkbox.disabled label,.radio.disabled label,fieldset[disabled] .checkbox label,fieldset[disabled] .radio label{cursor:not-allowed}.form-control-static{min-height:34px;padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm{height:30px;line-height:30px}select[multiple].input-sm,textarea.input-sm{height:auto}.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.form-group-sm select.form-control{height:30px;line-height:30px}.form-group-sm select[multiple].form-control,.form-group-sm textarea.form-control{height:auto}.form-group-sm .form-control-static{height:30px;min-height:32px;padding:6px 10px;font-size:12px;line-height:1.5}.input-lg{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-lg{height:46px;line-height:46px}select[multiple].input-lg,textarea.input-lg{height:auto}.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.form-group-lg select.form-control{height:46px;line-height:46px}.form-group-lg select[multiple].form-control,.form-group-lg textarea.form-control{height:auto}.form-group-lg .form-control-static{height:46px;min-height:38px;padding:11px 16px;font-size:18px;line-height:1.3333333}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.form-group-lg .form-control+.form-control-feedback,.input-group-lg+.form-control-feedback,.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.form-group-sm .form-control+.form-control-feedback,.input-group-sm+.form-control-feedback,.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .checkbox,.has-success .checkbox-inline,.has-success .control-label,.has-success .help-block,.has-success .radio,.has-success .radio-inline,.has-success.checkbox label,.has-success.checkbox-inline label,.has-success.radio label,.has-success.radio-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .checkbox,.has-warning .checkbox-inline,.has-warning .control-label,.has-warning .help-block,.has-warning .radio,.has-warning .radio-inline,.has-warning.checkbox label,.has-warning.checkbox-inline label,.has-warning.radio label,.has-warning.radio-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .checkbox,.has-error .checkbox-inline,.has-error .control-label,.has-error .help-block,.has-error .radio,.has-error .radio-inline,.has-error.checkbox label,.has-error.checkbox-inline label,.has-error.radio label,.has-error.radio-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .form-control,.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .checkbox,.form-inline .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .checkbox label,.form-inline .radio label{padding-left:0}.form-inline .checkbox input[type=checkbox],.form-inline .radio input[type=radio]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .checkbox,.form-horizontal .checkbox-inline,.form-horizontal .radio,.form-horizontal .radio-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .checkbox,.form-horizontal .radio{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:11px;font-size:18px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px;font-size:12px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn.active.focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn:active:focus,.btn:focus{outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn.focus,.btn:focus,.btn:hover{color:#333;text-decoration:none}.btn.active,.btn:active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}a.btn.disabled,fieldset[disabled] a.btn{pointer-events:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default.focus,.btn-default:focus{color:#333;background-color:#e6e6e6;border-color:#8c8c8c}.btn-default:hover{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default.active.focus,.btn-default.active:focus,.btn-default.active:hover,.btn-default:active.focus,.btn-default:active:focus,.btn-default:active:hover,.open>.dropdown-toggle.btn-default.focus,.open>.dropdown-toggle.btn-default:focus,.open>.dropdown-toggle.btn-default:hover{color:#333;background-color:#d4d4d4;border-color:#8c8c8c}.btn-default.active,.btn-default:active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled.focus,.btn-default.disabled:focus,.btn-default.disabled:hover,.btn-default[disabled].focus,.btn-default[disabled]:focus,.btn-default[disabled]:hover,fieldset[disabled] .btn-default.focus,fieldset[disabled] .btn-default:focus,fieldset[disabled] .btn-default:hover{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary.focus,.btn-primary:focus{color:#fff;background-color:#286090;border-color:#122b40}.btn-primary:hover{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary.active.focus,.btn-primary.active:focus,.btn-primary.active:hover,.btn-primary:active.focus,.btn-primary:active:focus,.btn-primary:active:hover,.open>.dropdown-toggle.btn-primary.focus,.open>.dropdown-toggle.btn-primary:focus,.open>.dropdown-toggle.btn-primary:hover{color:#fff;background-color:#204d74;border-color:#122b40}.btn-primary.active,.btn-primary:active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled.focus,.btn-primary.disabled:focus,.btn-primary.disabled:hover,.btn-primary[disabled].focus,.btn-primary[disabled]:focus,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary.focus,fieldset[disabled] .btn-primary:focus,fieldset[disabled] .btn-primary:hover{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success.focus,.btn-success:focus{color:#fff;background-color:#449d44;border-color:#255625}.btn-success:hover{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success.active.focus,.btn-success.active:focus,.btn-success.active:hover,.btn-success:active.focus,.btn-success:active:focus,.btn-success:active:hover,.open>.dropdown-toggle.btn-success.focus,.open>.dropdown-toggle.btn-success:focus,.open>.dropdown-toggle.btn-success:hover{color:#fff;background-color:#398439;border-color:#255625}.btn-success.active,.btn-success:active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled.focus,.btn-success.disabled:focus,.btn-success.disabled:hover,.btn-success[disabled].focus,.btn-success[disabled]:focus,.btn-success[disabled]:hover,fieldset[disabled] .btn-success.focus,fieldset[disabled] .btn-success:focus,fieldset[disabled] .btn-success:hover{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info.focus,.btn-info:focus{color:#fff;background-color:#31b0d5;border-color:#1b6d85}.btn-info:hover{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info.active.focus,.btn-info.active:focus,.btn-info.active:hover,.btn-info:active.focus,.btn-info:active:focus,.btn-info:active:hover,.open>.dropdown-toggle.btn-info.focus,.open>.dropdown-toggle.btn-info:focus,.open>.dropdown-toggle.btn-info:hover{color:#fff;background-color:#269abc;border-color:#1b6d85}.btn-info.active,.btn-info:active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled.focus,.btn-info.disabled:focus,.btn-info.disabled:hover,.btn-info[disabled].focus,.btn-info[disabled]:focus,.btn-info[disabled]:hover,fieldset[disabled] .btn-info.focus,fieldset[disabled] .btn-info:focus,fieldset[disabled] .btn-info:hover{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning.focus,.btn-warning:focus{color:#fff;background-color:#ec971f;border-color:#985f0d}.btn-warning:hover{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning.active.focus,.btn-warning.active:focus,.btn-warning.active:hover,.btn-warning:active.focus,.btn-warning:active:focus,.btn-warning:active:hover,.open>.dropdown-toggle.btn-warning.focus,.open>.dropdown-toggle.btn-warning:focus,.open>.dropdown-toggle.btn-warning:hover{color:#fff;background-color:#d58512;border-color:#985f0d}.btn-warning.active,.btn-warning:active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled.focus,.btn-warning.disabled:focus,.btn-warning.disabled:hover,.btn-warning[disabled].focus,.btn-warning[disabled]:focus,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning.focus,fieldset[disabled] .btn-warning:focus,fieldset[disabled] .btn-warning:hover{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger.focus,.btn-danger:focus{color:#fff;background-color:#c9302c;border-color:#761c19}.btn-danger:hover{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger.active.focus,.btn-danger.active:focus,.btn-danger.active:hover,.btn-danger:active.focus,.btn-danger:active:focus,.btn-danger:active:hover,.open>.dropdown-toggle.btn-danger.focus,.open>.dropdown-toggle.btn-danger:focus,.open>.dropdown-toggle.btn-danger:hover{color:#fff;background-color:#ac2925;border-color:#761c19}.btn-danger.active,.btn-danger:active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled.focus,.btn-danger.disabled:focus,.btn-danger.disabled:hover,.btn-danger[disabled].focus,.btn-danger[disabled]:focus,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger.focus,fieldset[disabled] .btn-danger:focus,fieldset[disabled] .btn-danger:hover{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link.active,.btn-link:active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:active,.btn-link:focus,.btn-link:hover{border-color:transparent}.btn-link:focus,.btn-link:hover{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:focus,.btn-link[disabled]:hover,fieldset[disabled] .btn-link:focus,fieldset[disabled] .btn-link:hover{color:#777;text-decoration:none}.btn-group-lg>.btn,.btn-lg{padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}.btn-group-sm>.btn,.btn-sm{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-group-xs>.btn,.btn-xs{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=button].btn-block,input[type=reset].btn-block,input[type=submit].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none}.collapse.in{display:block}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown,.dropup{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:focus,.dropdown-menu>li>a:hover{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:focus,.dropdown-menu>.active>a:hover{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{color:#777}.dropdown-menu>.disabled>a:focus,.dropdown-menu>.disabled>a:hover{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px dashed;border-bottom:4px solid\9}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:2px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group-vertical>.btn,.btn-group>.btn{position:relative;float:left}.btn-group-vertical>.btn.active,.btn-group-vertical>.btn:active,.btn-group-vertical>.btn:focus,.btn-group-vertical>.btn:hover,.btn-group>.btn.active,.btn-group>.btn:active,.btn-group>.btn:focus,.btn-group>.btn:hover{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn,.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=radio]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group .form-control:focus{z-index:3}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.3333333;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn,textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn,textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn{height:auto}.input-group .form-control,.input-group-addon,.input-group-btn{display:table-cell}.input-group .form-control:not(:first-child):not(:last-child),.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=checkbox],.input-group-addon input[type=radio]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn-group:not(:last-child)>.btn,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:first-child>.btn-group:not(:first-child)>.btn,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:active,.input-group-btn>.btn:focus,.input-group-btn>.btn:hover{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:focus,.nav-tabs.nav-justified>.active>a:hover{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:focus,.nav-pills>li.active>a:hover{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:focus,.nav-tabs-justified>.active>a:hover{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none}.tab-content>.active{display:block}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-bottom .navbar-collapse,.navbar-fixed-top .navbar-collapse{max-height:200px}}.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container-fluid>.navbar-collapse,.container-fluid>.navbar-header,.container>.navbar-collapse,.container>.navbar-header{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-bottom,.navbar-fixed-top{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-bottom,.navbar-fixed-top{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:focus,.navbar-brand:hover{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu .dropdown-header,.navbar-nav .open .dropdown-menu>li>a{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:focus,.navbar-nav .open .dropdown-menu>li>a:hover{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .form-control,.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .checkbox,.navbar-form .radio{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .checkbox label,.navbar-form .radio label{padding-left:0}.navbar-form .checkbox input[type=checkbox],.navbar-form .radio input[type=radio]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{margin-bottom:0;border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:focus,.navbar-default .navbar-brand:hover{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:focus,.navbar-default .navbar-nav>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:focus,.navbar-default .navbar-nav>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:focus,.navbar-default .navbar-nav>.disabled>a:hover{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:focus,.navbar-default .navbar-toggle:hover{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:focus,.navbar-default .navbar-nav>.open>a:hover{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:focus,.navbar-default .btn-link:hover{color:#333}.navbar-default .btn-link[disabled]:focus,.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:focus,fieldset[disabled] .navbar-default .btn-link:hover{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:focus,.navbar-inverse .navbar-brand:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:focus,.navbar-inverse .navbar-nav>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:focus,.navbar-inverse .navbar-nav>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:focus,.navbar-inverse .navbar-nav>.disabled>a:hover{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:focus,.navbar-inverse .navbar-toggle:hover{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:focus,.navbar-inverse .navbar-nav>.open>a:hover{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:focus,.navbar-inverse .btn-link:hover{color:#fff}.navbar-inverse .btn-link[disabled]:focus,.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:focus,fieldset[disabled] .navbar-inverse .btn-link:hover{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:focus,.pagination>li>a:hover,.pagination>li>span:focus,.pagination>li>span:hover{z-index:2;color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>a:focus,.pagination>.active>a:hover,.pagination>.active>span,.pagination>.active>span:focus,.pagination>.active>span:hover{z-index:3;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>a,.pagination>.disabled>a:focus,.pagination>.disabled>a:hover,.pagination>.disabled>span,.pagination>.disabled>span:focus,.pagination>.disabled>span:hover{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px;line-height:1.3333333}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px;line-height:1.5}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:focus,.pager li>a:hover{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:focus,.pager .disabled>a:hover,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:focus,a.label:hover{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:focus,.label-default[href]:hover{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:focus,.label-primary[href]:hover{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:focus,.label-success[href]:hover{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:focus,.label-info[href]:hover{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:focus,.label-warning[href]:hover{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:focus,.label-danger[href]:hover{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:middle;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-group-xs>.btn .badge,.btn-xs .badge{top:0;padding:1px 5px}a.badge:focus,a.badge:hover{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding-top:30px;padding-bottom:30px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron .h1,.jumbotron h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{padding-right:15px;padding-left:15px;border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding-top:48px;padding-bottom:48px}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron .h1,.jumbotron h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail a>img,.thumbnail>img{margin-right:auto;margin-left:auto}a.thumbnail.active,a.thumbnail:focus,a.thumbnail:hover{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-bar-striped,.progress-striped .progress-bar{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress-bar.active,.progress.active .progress-bar{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media,.media-body{overflow:hidden;zoom:1}.media-body{width:10000px}.media-object{display:block}.media-object.img-thumbnail{max-width:none}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-body,.media-left,.media-right{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item,button.list-group-item{color:#555}a.list-group-item .list-group-item-heading,button.list-group-item .list-group-item-heading{color:#333}a.list-group-item:focus,a.list-group-item:hover,button.list-group-item:focus,button.list-group-item:hover{color:#555;text-decoration:none;background-color:#f5f5f5}button.list-group-item{width:100%;text-align:left}.list-group-item.disabled,.list-group-item.disabled:focus,.list-group-item.disabled:hover{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:focus,.list-group-item.active:hover{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:focus .list-group-item-text,.list-group-item.active:hover .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success,button.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading,button.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:focus,a.list-group-item-success:hover,button.list-group-item-success:focus,button.list-group-item-success:hover{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:focus,a.list-group-item-success.active:hover,button.list-group-item-success.active,button.list-group-item-success.active:focus,button.list-group-item-success.active:hover{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info,button.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading,button.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:focus,a.list-group-item-info:hover,button.list-group-item-info:focus,button.list-group-item-info:hover{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:focus,a.list-group-item-info.active:hover,button.list-group-item-info.active,button.list-group-item-info.active:focus,button.list-group-item-info.active:hover{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning,button.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading,button.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:focus,a.list-group-item-warning:hover,button.list-group-item-warning:focus,button.list-group-item-warning:hover{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:focus,a.list-group-item-warning.active:hover,button.list-group-item-warning.active,button.list-group-item-warning.active:focus,button.list-group-item-warning.active:hover{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger,button.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading,button.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:focus,a.list-group-item-danger:hover,button.list-group-item-danger:focus,button.list-group-item-danger:hover{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:focus,a.list-group-item-danger.active:hover,button.list-group-item-danger.active,button.list-group-item-danger.active:focus,button.list-group-item-danger.active:hover{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>.small,.panel-title>.small>a,.panel-title>a,.panel-title>small,.panel-title>small>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.panel-heading+.panel-collapse>.list-group .list-group-item:first-child{border-top-left-radius:0;border-top-right-radius:0}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.panel-collapse>.table,.panel>.table,.panel>.table-responsive>.table{margin-bottom:0}.panel>.panel-collapse>.table caption,.panel>.table caption,.panel>.table-responsive>.table caption{padding-right:15px;padding-left:15px}.panel>.table-responsive:first-child>.table:first-child,.panel>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table:first-child>thead:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table-responsive:last-child>.table:last-child,.panel>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child td,.panel>.table>tbody:first-child>tr:first-child th{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child{border-left:0}.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child{border-right:0}.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.list-group,.panel-group .panel-heading+.panel-collapse>.panel-body{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive embed,.embed-responsive iframe,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:focus,.close:hover{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1050;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;filter:alpha(opacity=0);opacity:0;line-break:auto}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-style:normal;font-weight:400;line-height:1.42857143;text-align:left;text-align:start;text-decoration:none;text-shadow:none;text-transform:none;letter-spacing:normal;word-break:normal;word-spacing:normal;word-wrap:normal;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2);line-break:auto}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>a>img,.carousel-inner>.item>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000px;perspective:1000px}.carousel-inner>.item.active.right,.carousel-inner>.item.next{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.active.left,.carousel-inner>.item.prev{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.active,.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);background-color:rgba(0,0,0,0);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:focus,.carousel-control:hover{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{position:absolute;top:50%;z-index:5;display:inline-block;margin-top:-10px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{left:50%;margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{right:50%;margin-right:-10px}.carousel-control .icon-next,.carousel-control .icon-prev{width:20px;height:20px;font-family:serif;line-height:1}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000\9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next,.carousel-control .icon-prev{width:30px;height:30px;margin-top:-10px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-10px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-10px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.btn-group-vertical>.btn-group:after,.btn-group-vertical>.btn-group:before,.btn-toolbar:after,.btn-toolbar:before,.clearfix:after,.clearfix:before,.container-fluid:after,.container-fluid:before,.container:after,.container:before,.dl-horizontal dd:after,.dl-horizontal dd:before,.form-horizontal .form-group:after,.form-horizontal .form-group:before,.modal-footer:after,.modal-footer:before,.modal-header:after,.modal-header:before,.nav:after,.nav:before,.navbar-collapse:after,.navbar-collapse:before,.navbar-header:after,.navbar-header:before,.navbar:after,.navbar:before,.pager:after,.pager:before,.panel-body:after,.panel-body:before,.row:after,.row:before{display:table;content:" "}.btn-group-vertical>.btn-group:after,.btn-toolbar:after,.clearfix:after,.container-fluid:after,.container:after,.dl-horizontal dd:after,.form-horizontal .form-group:after,.modal-footer:after,.modal-header:after,.nav:after,.navbar-collapse:after,.navbar-header:after,.navbar:after,.pager:after,.panel-body:after,.row:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-lg,.visible-md,.visible-sm,.visible-xs{display:none!important}.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table!important}tr.visible-xs{display:table-row!important}td.visible-xs,th.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table!important}tr.visible-sm{display:table-row!important}td.visible-sm,th.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table!important}tr.visible-md{display:table-row!important}td.visible-md,th.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table!important}tr.visible-lg{display:table-row!important}td.visible-lg,th.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table!important}tr.visible-print{display:table-row!important}td.visible-print,th.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}
        /*# sourceMappingURL=bootstrap.min.css.map */
    </style>
    <!--<link rel="stylesheet" type="text/css" media="all" href="../../resources/bootstrap-3.2.0/css/bootstrap.min.css" th:href="@{resources/bootstrap-3.2.0/css/bootstrap.min.css}"/>-->
</head>
<body>
<div id="content" class="container">
    <h2>
        Exomiser Analysis Results for
        manuel
    </h2>
    <ul class="nav nav-pills" id="tabs">
        <li><a href="#settings" role="tab" data-toggle="tab"><b>Analysis Settings</b></a></li>
        <li><a href="#filtering" role="tab" data-toggle="tab"><b>Filtering Summary</b></a></li>
        <li><a href="#distribution" role="tab" data-toggle="tab"><b>Variant Type Distribution</b></a></li>
        <li><a href="#prioritisation" role="tab" data-toggle="tab"><b>Prioritised Genes</b></a></li>
        
        <li><a href="#about" role="tab" data-toggle="tab"><b>About</b></a></li>
    </ul>
    <div class="panel panel-default" id="settings">
        <div class="panel-heading">
            <h3>Analysis Settings</h3>
        </div>
        <div class="panel-body"><p>Settings used for this analysis:</p>
            <pre>---
sample:
  genomeAssembly: &quot;hg19&quot;
  vcf: &quot;exomiser/Pfeiffer.vcf.gz&quot;
  proband: &quot;manuel&quot;
  hpoIds:
  - &quot;HP:0001159&quot;
  - &quot;HP:0000486&quot;
  - &quot;HP:0000327&quot;
  - &quot;HP:0000520&quot;
  - &quot;HP:0000601&quot;
  - &quot;HP:0000244&quot;
  pedigree:
    persons:
    - individualId: &quot;manuel&quot;
      sex: &quot;MALE&quot;
      affectedStatus: &quot;AFFECTED&quot;
  sex: &quot;MALE&quot;
  age: {}
analysis:
  inheritanceModes:
    AUTOSOMAL_RECESSIVE_COMP_HET: 2.0
    MITOCHONDRIAL: 0.2
    AUTOSOMAL_RECESSIVE_HOM_ALT: 0.1
    X_DOMINANT: 0.1
    AUTOSOMAL_DOMINANT: 0.1
    X_RECESSIVE_COMP_HET: 2.0
    X_RECESSIVE_HOM_ALT: 0.1
  frequencySources:
  - &quot;THOUSAND_GENOMES&quot;
  - &quot;TOPMED&quot;
  - &quot;UK10K&quot;
  - &quot;ESP_AFRICAN_AMERICAN&quot;
  - &quot;ESP_EUROPEAN_AMERICAN&quot;
  - &quot;ESP_ALL&quot;
  - &quot;EXAC_AFRICAN_INC_AFRICAN_AMERICAN&quot;
  - &quot;EXAC_AMERICAN&quot;
  - &quot;EXAC_EAST_ASIAN&quot;
  - &quot;EXAC_FINNISH&quot;
  - &quot;EXAC_NON_FINNISH_EUROPEAN&quot;
  - &quot;EXAC_OTHER&quot;
  - &quot;EXAC_SOUTH_ASIAN&quot;
  - &quot;GNOMAD_E_AFR&quot;
  - &quot;GNOMAD_E_AMR&quot;
  - &quot;GNOMAD_E_EAS&quot;
  - &quot;GNOMAD_E_FIN&quot;
  - &quot;GNOMAD_E_NFE&quot;
  - &quot;GNOMAD_E_OTH&quot;
  - &quot;GNOMAD_E_SAS&quot;
  - &quot;GNOMAD_G_AFR&quot;
  - &quot;GNOMAD_G_AMR&quot;
  - &quot;GNOMAD_G_EAS&quot;
  - &quot;GNOMAD_G_FIN&quot;
  - &quot;GNOMAD_G_NFE&quot;
  - &quot;GNOMAD_G_OTH&quot;
  - &quot;GNOMAD_G_SAS&quot;
  pathogenicitySources:
  - &quot;REVEL&quot;
  - &quot;MVP&quot;
  steps:
  - failedVariantFilter: {}
  - variantEffectFilter:
      remove:
      - &quot;CODING_TRANSCRIPT_INTRON_VARIANT&quot;
      - &quot;FIVE_PRIME_UTR_EXON_VARIANT&quot;
      - &quot;THREE_PRIME_UTR_EXON_VARIANT&quot;
      - &quot;FIVE_PRIME_UTR_INTRON_VARIANT&quot;
      - &quot;THREE_PRIME_UTR_INTRON_VARIANT&quot;
      - &quot;NON_CODING_TRANSCRIPT_EXON_VARIANT&quot;
      - &quot;NON_CODING_TRANSCRIPT_INTRON_VARIANT&quot;
      - &quot;UPSTREAM_GENE_VARIANT&quot;
      - &quot;DOWNSTREAM_GENE_VARIANT&quot;
      - &quot;INTERGENIC_VARIANT&quot;
      - &quot;REGULATORY_REGION_VARIANT&quot;
  - frequencyFilter:
      maxFrequency: 2.0
  - pathogenicityFilter:
      keepNonPathogenic: true
  - inheritanceFilter: {}
  - omimPrioritiser: {}
  - hiPhivePrioritiser:
      runParams: &quot;human, mouse, fish, ppi&quot;
outputOptions:
  outputFormats:
  - &quot;HTML&quot;
  - &quot;TSV_GENE&quot;
  - &quot;TSV_VARIANT&quot;
  - &quot;JSON&quot;
  outputFileName: &quot;Pfeiffer-HIPHIVE-exome&quot;
  outputDirectory: &quot;results&quot;
</pre>
        </div>
    </div>

    <div class="panel panel-default" id="filtering">
        <div class="panel-heading">
            <h3>Filtering Summary</h3>
        </div>
        <div class="panel-body">
            <table class="table">
                <thead>
                <tr>
                    <th>Filter</th>
                    <th>Report</th>
                    <th>Passed filter</th>
                    <th>Failed filter</th>
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td><b>Failed previous VCF filters</b></td>
                    <td>
                        
                    </td>
                    <td>265</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Variant effect</b></td>
                    <td>
                        <ul>Removed variants with effects of type: [CODING_TRANSCRIPT_INTRON_VARIANT, FIVE_PRIME_UTR_EXON_VARIANT, THREE_PRIME_UTR_EXON_VARIANT, FIVE_PRIME_UTR_INTRON_VARIANT, THREE_PRIME_UTR_INTRON_VARIANT, NON_CODING_TRANSCRIPT_EXON_VARIANT, NON_CODING_TRANSCRIPT_INTRON_VARIANT, UPSTREAM_GENE_VARIANT, DOWNSTREAM_GENE_VARIANT, INTERGENIC_VARIANT, REGULATORY_REGION_VARIANT]</ul>
                    </td>
                    <td>265</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Frequency</b></td>
                    <td>
                        <ul>Variants filtered for maximum allele frequency of 2.00%</ul>
                    </td>
                    <td>265</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Pathogenicity</b></td>
                    <td>
                        <ul>Retained all non-pathogenic variants of all types. Scoring was applied, but the filter passed all variants.</ul>
                    </td>
                    <td>265</td>
                    <td>0</td>
                </tr>
                <tr>
                    <td><b>Inheritance</b></td>
                    <td>
                        <ul>Genes filtered for compatibility with AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL inheritance.</ul>
                    </td>
                    <td>228</td>
                    <td>0</td>
                </tr>
                </tbody>
            </table>
        </div>
    </div>
    <div class="panel panel-default" id="distribution">
        <div class="panel-heading">
            <h3>Variant Type Distribution</h3>
        </div>
        <div class="panel-body">
            <table class="table-striped table-condensed">
                <thead>
                <tr>
                    <th>Variant Type</th>
                    
                        <th>manuel</th>
                    
                </tr>
                </thead>
                <tbody>
                <tr>
                    <td>FRAMESHIFT_ELONGATION</td>
                    
                        <td>4</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_TRUNCATION</td>
                    
                        <td>4</td>
                    
                </tr>
                <tr>
                    <td>FRAMESHIFT_VARIANT</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>INTERNAL_FEATURE_ELONGATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FEATURE_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>MNV</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>STOP_GAINED</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>STOP_LOST</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>START_LOST</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_ACCEPTOR_VARIANT</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_DONOR_VARIANT</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>MISSENSE_VARIANT</td>
                    
                        <td>148</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_INSERTION</td>
                    
                        <td>2</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_INSERTION</td>
                    
                        <td>3</td>
                    
                </tr>
                <tr>
                    <td>INFRAME_DELETION</td>
                    
                        <td>1</td>
                    
                </tr>
                <tr>
                    <td>DISRUPTIVE_INFRAME_DELETION</td>
                    
                        <td>2</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_TRUNCATION</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SPLICE_REGION_VARIANT</td>
                    
                        <td>26</td>
                    
                </tr>
                <tr>
                    <td>STOP_RETAINED_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INITIATOR_CODON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>SYNONYMOUS_VARIANT</td>
                    
                        <td>65</td>
                    
                </tr>
                <tr>
                    <td>CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>FIVE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>THREE_PRIME_UTR_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_EXON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>NON_CODING_TRANSCRIPT_INTRON_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>UPSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>DOWNSTREAM_GENE_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>INTERGENIC_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                <tr>
                    <td>REGULATORY_REGION_VARIANT</td>
                    
                        <td>0</td>
                    
                </tr>
                </tbody>
            </table>
        </div>
    </div>

    

    <div class="panel panel-default" id="prioritisation">
        <div class="panel-heading">
            <h3>Prioritised Genes</h3>
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FGFR2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000066468">FGFR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.995</b>
                    <data>(p=1.6E-5)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.896</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.848 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=87">Apert syndrome</a> associated with FGFR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000244, Brachyturricephaly</dd></dl><dl><dt>Phenotypic similarity 0.896 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FGFR2">FGFR2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0004322, abnormal sternebra morphology</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0030350, premature coronal suture closure</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FGFR2%0DFGF18&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FGF18</a> and phenotypic similarity 0.841 to mouse mutant of FGF18.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003072, abnormal metatarsal bone morphology</dd><dd>HP:0000486, Strabismus - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000457, maxilla hypoplasia</dd><dd>HP:0000520, Proptosis - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000074, abnormal neurocranium morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/101200">OMIM:101200</a> Apert syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/101400">OMIM:101400</a> Saethre-Chotzen syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/101600">OMIM:101600</a> Craniofacial-skeletal-dermatologic dysplasia - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/123150">OMIM:123150</a> Jackson-Weiss syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/123500">OMIM:123500</a> Crouzon syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/123790">OMIM:123790</a> Beare-Stevenson cutis gyrata syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/149730">OMIM:149730</a> LADD syndrome 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/207410">OMIM:207410</a> Antley-Bixler syndrome without genital anomalies or disordered steroidogenesis - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/609579">OMIM:609579</a> ?Scaphocephaly, maxillary retrusion, and impaired intellectual development (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/613659">OMIM:613659</a> Gastric cancer, somatic - somatic</dd>
<dd><a href="http://www.omim.org/entry/614592">OMIM:614592</a> Bent bone dysplasia syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1540">ORPHA:1540</a> Jackson-Weiss syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1555">ORPHA:1555</a> Cutis gyrata-acanthosis nigricans-craniosynostosis syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=168624">ORPHA:168624</a> Familial scaphocephaly syndrome, McGillivray type - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=207">ORPHA:207</a> Crouzon syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2363">ORPHA:2363</a> Lacrimoauriculodentodigital syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=313855">ORPHA:313855</a> FGFR2-related bent bone dysplasia - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=794">ORPHA:794</a> Saethre-Chotzen syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=87">ORPHA:87</a> Apert syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93258">ORPHA:93258</a> Pfeiffer syndrome type 1 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93259">ORPHA:93259</a> Pfeiffer syndrome type 2 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93260">ORPHA:93260</a> Pfeiffer syndrome type 3 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.995</b>
                                    <data>(p=1.6E-5)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.896</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.848 to ORPHA:87 Apert syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.840 to OMIM:101600 Craniofacial-skeletal-dermatologic dysplasia</dt>
                                
                                    <dt>Phenotypic similarity 0.814 to OMIM:101200 Apert syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.785 to ORPHA:93260 Pfeiffer syndrome type 3</dt>
                                
                                    <dt>Phenotypic similarity 0.778 to ORPHA:207 Crouzon syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.748 to OMIM:123500 Crouzon syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.748 to ORPHA:1540 Jackson-Weiss syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.739 to ORPHA:794 Saethre-Chotzen syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.732 to OMIM:101400 Saethre-Chotzen syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.684 to ORPHA:1555 Cutis gyrata-acanthosis nigricans-craniosynostosis syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.648 to OMIM:207410 Antley-Bixler syndrome without genital anomalies or disordered steroidogenesis</dt>
                                
                                    <dt>Phenotypic similarity 0.639 to ORPHA:93258 Pfeiffer syndrome type 1</dt>
                                
                                    <dt>Phenotypic similarity 0.634 to OMIM:123790 Beare-Stevenson cutis gyrata syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.633 to ORPHA:93259 Pfeiffer syndrome type 2</dt>
                                
                                    <dt>Phenotypic similarity 0.596 to ORPHA:168624 Familial scaphocephaly syndrome, McGillivray type</dt>
                                
                                    <dt>Phenotypic similarity 0.538 to OMIM:614592 Bent bone dysplasia syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.529 to ORPHA:313855 FGFR2-related bent bone dysplasia</dt>
                                
                                    <dt>Phenotypic similarity 0.507 to ORPHA:2363 Lacrimoauriculodentodigital syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.494 to OMIM:149730 LADD syndrome 1</dt>
                                
                                    <dt>Phenotypic similarity 0.463 to OMIM:123150 Jackson-Weiss syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-123256215-T-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs121918506">rs121918506</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                <span class="label label-danger">PATHOGENIC</span>
                                
                                
                                <b>[PM2, PP3_Strong, PP4, PP5_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-danger">PATHOGENIC_OR_LIKELY_PATHOGENIC</span>
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=28333%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                <span class="label label-danger">WHITELIST VARIANT</span>
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346997.2">FGFR2:ENST00000346997.2:c.1688A&gt;C:p.(Glu563Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351936.6">FGFR2:ENST00000351936.6:c.1688A&gt;C:p.(Glu563Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356226.4">FGFR2:ENST00000356226.4:c.1343A&gt;C:p.(Glu448Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357555.5">FGFR2:ENST00000357555.5:c.1427A&gt;C:p.(Glu476Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358487.5">FGFR2:ENST00000358487.5:c.1694A&gt;C:p.(Glu565Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360144.3">FGFR2:ENST00000360144.3:c.1430A&gt;C:p.(Glu477Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369056.1">FGFR2:ENST00000369056.1:c.1697A&gt;C:p.(Glu566Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369059.1">FGFR2:ENST00000369059.1:c.1352A&gt;C:p.(Glu451Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369060.4">FGFR2:ENST00000369060.4:c.1346A&gt;C:p.(Glu449Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369061.4">FGFR2:ENST00000369061.4:c.1358A&gt;C:p.(Glu453Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457416.2">FGFR2:ENST00000457416.2:c.1697A&gt;C:p.(Glu566Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000478859.1">FGFR2:ENST00000478859.1:c.1010A&gt;C:p.(Glu337Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.965</dd>
                                    
                                    <dd>REVEL: 0.965</dd>
                                    <dd>MVP: 0.952</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF366" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178175">ZNF366</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.910</b>
                    <data>(p=2.1E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.802</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.791</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.802 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZNF366">ZNF366</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus - MP:0001297, microphthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0001297, microphthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001297, microphthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0001297, microphthalmia</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF366%0DCTBP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CTBP1</a> and phenotypic similarity 0.646 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=280">Wolf-Hirschhorn syndrome</a> associated with CTBP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001166, Arachnodactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001362, Calvarial skull defect</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.910</b>
                                    <data>(p=2.1E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.802</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.791</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>5-71755984-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs375204168">rs375204168</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.791</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000318442.5">ZNF366:ENST00000318442.5:c.1332+8G&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0110%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AMR: 0.0798%</dd>
                                    <dd>ExAC NFE: 0.0109%</dd>
                                    <dd>gnomAD_E_AMR: 0.0715%</dd>
                                    <dd>gnomAD_E_NFE: 0.0109%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PKD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000008710">PKD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.807</b>
                    <data>(p=5.6E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.647</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.871</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.647 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PKD1">PKD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002639, micrognathia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0002639, micrognathia</dd><dd>HP:0000601, Hypotelorism - MP:0002639, micrognathia</dd><dd>HP:0000244, Brachyturricephaly - MP:0002639, micrognathia</dd></dl><dl><dt>Phenotypic similarity 0.266 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=PKD1">PKD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001708, mandibular arch skeleton malformed, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKD1%0DIFT88&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IFT88</a> and phenotypic similarity 0.312 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with IFT88.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000602, Ophthalmoplegia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000431, Wide nasal bridge</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKD1%0DIFT88&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to IFT88</a> and phenotypic similarity 0.687 to mouse mutant of IFT88.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000562, polydactyly</dd><dd>HP:0000486, Strabismus - MP:0010773, supernumerary molars</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0010773, supernumerary molars</dd><dd>HP:0000520, Proptosis - MP:0010773, supernumerary molars</dd><dd>HP:0000601, Hypotelorism - MP:0010773, supernumerary molars</dd><dd>HP:0000244, Brachyturricephaly - MP:0003840, abnormal coronal suture morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/173900">OMIM:173900</a> Polycystic kidney disease 1 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=730">ORPHA:730</a> Autosomal dominant polycystic kidney disease - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.807</b>
                                    <data>(p=5.6E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.647</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.871</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-2150254-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147967021">rs147967021</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1319391%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.871</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262304.4">PKD1:ENST00000262304.4:c.9625C&gt;T:p.(Arg3209Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423118.1">PKD1:ENST00000423118.1:c.9625C&gt;T:p.(Arg3209Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8792868</dd>
                                    
                                    <dd>REVEL: 0.346</dd>
                                    <dd>MVP: 0.879</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0079%</dd>
                                    <dd>ExAC AMR: 0.0698%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.0067%</dd>
                                    <dd>gnomAD_E_AMR: 0.0507%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0130%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CCDC66" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000180376">CCDC66</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.807</b>
                    <data>(p=5.6E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.534</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.534 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CCDC66">CCDC66</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004022, abnormal cone electrophysiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0000520, Proptosis - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0000601, Hypotelorism - MP:0004021, abnormal rod electrophysiology</dd><dd>HP:0000244, Brachyturricephaly - MP:0003728, abnormal retina photoreceptor layer morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.807</b>
                                    <data>(p=5.6E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.534</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.998</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                            <b>DEL</b>
                            <b>3-56653839-CTG-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs751329549">rs751329549</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326595.7">CCDC66:ENST00000326595.7:c.2572_2573del:p.(Val858Glnfs*6)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394672.3">CCDC66:ENST00000394672.3:c.2674_2675del:p.(Val892Glnfs*6)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000436465.2">CCDC66:ENST00000436465.2:c.2674_2675del:p.(Val892Glnfs*6)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MUC6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184956">MUC6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.756</b>
                    <data>(p=8.3E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MUC6%0DGALNT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GALNT2</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/618885">Congenital disorder of glycosylation, type IIt</a> associated with GALNT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001852, Sandal gap</dd><dd>HP:0000486, Strabismus - HP:0031717, Alternating exotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0010804, Tented upper lip vermilion</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.756</b>
                                    <data>(p=8.3E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.999</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                            <b>DEL</b>
                            <b>11-1018088-TG-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765231061">rs765231061</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421673.2">MUC6:ENST00000421673.2:c.4712del:p.(Pro1571Hisfs*21)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                    <dd>gnomAD_G_NFE: 0.0070%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.756</b>
                                    <data>(p=8.3E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.999</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                            <b>DEL</b>
                            <b>11-1018088-TG-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765231061">rs765231061</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421673.2">MUC6:ENST00000421673.2:c.4712del:p.(Pro1571Hisfs*21)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0031%</dd>
                                    <dd>gnomAD_G_NFE: 0.0070%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                            <b>INS</b>
                            <b>11-1018093-G-GT [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs376177791">rs376177791</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421673.2">MUC6:ENST00000421673.2:c.4707dup:p.(Pro1570Thrfs*136)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_G_NFE: 0.0078%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DDX56" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136271">DDX56</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.754</b>
                    <data>(p=8.5E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.374 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=DDX56">DDX56</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000407, head decreased width, abnormal</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX56%0DPAK1IP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PAK1IP1</a> and phenotypic similarity 0.617 to mouse mutant of PAK1IP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000520, Proptosis - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0012138, decreased forebrain size</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.754</b>
                                    <data>(p=8.5E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.999</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>7-44610376-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs774566321">rs774566321</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258772.5">DDX56:ENST00000258772.5:c.991C&gt;T:p.(Arg331*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431640.1">DDX56:ENST00000431640.1:c.891-649C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ENPP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197594">ENPP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.754</b>
                    <data>(p=8.5E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.367 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=289176">Autosomal recessive hypophosphatemic rickets</a> associated with ENPP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0030757, Tooth abscess</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0001363, Craniosynostosis</dd></dl><dl><dt>Phenotypic similarity 0.499 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ENPP1">ENPP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0013945, abnormal elbow joint morphology</dd><dd>HP:0000486, Strabismus - MP:0001325, abnormal retina morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030533, abnormal snout skin morphology</dd><dd>HP:0000520, Proptosis - MP:0001325, abnormal retina morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001325, abnormal retina morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001325, abnormal retina morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ENPP1%0DGALNT2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GALNT2</a> and phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/618885">Congenital disorder of glycosylation, type IIt</a> associated with GALNT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001852, Sandal gap</dd><dd>HP:0000486, Strabismus - HP:0031717, Alternating exotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0010804, Tented upper lip vermilion</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/125853">OMIM:125853</a> Diabetes mellitus, non-insulin-dependent, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/208000">OMIM:208000</a> Arterial calcification, generalized, of infancy, 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/601665">OMIM:601665</a> Obesity, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/613312">OMIM:613312</a> Hypophosphatemic rickets, autosomal recessive, 2 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/615522">OMIM:615522</a> Cole disease - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=289176">ORPHA:289176</a> Autosomal recessive hypophosphatemic rickets - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=51608">ORPHA:51608</a> Generalized arterial calcification of infancy - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=758">ORPHA:758</a> Pseudoxanthoma elasticum - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.754</b>
                                    <data>(p=8.5E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                            <b>SNV</b>
                            <b>6-132203615-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs770775549">rs770775549</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PVS1_Strong]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360971.2">ENPP1:ENST00000360971.2:c.2230+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BDKRB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100739">BDKRB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.752</b>
                    <data>(p=8.7E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BDKRB1%0DOPN4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to OPN4</a> and phenotypic similarity 0.812 to mouse mutant of OPN4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006156, abnormal visual pursuit</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0006156, abnormal visual pursuit</dd><dd>HP:0000520, Proptosis - MP:0006156, abnormal visual pursuit</dd><dd>HP:0000601, Hypotelorism - MP:0006156, abnormal visual pursuit</dd><dd>HP:0000244, Brachyturricephaly - MP:0008056, abnormal retina ganglion cell morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.752</b>
                                    <data>(p=8.7E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>14-96730313-G-A [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000216629.6">BDKRB1:ENST00000216629.6:c.294G&gt;A:p.(Trp98*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553356.1">BDKRB1:ENST00000553356.1:c.294G&gt;A:p.(Trp98*)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="COL4A1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187498">COL4A1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.752</b>
                    <data>(p=8.8E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.984</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.383 to <a href="http://www.omim.org/entry/175780">Brain small vessel disease with or without ocular anomalies</a> associated with COL4A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000577, Exotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.516 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=COL4A1">COL4A1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006203, eye hemorrhage</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0010263, total cataract</dd><dd>HP:0000520, Proptosis - MP:0010263, total cataract</dd><dd>HP:0000601, Hypotelorism - MP:0010263, total cataract</dd><dd>HP:0000244, Brachyturricephaly - MP:0010263, total cataract</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=COL4A1%0DCOL7A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL7A1</a> and phenotypic similarity 0.338 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79410">Localized dystrophic epidermolysis bullosa, pretibial form</a> associated with COL7A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0008404, Nail dystrophy</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000670, Carious teeth</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=COL4A1%0DCOL7A1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL7A1</a> and phenotypic similarity 0.640 to mouse mutant of COL7A1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000520, Proptosis - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000762, abnormal tongue morphology</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/175780">OMIM:175780</a> Brain small vessel disease with or without ocular anomalies - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/180000">OMIM:180000</a> ?Retinal arteries, tortuosity of (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/611773">OMIM:611773</a> Angiopathy, hereditary, with nephropathy, aneurysms, and muscle cramps - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618564">OMIM:618564</a> Microangiopathy and leukoencephalopathy, pontine, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=73229">ORPHA:73229</a> HANAC syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=899">ORPHA:899</a> Walker-Warburg syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.752</b>
                                    <data>(p=8.8E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.984</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.383 to OMIM:175780 Brain small vessel disease with or without ocular anomalies</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-110855918-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150182714">rs150182714</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP3_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=333515%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.984</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375820.4">COL4A1:ENST00000375820.4:c.994G&gt;C:p.(Gly332Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543140.1">COL4A1:ENST00000543140.1:c.994G&gt;C:p.(Gly332Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9869305</dd>
                                    
                                    <dd>REVEL: 0.886</dd>
                                    <dd>MVP: 0.987</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0068%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0075%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRT4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170477">KRT4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.749</b>
                    <data>(p=9.2E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.484 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=KRT4">KRT4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006000, abnormal corneal epithelium morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000764, abnormal tongue epithelium morphology</dd><dd>HP:0000520, Proptosis - MP:0006000, abnormal corneal epithelium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0006000, abnormal corneal epithelium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0006000, abnormal corneal epithelium morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT4%0DSMO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMO</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2495">Meningioma</a> associated with SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0012246, Oculomotor nerve palsy</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT4%0DSMO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMO</a> and phenotypic similarity 0.467 to mouse mutant of SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000520, Proptosis - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000601, Hypotelorism - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000244, Brachyturricephaly - MP:0003425, abnormal optic vesicle formation</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KRT4%0DSMO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMO</a> and phenotypic similarity 0.462 to fish mutant of SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - ZP:0010276, eye adjacent to eye, abnormal</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001147, mandibular arch skeleton aplastic, abnormal</dd><dd>HP:0000520, Proptosis - ZP:0010276, eye adjacent to eye, abnormal</dd><dd>HP:0000601, Hypotelorism - ZP:0010276, eye adjacent to eye, abnormal</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/193900">OMIM:193900</a> White sponge nevus 1 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.749</b>
                                    <data>(p=9.2E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                            <b>INS</b>
                            <b>12-53207583-C-CCCGG [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                <span class="label label-danger">PATHOGENIC</span>
                                
                                
                                <b>[PVS1, PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000293774.4">KRT4:ENST00000293774.4:c.481_482insCCGG:p.(Gly161Alafs*11)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458244.2">KRT4:ENST00000458244.2:c.199_200insCCGG:p.(Gly67Alafs*11)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000551956.1">KRT4:ENST00000551956.1:c.259_260insCCGG:p.(Gly87Alafs*11)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPAG11A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178287">SPAG11A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.747</b>
                    <data>(p=9.7E-3)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPAG11A%0DSOX11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SOX11</a> and phenotypic similarity 0.499 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1465">Coffin-Siris syndrome</a> associated with SOX11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000219, Thin upper lip vermilion</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPAG11A%0DSOX11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SOX11</a> and phenotypic similarity 0.864 to mouse mutant of SOX11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0005306, abnormal phalanx morphology</dd><dd>HP:0000486, Strabismus - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000457, maxilla hypoplasia</dd><dd>HP:0000520, Proptosis - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0004452, abnormal pterygoid process morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPAG11A%0DSOX11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SOX11</a> and phenotypic similarity 0.310 to fish mutant of SOX11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.747</b>
                                    <data>(p=9.7E-3)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.999</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                            <b>INS</b>
                            <b>8-7718227-C-CT [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs766847121">rs766847121</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326625.5">SPAG11A:ENST00000326625.5:c.122dup:p.(Cys42Leufs*5)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326558.5">SPAG11A:ENST00000326558.5:c.281dup:p.(Cys95Leufs*64)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351436.4">SPAG11A:ENST00000351436.4:c.215-2796_215-2795insT:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434307.2">SPAG11A:ENST00000434307.2:c.*45dup:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454911.2">SPAG11A:ENST00000454911.2:c.*45dup:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC AMR: 0.0089%</dd>
                                    <dd>ExAC NFE: 0.0080%</dd>
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                    <dd>gnomAD_E_NFE: 0.0058%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BTBD9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183826">BTBD9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.729</b>
                    <data>(p=1.4E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.988</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.172 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=BTBD9">BTBD9</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000877, abnormal Purkinje cell morphology</dd></dl><dl><dt>Phenotypic similarity 0.310 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=BTBD9">BTBD9</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BTBD9%0DTBX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX2</a> and phenotypic similarity 0.595 to <a href="http://www.omim.org/entry/618223">Vertebral anomalies and variable endocrine and T-cell dysfunction</a> associated with TBX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0009276, Contracture of the proximal interphalangeal joint of the 4th finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000189, Narrow palate</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BTBD9%0DTBX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TBX2</a> and phenotypic similarity 0.663 to mouse mutant of TBX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000562, polydactyly</dd><dd>HP:0000486, Strabismus - MP:0009883, palatal shelf hypoplasia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009883, palatal shelf hypoplasia</dd><dd>HP:0000520, Proptosis - MP:0009883, palatal shelf hypoplasia</dd><dd>HP:0000601, Hypotelorism - MP:0009883, palatal shelf hypoplasia</dd><dd>HP:0000244, Brachyturricephaly - MP:0009883, palatal shelf hypoplasia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.729</b>
                                    <data>(p=1.4E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.988</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-38256069-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs375379024">rs375379024</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.988</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000314100.6">BTBD9:ENST00000314100.6:c.1229C&gt;T:p.(Pro410Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403056.1">BTBD9:ENST00000403056.1:c.1433C&gt;T:p.(Pro478Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408958.1">BTBD9:ENST00000408958.1:c.1229C&gt;T:p.(Pro410Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000419706.2">BTBD9:ENST00000419706.2:c.1343C&gt;T:p.(Pro448Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000481247.1">BTBD9:ENST00000481247.1:c.1433C&gt;T:p.(Pro478Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9924212</dd>
                                    
                                    <dd>REVEL: 0.784</dd>
                                    <dd>MVP: 0.992</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0045%</dd>
                                    <dd>ESP EA: 0.0118%</dd>
                                    <dd>ESP All: 0.0079%</dd>
                                    <dd>ExAC FIN: 0.0302%</dd>
                                    <dd>ExAC NFE: 0.0210%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_FIN: 0.0224%</dd>
                                    <dd>gnomAD_E_NFE: 0.0152%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RBM23" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100461">RBM23</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.719</b>
                    <data>(p=1.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.983</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBM23%0DTET1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TET1</a> and phenotypic similarity 0.801 to mouse mutant of TET1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001300, ocular hypertelorism</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0001300, ocular hypertelorism</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.719</b>
                                    <data>(p=1.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.983</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                            <b>DEL</b>
                            <b>14-23371269-GCA-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs762848810">rs762848810</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399922.2">RBM23:ENST00000399922.2:c.1116_1117del:p.(Ala377Profs*82)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542016.2">RBM23:ENST00000542016.2:c.654_655del:p.(Ala223Profs*82)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346528.5">RBM23:ENST00000346528.5:c.1062_1063del:p.(Ala359Profs*82)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359890.3">RBM23:ENST00000359890.3:c.1164_1165del:p.(Ala393Profs*82)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555209.1">RBM23:ENST00000555209.1:c.414_415del:p.(Ala143Profs*82)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0092%</dd>
                                    <dd>gnomAD_E_NFE: 0.0092%</dd>
                                    <dd>gnomAD_E_SAS: 0.0100%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                            <b>DEL</b>
                            <b>14-23371267-CA-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs754204069">rs754204069</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.983</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346528.5">RBM23:ENST00000346528.5:c.1065del:p.(Ala355Alafs*12)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359890.3">RBM23:ENST00000359890.3:c.1167del:p.(Ala389Alafs*12)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399922.2">RBM23:ENST00000399922.2:c.1119del:p.(Ala373Alafs*12)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542016.2">RBM23:ENST00000542016.2:c.657del:p.(Ala219Alafs*12)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555209.1">RBM23:ENST00000555209.1:c.417del:p.(Ala139Alafs*12)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0124%</dd>
                                    <dd>gnomAD_E_EAS: 0.0059%</dd>
                                    <dd>gnomAD_E_NFE: 0.0094%</dd>
                                    <dd>gnomAD_E_SAS: 0.0103%</dd>
                                    <dd>gnomAD_G_AMR: 0.1202%</dd>
                                    <dd>gnomAD_G_NFE: 0.0334%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KCNG2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178342">KCNG2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.650</b>
                    <data>(p=3.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.944</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=KCNG2%0DKCNAB2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KCNAB2</a> and phenotypic similarity 0.615 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with KCNAB2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000307, Pointed chin</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.650</b>
                                    <data>(p=3.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.944</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>18-77659045-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200870939">rs200870939</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.944</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316249.3">KCNG2:ENST00000316249.3:c.630G&gt;C:p.(Glu210Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9478476</dd>
                                    
                                    <dd>REVEL: 0.429</dd>
                                    <dd>MVP: 0.948</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0057%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0261%</dd>
                                    <dd>gnomAD_E_AMR: 0.0034%</dd>
                                    <dd>gnomAD_E_NFE: 0.0327%</dd>
                                    <dd>gnomAD_E_OTH: 0.0227%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ABI3BP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000154175">ABI3BP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.610</b>
                    <data>(p=3.7E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.930</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ABI3BP%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.745 to <a href="http://www.omim.org/entry/259600">Multicentric osteolysis, nodulosis, and arthropathy</a> associated with MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0025131, Finger swelling</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ABI3BP%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.713 to mouse mutant of MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0004686, decreased length of long bones</dd><dd>HP:0000486, Strabismus - MP:0030166, increased inner canthal distance</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0030166, increased inner canthal distance</dd><dd>HP:0000601, Hypotelorism - MP:0030166, increased inner canthal distance</dd><dd>HP:0000244, Brachyturricephaly - MP:0003840, abnormal coronal suture morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ABI3BP%0DMMP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP2</a> and phenotypic similarity 0.272 to fish mutant of MMP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001052, mandibular arch skeleton morphology, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.610</b>
                                    <data>(p=3.7E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.930</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-100605051-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199740093">rs199740093</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.930</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000284322.5">ABI3BP:ENST00000284322.5:c.599G&gt;T:p.(Gly200Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000471714.1">ABI3BP:ENST00000471714.1:c.599G&gt;T:p.(Gly200Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000495063.1">ABI3BP:ENST00000495063.1:c.599G&gt;T:p.(Gly200Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9308898</dd>
                                    
                                    <dd>REVEL: 0.497</dd>
                                    <dd>MVP: 0.931</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                    <dd>ExAC AMR: 0.0088%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0091%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HNF1A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135100">HNF1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.555</b>
                    <data>(p=4.8E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.903</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HNF1A%0DCTBP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CTBP1</a> and phenotypic similarity 0.646 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=280">Wolf-Hirschhorn syndrome</a> associated with CTBP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001166, Arachnodactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001362, Calvarial skull defect</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/125853">OMIM:125853</a> Diabetes mellitus, noninsulin-dependent, 2 (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/142330">OMIM:142330</a> Hepatic adenoma, somatic - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/144700">OMIM:144700</a> Renal cell carcinoma - unknown</dd>
<dd><a href="http://www.omim.org/entry/222100">OMIM:222100</a> Diabetes mellitus, insulin-dependent (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/600496">OMIM:600496</a> MODY, type III - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/612520">OMIM:612520</a> Diabetes mellitus, insulin-dependent, 20 - unknown</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=552">ORPHA:552</a> MODY - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.555</b>
                                    <data>(p=4.8E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.903</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-121416797-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1320041799">rs1320041799</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2, PP3]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.903</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000257555.6">HNF1A:ENST00000257555.6:c.226G&gt;A:p.(Asp76Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400024.2">HNF1A:ENST00000400024.2:c.226G&gt;A:p.(Asp76Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000402929.1">HNF1A:ENST00000402929.1:c.226G&gt;A:p.(Asp76Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541395.1">HNF1A:ENST00000541395.1:c.226G&gt;A:p.(Asp76Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544413.1">HNF1A:ENST00000544413.1:c.226G&gt;A:p.(Asp76Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538626.1">HNF1A:ENST00000538626.1:c.72+154G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543427.1">HNF1A:ENST00000543427.1:c.-51-75G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.90334725</dd>
                                    
                                    <dd>REVEL: 0.669</dd>
                                    <dd>MVP: 0.903</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FARP2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000006607">FARP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.430</b>
                    <data>(p=7.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FARP2%0DSEMA3F&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SEMA3F</a> and phenotypic similarity 0.607 to mouse mutant of SEMA3F.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000579, abnormal nail morphology</dd><dd>HP:0000486, Strabismus - MP:0001304, cataract</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001304, cataract</dd><dd>HP:0000520, Proptosis - MP:0001304, cataract</dd><dd>HP:0000601, Hypotelorism - MP:0001304, cataract</dd><dd>HP:0000244, Brachyturricephaly - MP:0001304, cataract</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.430</b>
                                    <data>(p=7.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                            <b>INS</b>
                            <b>2-242432770-C-CGAT [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264042.3">FARP2:ENST00000264042.3:c.2960_2962dup:p.(Asp987dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000316586.4">FARP2:ENST00000316586.4:c.*2304_*2306dup:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PITRM1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107959">PITRM1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.430</b>
                    <data>(p=7.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.168 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PITRM1">PITRM1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0003224, neuron degeneration</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PITRM1%0DITGB1BP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ITGB1BP1</a> and phenotypic similarity 0.837 to mouse mutant of ITGB1BP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0004509, abnormal pelvic girdle bone morphology</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030189, broad snout</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0004420, parietal bone hypoplasia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619405">OMIM:619405</a> Spinocerebellar ataxia, autosomal recessive 30 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.430</b>
                                    <data>(p=7.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                            <b>INS</b>
                            <b>10-3208567-T-TGCACGCTAGGGAAGAGAGAGG [1/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000224949.4">PITRM1:ENST00000224949.4:c.271_272insCCTCTCTCTTCCCTAGCGTGC:p.(Val90_Gln91insProSerLeuPheProSerVal)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380989.2">PITRM1:ENST00000380989.2:c.271_272insCCTCTCTCTTCCCTAGCGTGC:p.(Val90_Gln91insProSerLeuPheProSerVal)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451104.2">PITRM1:ENST00000451104.2:c.175_176insCCTCTCTCTTCCCTAGCGTGC:p.(Val58_Gln59insProSerLeuPheProSerVal)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRB4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000230657">PRB4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.430</b>
                    <data>(p=7.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRB4%0DFBN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBN2</a> and phenotypic similarity 0.584 to <a href="http://www.omim.org/entry/121050">Contractural arachnodactyly, congenital</a> associated with FBN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001181, Adducted thumb</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRB4%0DFBN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FBN2</a> and phenotypic similarity 0.828 to mouse mutant of FBN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus - MP:0002092, abnormal eye morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002092, abnormal eye morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.430</b>
                                    <data>(p=7.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_INSERTION</span>
                            <b>INS</b>
                            <b>12-11461553-T-TTTCTGGCTTTCCTGGATGAGGTGGGGGACCTTGGGACTGGTTGCCTCCTTGTGGGGGTCGTCC [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs71057722">rs71057722</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000279575.1">PRB4:ENST00000279575.1:c.363_364insGGACGACCCCCACAAGGAGGCAACCAGTCCCAAGGTCCCCCACCTCATCCAGGAAAGCCAGAA:p.(Glu121_Arg122insGlyArgProProGlnGlyGlyAsnGlnSerGlnGlyProProProHisProGlyLysProGlu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535904.1">PRB4:ENST00000535904.1:c.363_364insGGACGACCCCCACAAGGAGGCAACCAGTCCCAAGGTCCCCCACCTCATCCAGGAAAGCCAGAA:p.(Glu121_Arg122insGlyArgProProGlnGlyGlyAsnGlnSerGlnGlyProProProHisProGlyLysProGlu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000445719.2">PRB4:ENST00000445719.2:c.334+29_334+30insGGACGACCCCCACAAGGAGGCAACCAGTCCCAAGGTCCCCCACCTCATCCAGGAAAGCCAGAA:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMPRSS7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000176040">TMPRSS7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.395</b>
                    <data>(p=7.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.835</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMPRSS7%0DTRAF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAF7</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2495">Meningioma</a> associated with TRAF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0012246, Oculomotor nerve palsy</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMPRSS7%0DTRAF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAF7</a> and phenotypic similarity 0.491 to mouse mutant of TRAF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.395</b>
                                    <data>(p=7.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.835</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-111795803-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1224317799">rs1224317799</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.835</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000419127.1">TMPRSS7:ENST00000419127.1:c.1658G&gt;T:p.(Arg553Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452346.2">TMPRSS7:ENST00000452346.2:c.2036G&gt;T:p.(Arg679Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.835</dd>
                                    
                                    <dd>REVEL: 0.835</dd>
                                    <dd>MVP: 0.718</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MYH3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000109063">MYH3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.373</b>
                    <data>(p=8.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.562</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.754</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.585 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2990">Autosomal recessive multiple pterygium syndrome</a> associated with MYH3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0006101, Finger syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MYH3%0DRABL2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RABL2B</a> and phenotypic similarity 0.697 to mouse mutant of RABL2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0006203, eye hemorrhage</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0006203, eye hemorrhage</dd><dd>HP:0000520, Proptosis - MP:0006203, eye hemorrhage</dd><dd>HP:0000601, Hypotelorism - MP:0006203, eye hemorrhage</dd><dd>HP:0000244, Brachyturricephaly - MP:0006203, eye hemorrhage</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/178110">OMIM:178110</a> Contractures, pterygia, and spondylocarpostarsal fusion syndrome 1A - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/193700">OMIM:193700</a> Arthrogryposis, distal, type 2A (Freeman-Sheldon) - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618436">OMIM:618436</a> Arthrogryposis, distal, type 2B3 (Sheldon-Hall) - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618469">OMIM:618469</a> Contractures, pterygia, and spondylocarpotarsal fusion syndrome 1B - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1146">ORPHA:1146</a> Distal arthrogryposis type 1 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1147">ORPHA:1147</a> Sheldon-Hall syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2053">ORPHA:2053</a> Freeman-Sheldon syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2053">ORPHA:2053</a> Freeman-Sheldon syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2990">ORPHA:2990</a> Autosomal recessive multiple pterygium syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.373</b>
                                    <data>(p=8.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.562</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.754</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.562 to OMIM:193700 Arthrogryposis, distal, type 2A (Freeman-Sheldon)</dt>
                                
                                    <dt>Phenotypic similarity 0.547 to ORPHA:2053 Freeman-Sheldon syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.421 to ORPHA:1147 Sheldon-Hall syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.376 to OMIM:178110 Contractures, pterygia, and spondylocarpostarsal fusion syndrome 1A</dt>
                                
                                    <dt>Phenotypic similarity 0.288 to ORPHA:1146 Distal arthrogryposis type 1</dt>
                                
                                    <dt>Phenotypic similarity 0.282 to OMIM:618436 Arthrogryposis, distal, type 2B3 (Sheldon-Hall)</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-10547753-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs769788909">rs769788909</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2003746%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.754</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000226209.7">MYH3:ENST00000226209.7:c.1325C&gt;T:p.(Thr442Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583535.1">MYH3:ENST00000583535.1:c.1325C&gt;T:p.(Thr442Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7579609</dd>
                                    
                                    <dd>REVEL: 0.247</dd>
                                    <dd>MVP: 0.758</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0083%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.034</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.585</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.419</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.585 to ORPHA:2990 Autosomal recessive multiple pterygium syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.308 to OMIM:618469 Contractures, pterygia, and spondylocarpotarsal fusion syndrome 1B</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-10547753-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs769788909">rs769788909</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2003746%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.754</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000226209.7">MYH3:ENST00000226209.7:c.1325C&gt;T:p.(Thr442Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583535.1">MYH3:ENST00000583535.1:c.1325C&gt;T:p.(Thr442Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7579609</dd>
                                    
                                    <dd>REVEL: 0.247</dd>
                                    <dd>MVP: 0.758</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0083%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-10551919-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147148934">rs147148934</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=256070%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.084</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000226209.7">MYH3:ENST00000226209.7:c.690C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000583535.1">MYH3:ENST00000583535.1:c.690C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3195%</dd>
                                    <dd>TOPMed: 0.4911%</dd>
                                    <dd>UK10K: 0.4761%</dd>
                                    <dd>ESP AA: 0.2270%</dd>
                                    <dd>ESP EA: 0.4767%</dd>
                                    <dd>ESP All: 0.3921%</dd>
                                    <dd>ExAC AFR: 0.1922%</dd>
                                    <dd>ExAC AMR: 0.1900%</dd>
                                    <dd>ExAC FIN: 0.1210%</dd>
                                    <dd>ExAC NFE: 0.6068%</dd>
                                    <dd>ExAC OTH: 0.4405%</dd>
                                    <dd>ExAC SAS: 0.7510%</dd>
                                    <dd>gnomAD_E_AFR: 0.1242%</dd>
                                    <dd>gnomAD_E_AMR: 0.3454%</dd>
                                    <dd>gnomAD_E_FIN: 0.1345%</dd>
                                    <dd>gnomAD_E_NFE: 0.5791%</dd>
                                    <dd>gnomAD_E_OTH: 0.6380%</dd>
                                    <dd>gnomAD_E_SAS: 0.7894%</dd>
                                    <dd>gnomAD_G_AFR: 0.2176%</dd>
                                    <dd>gnomAD_G_AMR: 0.5967%</dd>
                                    <dd>gnomAD_G_FIN: 0.0859%</dd>
                                    <dd>gnomAD_G_NFE: 0.3866%</dd>
                                    <dd>gnomAD_G_OTH: 0.3055%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMPRSS9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178297">TMPRSS9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.362</b>
                    <data>(p=8.3E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.819</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMPRSS9%0DBHLHA9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BHLHA9</a> and phenotypic similarity 0.520 to <a href="http://www.omim.org/entry/607539">?Camptosynpolydactyly, complex</a> associated with BHLHA9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001159, Syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TMPRSS9%0DBHLHA9&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BHLHA9</a> and phenotypic similarity 0.631 to mouse mutant of BHLHA9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.362</b>
                                    <data>(p=8.3E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.819</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-2415775-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765108993">rs765108993</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.819</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332578.3">TMPRSS9:ENST00000332578.3:c.1579C&gt;A:p.(His527Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.82390845</dd>
                                    
                                    <dd>REVEL: 0.571</dd>
                                    <dd>MVP: 0.824</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_AMR: 0.0426%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0186%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ECPAS" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136813">ECPAS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.361</b>
                    <data>(p=8.3E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.519 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ECPAS%0DUBE3B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE3B</a> and phenotypic similarity 0.637 to <a href="http://www.omim.org/entry/244450">Kaufman oculocerebrofacial syndrome</a> associated with UBE3B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004209, Clinodactyly of the 5th finger</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.519 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ECPAS%0DUBE3B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE3B</a> and phenotypic similarity 0.531 to mouse mutant of UBE3B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006203, eye hemorrhage</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0003733, abnormal retina inner nuclear layer morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.361</b>
                                    <data>(p=8.3E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.798</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>9-114131316-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs541919185">rs541919185</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000259335.4">ECPAS:ENST00000259335.4:c.5646A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338205.5">ECPAS:ENST00000338205.5:c.5112A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374378.3">ECPAS:ENST00000374378.3:c.366+1399A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_NFE: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RIT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000152214">RIT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.347</b>
                    <data>(p=8.5E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.516</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.793</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIT2%0DRIT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RIT1</a> and phenotypic similarity 0.668 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=648">Noonan syndrome</a> associated with RIT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.516 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIT2%0DRIT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RIT1</a> and phenotypic similarity 0.535 to mouse mutant of RIT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0030245, round head</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000445, short snout</dd><dd>HP:0000520, Proptosis - MP:0030245, round head</dd><dd>HP:0000601, Hypotelorism - MP:0030245, round head</dd><dd>HP:0000244, Brachyturricephaly - MP:0030245, round head</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.347</b>
                                    <data>(p=8.5E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.516</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.793</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>INS</b>
                            <b>18-40613838-A-AG [1/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs766505085">rs766505085</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.793</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000282028.4">RIT2:ENST00000282028.4:c.104-7_104-6insC:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326695.5">RIT2:ENST00000326695.5:c.104-7_104-6insC:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589109.1">RIT2:ENST00000589109.1:c.104-7_104-6insC:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000590910.1">RIT2:ENST00000590910.1:c.104-7_104-6insC:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ExAC AMR: 0.0532%</dd>
                                    <dd>ExAC NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_AFR: 0.0071%</dd>
                                    <dd>gnomAD_E_AMR: 0.0245%</dd>
                                    <dd>gnomAD_E_NFE: 0.0091%</dd>
                                    <dd>gnomAD_E_OTH: 0.0592%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ITGB8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105855">ITGB8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.341</b>
                    <data>(p=8.7E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.433 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ITGB8">ITGB8</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000520, Proptosis - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000601, Hypotelorism - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0003731, abnormal retina outer nuclear layer morphology</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ITGB8%0DSP3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SP3</a> and phenotypic similarity 0.671 to mouse mutant of SP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0005306, abnormal phalanx morphology</dd><dd>HP:0000486, Strabismus - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000116, abnormal tooth development</dd><dd>HP:0000520, Proptosis - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000077, abnormal interparietal bone morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.341</b>
                                    <data>(p=8.7E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>7-20449239-A-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000222573.4">ITGB8:ENST00000222573.4:c.2025A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537992.1">ITGB8:ENST00000537992.1:c.1620A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TJP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104067">TJP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.340</b>
                    <data>(p=8.7E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TJP1%0DGJA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GJA5</a> and phenotypic similarity 0.642 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3303">Tetralogy of Fallot</a> associated with GJA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0009891, Underdeveloped supraorbital ridges</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.340</b>
                                    <data>(p=8.7E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.798</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>15-30065566-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs751106557">rs751106557</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346128.6">TJP1:ENST00000346128.6:c.85-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356107.6">TJP1:ENST00000356107.6:c.85-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000400011.2">TJP1:ENST00000400011.2:c.97-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000495972.2">TJP1:ENST00000495972.2:c.85-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545208.2">TJP1:ENST00000545208.2:c.85-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0091%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AMR: 0.0150%</dd>
                                    <dd>gnomAD_E_NFE: 0.0117%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PMVK" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163344">PMVK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.329</b>
                    <data>(p=8.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PMVK%0DPEX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PEX5</a> and phenotypic similarity 0.690 to <a href="http://www.omim.org/entry/214110">Peroxisome biogenesis disorder 2A (Zellweger)</a> associated with PEX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000262, Turricephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PMVK%0DPEX5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PEX5</a> and phenotypic similarity 0.262 to mouse mutant of PEX5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000788, abnormal cerebral cortex morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/175800">OMIM:175800</a> Porokeratosis 1, multiple types - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=735">ORPHA:735</a> Porokeratosis of Mibelli - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.329</b>
                                    <data>(p=8.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>1-154901497-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1305787888">rs1305787888</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368467.3">PMVK:ENST00000368467.3:c.312+3A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CASP6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138794">CASP6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.328</b>
                    <data>(p=8.9E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.460 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CASP6">CASP6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005545, abnormal lens development</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005545, abnormal lens development</dd><dd>HP:0000520, Proptosis - MP:0005545, abnormal lens development</dd><dd>HP:0000601, Hypotelorism - MP:0005545, abnormal lens development</dd><dd>HP:0000244, Brachyturricephaly - MP:0005545, abnormal lens development</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CASP6%0DTFAP2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFAP2A</a> and phenotypic similarity 0.548 to <a href="http://www.omim.org/entry/113620">Branchiooculofacial syndrome</a> associated with TFAP2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0009623, Proximal placement of thumb</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CASP6%0DTFAP2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFAP2A</a> and phenotypic similarity 0.696 to mouse mutant of TFAP2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004377, small frontal bone</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0004377, small frontal bone</dd><dd>HP:0000601, Hypotelorism - MP:0004377, small frontal bone</dd><dd>HP:0000244, Brachyturricephaly - MP:0004383, absent interparietal bone</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CASP6%0DTFAP2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TFAP2A</a> and phenotypic similarity 0.418 to fish mutant of TFAP2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001965, mandibular arch skeleton decreased size, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.328</b>
                                    <data>(p=8.9E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.797</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-110618781-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs377499378">rs377499378</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2381122%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265164.2">CASP6:ENST00000265164.2:c.227G&gt;A:p.(Arg76His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505486.1">CASP6:ENST00000505486.1:c.227G&gt;A:p.(Arg76His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000352981.3">CASP6:ENST00000352981.3:c.41-2925G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.798583</dd>
                                    
                                    <dd>REVEL: 0.124</dd>
                                    <dd>MVP: 0.799</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0045%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0046%</dd>
                                    <dd>ExAC SAS: 0.0068%</dd>
                                    <dd>gnomAD_E_NFE: 0.0037%</dd>
                                    <dd>gnomAD_E_SAS: 0.0072%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MTA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182979">MTA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.325</b>
                    <data>(p=9.0E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.791</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.435 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MTA1">MTA1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0009772, abnormal retina development</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009772, abnormal retina development</dd><dd>HP:0000520, Proptosis - MP:0009772, abnormal retina development</dd><dd>HP:0000601, Hypotelorism - MP:0009772, abnormal retina development</dd><dd>HP:0000244, Brachyturricephaly - MP:0009772, abnormal retina development</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTA1%0DPITX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX3</a> and phenotypic similarity 0.745 to mouse mutant of PITX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000520, Proptosis - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000601, Hypotelorism - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000244, Brachyturricephaly - MP:0001325, abnormal retina morphology</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTA1%0DPITX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PITX3</a> and phenotypic similarity 0.363 to fish mutant of PITX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001052, mandibular arch skeleton morphology, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.325</b>
                                    <data>(p=9.0E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.791</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>14-105924603-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs188950256">rs188950256</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.791</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331320.7">MTA1:ENST00000331320.7:c.551-4G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405646.1">MTA1:ENST00000405646.1:c.500-4G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406191.1">MTA1:ENST00000406191.1:c.551-4G&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.0801%</dd>
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0814%</dd>
                                    <dd>ESP All: 0.0615%</dd>
                                    <dd>ExAC AFR: 0.0409%</dd>
                                    <dd>ExAC AMR: 0.0530%</dd>
                                    <dd>ExAC NFE: 0.0595%</dd>
                                    <dd>gnomAD_E_AFR: 0.0397%</dd>
                                    <dd>gnomAD_E_AMR: 0.0596%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0833%</dd>
                                    <dd>gnomAD_E_OTH: 0.0734%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0468%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MOXD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000079931">MOXD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.322</b>
                    <data>(p=9.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MOXD1%0DEVX2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EVX2</a> and phenotypic similarity 0.620 to mouse mutant of EVX2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000547, short limbs</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.322</b>
                                    <data>(p=9.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>6-132649738-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs909076324">rs909076324</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000336749.3">MOXD1:ENST00000336749.3:c.460-5T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367963.3">MOXD1:ENST00000367963.3:c.664-5T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF33A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000189180">ZNF33A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.322</b>
                    <data>(p=9.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF33A%0DCDON&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDON</a> and phenotypic similarity 0.694 to <a href="http://www.omim.org/entry/614226">Holoprosencephaly 11</a> associated with CDON.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000175, Cleft palate</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF33A%0DCDON&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDON</a> and phenotypic similarity 0.698 to mouse mutant of CDON.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004462, small basisphenoid bone</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000520, Proptosis - MP:0004462, small basisphenoid bone</dd><dd>HP:0000601, Hypotelorism - MP:0004462, small basisphenoid bone</dd><dd>HP:0000244, Brachyturricephaly - MP:0004462, small basisphenoid bone</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.322</b>
                                    <data>(p=9.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>10-38301222-T-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374618.3">ZNF33A:ENST00000374618.3:c.-44-4T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458705.2">ZNF33A:ENST00000458705.2:c.-44-4T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000469037.2">ZNF33A:ENST00000469037.2:c.-44-4T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432900.2">ZNF33A:ENST00000432900.2:c.30+1511T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="A2ML1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000166535">A2ML1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.322</b>
                    <data>(p=9.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=A2ML1%0DGBA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GBA1</a> and phenotypic similarity 0.613 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2072">Gaucher disease-ophthalmoplegia-cardiovascular calcification syndrome</a> associated with GBA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001822, Hallux valgus</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=A2ML1%0DGBA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GBA</a> and phenotypic similarity 0.249 to mouse mutant of GBA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0004077, abnormal striatum morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/166760">OMIM:166760</a> Otitis media, susceptibility to (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.322</b>
                                    <data>(p=9.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>12-8982318-T-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000299698.7">A2ML1:ENST00000299698.7:c.410-5T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF692" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171163">ZNF692</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.322</b>
                    <data>(p=9.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF692%0DSIRT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIRT1</a> and phenotypic similarity 0.897 to mouse mutant of SIRT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006197, ocular hypotelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000613, abnormal salivary gland morphology</dd><dd>HP:0000520, Proptosis - MP:0006197, ocular hypotelorism</dd><dd>HP:0000601, Hypotelorism - MP:0006197, ocular hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0006197, ocular hypotelorism</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.322</b>
                                    <data>(p=9.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>DEL</b>
                            <b>1-249149660-GA-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1308275555">rs1308275555</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000306601.4">ZNF692:ENST00000306601.4:c.960-4del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366469.5">ZNF692:ENST00000366469.5:c.957-4del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366471.3">ZNF692:ENST00000366471.3:c.825-4del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000427146.1">ZNF692:ENST00000427146.1:c.825-4del:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000451251.1">ZNF692:ENST00000451251.1:c.975-4del:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PKM" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000067225">PKM</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.319</b>
                    <data>(p=9.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.796</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.433 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PKM">PKM</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000520, Proptosis - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000601, Hypotelorism - MP:0003731, abnormal retina outer nuclear layer morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0003731, abnormal retina outer nuclear layer morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKM%0DARNT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARNT</a> and phenotypic similarity 0.752 to mouse mutant of ARNT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000435, shortened head</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000435, shortened head</dd><dd>HP:0000520, Proptosis - MP:0000435, shortened head</dd><dd>HP:0000601, Hypotelorism - MP:0000435, shortened head</dd><dd>HP:0000244, Brachyturricephaly - MP:0000435, shortened head</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.319</b>
                                    <data>(p=9.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.796</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>15-72502009-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs533408759">rs533408759</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.796</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000319622.6">PKM:ENST00000319622.6:c.565+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335181.5">PKM:ENST00000335181.5:c.565+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389093.3">PKM:ENST00000389093.3:c.565+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449901.2">PKM:ENST00000449901.2:c.520+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000565154.1">PKM:ENST00000565154.1:c.565+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000565184.1">PKM:ENST00000565184.1:c.565+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568459.1">PKM:ENST00000568459.1:c.565+5G&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568883.1">PKM:ENST00000568883.1:c.117-823G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRKAG3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115592">PRKAG3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.318</b>
                    <data>(p=9.1E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.798</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRKAG3%0DHERC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HERC1</a> and phenotypic similarity 0.642 to <a href="http://www.omim.org/entry/617011">Macrocephaly, dysmorphic facies, and psychomotor retardation</a> associated with HERC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001166, Arachnodactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000303, Mandibular prognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRKAG3%0DHERC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HERC1</a> and phenotypic similarity 0.491 to mouse mutant of HERC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.318</b>
                                    <data>(p=9.1E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.798</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>2-219693329-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs376723919">rs376723919</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.798</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392098.3">PRKAG3:ENST00000392098.3:c.634-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000439262.2">PRKAG3:ENST00000439262.2:c.559-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529249.1">PRKAG3:ENST00000529249.1:c.634-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545803.1">PRKAG3:ENST00000545803.1:c.82-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0091%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0199%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0118%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CRX" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105392">CRX</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.309</b>
                    <data>(p=9.3E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.548</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.739</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.312 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with CRX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000602, Ophthalmoplegia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000431, Wide nasal bridge</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.548 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CRX">CRX</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004022, abnormal cone electrophysiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000520, Proptosis - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000601, Hypotelorism - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001325, abnormal retina morphology</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CRX%0DBANF1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to BANF1</a> and phenotypic similarity 0.620 to <a href="http://www.omim.org/entry/614008">Nestor-Guillermo progeria syndrome</a> associated with BANF1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0008404, Nail dystrophy</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000308, Microretrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/120970">OMIM:120970</a> Cone-rod retinal dystrophy-2 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/613829">OMIM:613829</a> Leber congenital amaurosis 7 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1872">ORPHA:1872</a> Cone rod dystrophy - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=65">ORPHA:65</a> Leber congenital amaurosis - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">ORPHA:791</a> Retinitis pigmentosa - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.309</b>
                                    <data>(p=9.3E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.548</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.739</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.312 to ORPHA:791 Retinitis pigmentosa</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-48343000-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1183477519">rs1183477519</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2, BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.739</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221996.7">CRX:ENST00000221996.7:c.676G&gt;A:p.(Val226Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539067.1">CRX:ENST00000539067.1:c.676G&gt;A:p.(Val226Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535362.1">CRX:ENST00000535362.1:c.100+5200G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.73910236</dd>
                                    
                                    <dd>REVEL: 0.138</dd>
                                    <dd>MVP: 0.739</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CRYGB" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182187">CRYGB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.296</b>
                    <data>(p=9.6E-2)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.497</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.791</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.497 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CRYGB">CRYGB</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615188">OMIM:615188</a> Cataract 39, multiple types, autosomal dominant - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.296</b>
                                    <data>(p=9.6E-2)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.497</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.791</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-209010521-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201502749">rs201502749</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.791</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260988.4">CRYGB:ENST00000260988.4:c.229C&gt;T:p.(Arg77Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.79307425</dd>
                                    
                                    <dd>REVEL: 0.453</dd>
                                    <dd>MVP: 0.793</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0034%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 0.0232%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SYNE1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131018">SYNE1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.275</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.434</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.504 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=319332">Autosomal recessive myogenic arthrogryposis multiplex congenita</a> associated with SYNE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001181, Adducted thumb</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000343, Long philtrum</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0005469, Flat occiput</dd></dl><dl><dt>Phenotypic similarity 0.434 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SYNE1">SYNE1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0009434, paraparesis</dd><dd>HP:0000486, Strabismus - MP:0001852, conjunctivitis</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001852, conjunctivitis</dd><dd>HP:0000520, Proptosis - MP:0001852, conjunctivitis</dd><dd>HP:0000601, Hypotelorism - MP:0001852, conjunctivitis</dd><dd>HP:0000244, Brachyturricephaly - MP:0001852, conjunctivitis</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/610743">OMIM:610743</a> Spinocerebellar ataxia, autosomal recessive 8 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/612998">OMIM:612998</a> Emery-Dreifuss muscular dystrophy 4, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/618484">OMIM:618484</a> Arthrogryposis multiplex congenita 3, myogenic type - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=319332">ORPHA:319332</a> Autosomal recessive myogenic arthrogryposis multiplex congenita - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=88644">ORPHA:88644</a> Autosomal recessive ataxia, Beauce type - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98853">ORPHA:98853</a> Autosomal dominant Emery-Dreifuss muscular dystrophy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.275</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.434</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>6-152751786-GCTT-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265368.4">SYNE1:ENST00000265368.4:c.4517_4519del:p.(Glu1506del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341594.5">SYNE1:ENST00000341594.5:c.4727_4729del:p.(Glu1576del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367253.4">SYNE1:ENST00000367253.4:c.4517_4519del:p.(Glu1506del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367255.5">SYNE1:ENST00000367255.5:c.4517_4519del:p.(Glu1506del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423061.1">SYNE1:ENST00000423061.1:c.4538_4540del:p.(Glu1513del)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448038.1">SYNE1:ENST00000448038.1:c.4538_4540del:p.(Glu1513del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ECM2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106823">ECM2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.268</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.771</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ECM2%0DPRKAR1B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRKAR1B</a> and phenotypic similarity 0.703 to <a href="http://www.omim.org/entry/619680">Marbach-Schaaf neurodevelopmental syndrome</a> associated with PRKAR1B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000176, Submucous cleft hard palate</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001357, Plagiocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.268</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.771</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-95263039-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs776930712">rs776930712</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.771</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344604.5">ECM2:ENST00000344604.5:c.1901A&gt;G:p.(His634Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000444490.2">ECM2:ENST00000444490.2:c.1835A&gt;G:p.(His612Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375587.3">ECM2:ENST00000375587.3:c.565-110557T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7711705</dd>
                                    
                                    <dd>REVEL: 0.100</dd>
                                    <dd>MVP: 0.771</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBC1D4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136111">TBC1D4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.257</b>
                    <data>(p=1.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.764</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.481 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TBC1D4">TBC1D4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBC1D4%0DSFN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SFN</a> and phenotypic similarity 0.710 to mouse mutant of SFN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009553, fused lips</dd><dd>HP:0000520, Proptosis - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001340, abnormal eyelid morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.257</b>
                                    <data>(p=1.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.764</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-75900394-C-T [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.764</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377625.2">TBC1D4:ENST00000377625.2:c.1972G&gt;A:p.(Asp658Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377636.3">TBC1D4:ENST00000377636.3:c.1972G&gt;A:p.(Asp658Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431480.2">TBC1D4:ENST00000431480.2:c.1972G&gt;A:p.(Asp658Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000425511.1">TBC1D4:ENST00000425511.1:c.-378G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7635459</dd>
                                    
                                    <dd>REVEL: 0.048</dd>
                                    <dd>MVP: 0.764</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ROR2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000169071">ROR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.252</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.772</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.454</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.631 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1507">Autosomal recessive Robinow syndrome</a> associated with ROR2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Phenotypic similarity 0.772 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ROR2">ROR2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000562, polydactyly</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0000441, increased cranium width</dd></dl><dl><dt>Phenotypic similarity 0.310 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=ROR2">ROR2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ROR2%0DFZD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FZD2</a> and phenotypic similarity 0.630 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3107">Autosomal dominant Robinow syndrome</a> associated with FZD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0006101, Finger syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ROR2%0DFZD2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FZD2</a> and phenotypic similarity 0.374 to mouse mutant of FZD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000111, cleft palate</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000111, cleft palate</dd><dd>HP:0000520, Proptosis - MP:0000111, cleft palate</dd><dd>HP:0000601, Hypotelorism - MP:0000111, cleft palate</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/113000">OMIM:113000</a> Brachydactyly, type B1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/268310">OMIM:268310</a> Robinow syndrome, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1507">ORPHA:1507</a> Autosomal recessive Robinow syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.252</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.772</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.454</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.631 to ORPHA:1507 Autosomal recessive Robinow syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.591 to OMIM:268310 Robinow syndrome, autosomal recessive</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-94486381-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141235720">rs141235720</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=196255%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.816</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375708.3">ROR2:ENST00000375708.3:c.2395C&gt;T:p.(Pro799Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375715.1">ROR2:ENST00000375715.1:c.1920+55C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.88252985</dd>
                                    
                                    <dd>REVEL: 0.288</dd>
                                    <dd>MVP: 0.883</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.1791%</dd>
                                    <dd>UK10K: 0.1322%</dd>
                                    <dd>ESP AA: 0.0908%</dd>
                                    <dd>ESP EA: 0.3372%</dd>
                                    <dd>ESP All: 0.2537%</dd>
                                    <dd>ExAC AFR: 0.0493%</dd>
                                    <dd>ExAC AMR: 0.1125%</dd>
                                    <dd>ExAC FIN: 0.0606%</dd>
                                    <dd>ExAC NFE: 0.2518%</dd>
                                    <dd>ExAC OTH: 0.4425%</dd>
                                    <dd>ExAC SAS: 0.3706%</dd>
                                    <dd>gnomAD_E_AFR: 0.0459%</dd>
                                    <dd>gnomAD_E_AMR: 0.1013%</dd>
                                    <dd>gnomAD_E_FIN: 0.0648%</dd>
                                    <dd>gnomAD_E_NFE: 0.2843%</dd>
                                    <dd>gnomAD_E_OTH: 0.1832%</dd>
                                    <dd>gnomAD_E_SAS: 0.4192%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.3271%</dd>
                                    <dd>gnomAD_G_OTH: 0.2037%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-94519645-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145568368">rs145568368</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=193554%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.092</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375708.3">ROR2:ENST00000375708.3:c.372C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375715.1">ROR2:ENST00000375715.1:c.-49C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.2535%</dd>
                                    <dd>UK10K: 0.2248%</dd>
                                    <dd>ESP AA: 0.1362%</dd>
                                    <dd>ESP EA: 0.4651%</dd>
                                    <dd>ESP All: 0.3537%</dd>
                                    <dd>ExAC AFR: 0.1057%</dd>
                                    <dd>ExAC AMR: 0.1814%</dd>
                                    <dd>ExAC FIN: 0.0605%</dd>
                                    <dd>ExAC NFE: 0.3342%</dd>
                                    <dd>ExAC OTH: 0.3304%</dd>
                                    <dd>ExAC SAS: 0.3997%</dd>
                                    <dd>gnomAD_E_AFR: 0.0784%</dd>
                                    <dd>gnomAD_E_AMR: 0.1370%</dd>
                                    <dd>gnomAD_E_FIN: 0.0448%</dd>
                                    <dd>gnomAD_E_NFE: 0.3876%</dd>
                                    <dd>gnomAD_E_OTH: 0.3463%</dd>
                                    <dd>gnomAD_E_SAS: 0.4451%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.3002%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FCGBP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000275395">FCGBP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.226</b>
                    <data>(p=1.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.739</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.507 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FCGBP">FCGBP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002092, abnormal eye morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002092, abnormal eye morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002092, abnormal eye morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.226</b>
                                    <data>(p=1.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.739</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-40421270-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs751583225">rs751583225</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.739</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221347.6">FCGBP:ENST00000221347.6:c.2651G&gt;A:p.(Gly884Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.74</dd>
                                    
                                    <dd>REVEL: 0.740</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0101%</dd>
                                    <dd>gnomAD_E_AMR: 0.0120%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CYP2E1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000130649">CYP2E1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.200</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.524</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.703</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.524 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CYP2E1">CYP2E1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000520, Proptosis - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002100, abnormal tooth morphology</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP2E1%0DPOR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POR</a> and phenotypic similarity 0.664 to <a href="http://www.omim.org/entry/201750">Antley-Bixler syndrome with genital anomalies and disordered steroidogenesis</a> associated with POR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001166, Arachnodactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000272, Malar flattening</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.517 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CYP2E1%0DPOR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POR</a> and phenotypic similarity 0.499 to mouse mutant of POR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0012531, delayed limb development</dd><dd>HP:0000486, Strabismus - MP:0002092, abnormal eye morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002092, abnormal eye morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002092, abnormal eye morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.200</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.524</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.703</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-135342118-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs543855005">rs543855005</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2384005%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.703</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000252945.3">CYP2E1:ENST00000252945.3:c.311C&gt;T:p.(Pro104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000463117.2">CYP2E1:ENST00000463117.2:c.311C&gt;T:p.(Pro104Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599428.1">CYP2E1:ENST00000599428.1:c.19+525G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000541506.1">CYP2E1:ENST00000541506.1:c.-17+40658G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.70660913</dd>
                                    
                                    <dd>REVEL: 0.094</dd>
                                    <dd>MVP: 0.707</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0247%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC FIN: 0.0305%</dd>
                                    <dd>ExAC NFE: 0.0261%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0343%</dd>
                                    <dd>gnomAD_E_OTH: 0.0366%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FKBP4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000004478">FKBP4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.196</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.726</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FKBP4%0DSIRT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SIRT1</a> and phenotypic similarity 0.897 to mouse mutant of SIRT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006197, ocular hypotelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000613, abnormal salivary gland morphology</dd><dd>HP:0000520, Proptosis - MP:0006197, ocular hypotelorism</dd><dd>HP:0000601, Hypotelorism - MP:0006197, ocular hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0006197, ocular hypotelorism</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.196</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.726</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-2907914-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs191293085">rs191293085</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.726</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000001008.4">FKBP4:ENST00000001008.4:c.436G&gt;A:p.(Glu146Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.732964</dd>
                                    
                                    <dd>REVEL: 0.211</dd>
                                    <dd>MVP: 0.733</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0094%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AMR: 0.0605%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0655%</dd>
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NPHS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000161270">NPHS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.185</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.703</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPHS1%0DDISP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DISP1</a> and phenotypic similarity 0.688 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=280200">Microform holoprosencephaly</a> associated with DISP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0006315, Solitary median maxillary central incisor</dd><dd>HP:0000520, Proptosis - HP:0009914, Cyclopia</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPHS1%0DDISP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DISP1</a> and phenotypic similarity 0.872 to mouse mutant of DISP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006197, ocular hypotelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030314, absent primary palate</dd><dd>HP:0000520, Proptosis - MP:0006197, ocular hypotelorism</dd><dd>HP:0000601, Hypotelorism - MP:0006197, ocular hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0002659, pituitary gland hypoplasia</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NPHS1%0DDISP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DISP1</a> and phenotypic similarity 0.464 to fish mutant of DISP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - ZP:0000620, eye mislocalised medially, abnormal</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0000235, mandibular arch skeleton hypoplastic, abnormal</dd><dd>HP:0000520, Proptosis - ZP:0000620, eye mislocalised medially, abnormal</dd><dd>HP:0000601, Hypotelorism - ZP:0000620, eye mislocalised medially, abnormal</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/256300">OMIM:256300</a> Nephrotic syndrome, type 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=656">ORPHA:656</a> Genetic steroid-resistant nephrotic syndrome - autosomal dominant/recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=839">ORPHA:839</a> Congenital nephrotic syndrome, Finnish type - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.185</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.515</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.703</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-36342551-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs768721706">rs768721706</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2205946%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.703</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000353632.6">NPHS1:ENST00000353632.6:c.82G&gt;C:p.(Ala28Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378910.5">NPHS1:ENST00000378910.5:c.82G&gt;C:p.(Ala28Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.70311683</dd>
                                    
                                    <dd>REVEL: 0.166</dd>
                                    <dd>MVP: 0.703</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>ExAC NFE: 0.0035%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SCN7A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136546">SCN7A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.184</b>
                    <data>(p=1.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.717</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN7A%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.717 to <a href="http://www.omim.org/entry/617798">Intellectual developmental disorder, autosomal dominant 53</a> associated with CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0003097, Short femur</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SCN7A%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.251 to mouse mutant of CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000812, abnormal dentate gyrus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.184</b>
                                    <data>(p=1.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.717</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-167273315-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs774846956">rs774846956</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.717</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409855.1">SCN7A:ENST00000409855.1:c.3316C&gt;T:p.(Arg1106Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.71923494</dd>
                                    
                                    <dd>REVEL: 0.339</dd>
                                    <dd>MVP: 0.719</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0016%</dd>
                                    <dd>ExAC AMR: 0.0181%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AMR: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EIF2B2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000119718">EIF2B2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.176</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.995</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.293 to <a href="http://www.omim.org/entry/603896">Leukoencephalopathy with vanishing white matter</a> associated with EIF2B2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EIF2B2%0DRANBP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RANBP1</a> and phenotypic similarity 0.737 to mouse mutant of RANBP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000520, Proptosis - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000601, Hypotelorism - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/603896">OMIM:603896</a> Leukoencephalopathy with vanishing white matter - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.176</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.995</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-75472653-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs748685143">rs748685143</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1874725%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.995</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000266126.5">EIF2B2:ENST00000266126.5:c.682A&gt;G:p.(Arg228Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9962535</dd>
                                    
                                    <dd>REVEL: 0.907</dd>
                                    <dd>MVP: 0.996</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PGF" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000119630">PGF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.161</b>
                    <data>(p=1.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.538</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.658</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.395 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PGF">PGF</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005542, corneal vascularization</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005542, corneal vascularization</dd><dd>HP:0000520, Proptosis - MP:0005542, corneal vascularization</dd><dd>HP:0000601, Hypotelorism - MP:0005542, corneal vascularization</dd><dd>HP:0000244, Brachyturricephaly - MP:0005542, corneal vascularization</dd></dl><dl><dt>Proximity score 0.538 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PGF%0DSLC12A6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC12A6</a> and phenotypic similarity 0.771 to <a href="http://www.omim.org/entry/218000">Agenesis of the corpus callosum with peripheral neuropathy</a> associated with SLC12A6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004691, 2-3 toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.538 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PGF%0DSLC12A6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC12A6</a> and phenotypic similarity 0.421 to mouse mutant of SLC12A6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004268, abnormal optic stalk morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004268, abnormal optic stalk morphology</dd><dd>HP:0000520, Proptosis - MP:0004268, abnormal optic stalk morphology</dd><dd>HP:0000601, Hypotelorism - MP:0004268, abnormal optic stalk morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0004268, abnormal optic stalk morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.161</b>
                                    <data>(p=1.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.538</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.658</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-75416073-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs760252361">rs760252361</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.658</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000238607.6">PGF:ENST00000238607.6:c.299A&gt;G:p.(Asn100Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405431.2">PGF:ENST00000405431.2:c.302A&gt;G:p.(Asn101Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553716.1">PGF:ENST00000553716.1:c.302A&gt;G:p.(Asn101Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555567.1">PGF:ENST00000555567.1:c.302A&gt;G:p.(Asn101Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.65972406</dd>
                                    
                                    <dd>REVEL: 0.430</dd>
                                    <dd>MVP: 0.660</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 0.0117%</dd>
                                    <dd>gnomAD_E_EAS: 0.0130%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_E_OTH: 0.0209%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GMPR2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100938">GMPR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.127</b>
                    <data>(p=1.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.668</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GMPR2%0DPDE6D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDE6D</a> and phenotypic similarity 0.628 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">Orofaciodigital syndrome type 6</a> associated with PDE6D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001159, Syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0004422, Biparietal narrowing</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GMPR2%0DPDE6D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDE6D</a> and phenotypic similarity 0.548 to mouse mutant of PDE6D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004022, abnormal cone electrophysiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000520, Proptosis - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000601, Hypotelorism - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001326, retina degeneration</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.127</b>
                                    <data>(p=1.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.668</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-24705004-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45530636">rs45530636</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2277296%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.668</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000348719.7">GMPR2:ENST00000348719.7:c.269G&gt;T:p.(Gly90Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355299.4">GMPR2:ENST00000355299.4:c.269G&gt;T:p.(Gly90Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399440.2">GMPR2:ENST00000399440.2:c.269G&gt;T:p.(Gly90Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420554.2">GMPR2:ENST00000420554.2:c.323G&gt;T:p.(Gly108Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557854.1">GMPR2:ENST00000557854.1:c.323G&gt;T:p.(Gly108Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559104.1">GMPR2:ENST00000559104.1:c.323G&gt;T:p.(Gly108Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559836.1">GMPR2:ENST00000559836.1:c.269G&gt;T:p.(Gly90Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559910.1">GMPR2:ENST00000559910.1:c.269G&gt;T:p.(Gly90Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456667.3">GMPR2:ENST00000456667.3:c.208-169G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6758346</dd>
                                    
                                    <dd>REVEL: 0.261</dd>
                                    <dd>MVP: 0.676</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0453%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP EA: 0.0843%</dd>
                                    <dd>ESP All: 0.0573%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0420%</dd>
                                    <dd>ExAC SAS: 0.0666%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0582%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_E_SAS: 0.0552%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                    <dd>gnomAD_G_NFE: 0.0666%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CACNA2D3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157445">CACNA2D3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.122</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.665</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA2D3%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.717 to <a href="http://www.omim.org/entry/617798">Intellectual developmental disorder, autosomal dominant 53</a> associated with CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0003097, Short femur</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CACNA2D3%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.251 to mouse mutant of CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000812, abnormal dentate gyrus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.122</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.665</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-54933874-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs771363813">rs771363813</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.665</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000288197.5">CACNA2D3:ENST00000288197.5:c.2428C&gt;T:p.(Arg810Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415676.2">CACNA2D3:ENST00000415676.2:c.2428C&gt;T:p.(Arg810Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000474759.1">CACNA2D3:ENST00000474759.1:c.2428C&gt;T:p.(Arg810Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000490478.1">CACNA2D3:ENST00000490478.1:c.2146C&gt;T:p.(Arg716Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6682881</dd>
                                    
                                    <dd>REVEL: 0.459</dd>
                                    <dd>MVP: 0.668</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC EAS: 0.0229%</dd>
                                    <dd>ExAC NFE: 0.0142%</dd>
                                    <dd>gnomAD_E_AMR: 0.0093%</dd>
                                    <dd>gnomAD_E_EAS: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0077%</dd>
                                    <dd>gnomAD_E_OTH: 0.0384%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TAX1BP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000106052">TAX1BP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.121</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.665</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAX1BP1%0DSTAMBP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STAMBP</a> and phenotypic similarity 0.693 to <a href="http://www.omim.org/entry/614261">Microcephaly-capillary malformation syndrome</a> associated with STAMBP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000253, Progressive microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TAX1BP1%0DSTAMBP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STAMBP</a> and phenotypic similarity 0.508 to mouse mutant of STAMBP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001344, blepharoptosis</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001344, blepharoptosis</dd><dd>HP:0000520, Proptosis - MP:0001344, blepharoptosis</dd><dd>HP:0000601, Hypotelorism - MP:0001344, blepharoptosis</dd><dd>HP:0000244, Brachyturricephaly - MP:0000788, abnormal cerebral cortex morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.121</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.665</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-27856536-A-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.665</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000265393.6">TAX1BP1:ENST00000265393.6:c.1838A&gt;G:p.(Asp613Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396319.2">TAX1BP1:ENST00000396319.2:c.1964A&gt;G:p.(Asp655Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409980.1">TAX1BP1:ENST00000409980.1:c.2036A&gt;G:p.(Asp679Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433216.2">TAX1BP1:ENST00000433216.2:c.1367A&gt;G:p.(Asp456Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543117.1">TAX1BP1:ENST00000543117.1:c.1838A&gt;G:p.(Asp613Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.66470313</dd>
                                    
                                    <dd>REVEL: 0.214</dd>
                                    <dd>MVP: 0.665</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR7D2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188000">OR7D2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.119</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.663</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR7D2%0DSTAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STAG2</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93925">Alobar holoprosencephaly</a> associated with STAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0006315, Solitary median maxillary central incisor</dd><dd>HP:0000520, Proptosis - HP:0009914, Cyclopia</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.119</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.663</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-9296702-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1222262420">rs1222262420</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.663</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344248.2">OR7D2:ENST00000344248.2:c.245T&gt;C:p.(Leu82Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6633658</dd>
                                    
                                    <dd>REVEL: 0.245</dd>
                                    <dd>MVP: 0.663</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PTPRD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153707">PTPRD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.111</b>
                    <data>(p=1.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.654</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPRD%0DGRIP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRIP1</a> and phenotypic similarity 0.541 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2052">Fraser syndrome</a> associated with GRIP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000678, Dental crowding</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001362, Calvarial skull defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PTPRD%0DGRIP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GRIP1</a> and phenotypic similarity 0.765 to mouse mutant of GRIP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus - MP:0001293, anophthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001293, anophthalmia</dd><dd>HP:0000520, Proptosis - MP:0001293, anophthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001293, anophthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0001293, anophthalmia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.111</b>
                                    <data>(p=1.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.654</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>9-8486355-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs183119115">rs183119115</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.703</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356435.5">PTPRD:ENST00000356435.5:c.2468-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358503.5">PTPRD:ENST00000358503.5:c.2402-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360074.4">PTPRD:ENST00000360074.4:c.2429-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381196.4">PTPRD:ENST00000381196.4:c.2468-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540109.1">PTPRD:ENST00000540109.1:c.2468-6C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355233.5">PTPRD:ENST00000355233.5:c.1823-1031C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397606.3">PTPRD:ENST00000397606.3:c.1793-1031C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397611.3">PTPRD:ENST00000397611.3:c.1814-1031C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397617.3">PTPRD:ENST00000397617.3:c.1793-1031C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486161.1">PTPRD:ENST00000486161.1:c.1823-1031C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537002.1">PTPRD:ENST00000537002.1:c.1814-1031C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.2157%</dd>
                                    <dd>UK10K: 0.2777%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP EA: 0.3140%</dd>
                                    <dd>ESP All: 0.2230%</dd>
                                    <dd>ExAC AFR: 0.0685%</dd>
                                    <dd>ExAC AMR: 0.0433%</dd>
                                    <dd>ExAC FIN: 0.1663%</dd>
                                    <dd>ExAC NFE: 0.3332%</dd>
                                    <dd>ExAC OTH: 0.1119%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0657%</dd>
                                    <dd>gnomAD_E_AMR: 0.1073%</dd>
                                    <dd>gnomAD_E_FIN: 0.1708%</dd>
                                    <dd>gnomAD_E_NFE: 0.3751%</dd>
                                    <dd>gnomAD_E_OTH: 0.1835%</dd>
                                    <dd>gnomAD_G_AFR: 0.1031%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_FIN: 0.2290%</dd>
                                    <dd>gnomAD_G_NFE: 0.6395%</dd>
                                    <dd>gnomAD_G_OTH: 0.3061%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-8465666-C-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.605</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355233.5">PTPRD:ENST00000355233.5:c.2281G&gt;C:p.(Glu761Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356435.5">PTPRD:ENST00000356435.5:c.3514G&gt;C:p.(Glu1172Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358503.5">PTPRD:ENST00000358503.5:c.3448G&gt;C:p.(Glu1150Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360074.4">PTPRD:ENST00000360074.4:c.3475G&gt;C:p.(Glu1159Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381196.4">PTPRD:ENST00000381196.4:c.3514G&gt;C:p.(Glu1172Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397606.3">PTPRD:ENST00000397606.3:c.2251G&gt;C:p.(Glu751Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397611.3">PTPRD:ENST00000397611.3:c.2272G&gt;C:p.(Glu758Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397617.3">PTPRD:ENST00000397617.3:c.2251G&gt;C:p.(Glu751Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486161.1">PTPRD:ENST00000486161.1:c.2281G&gt;C:p.(Glu761Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537002.1">PTPRD:ENST00000537002.1:c.2272G&gt;C:p.(Glu758Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540109.1">PTPRD:ENST00000540109.1:c.3514G&gt;C:p.(Glu1172Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6048526</dd>
                                    
                                    <dd>REVEL: 0.156</dd>
                                    <dd>MVP: 0.605</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.073</b>
                                    <data>(p=1.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.605</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-8465666-C-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.605</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355233.5">PTPRD:ENST00000355233.5:c.2281G&gt;C:p.(Glu761Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356435.5">PTPRD:ENST00000356435.5:c.3514G&gt;C:p.(Glu1172Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358503.5">PTPRD:ENST00000358503.5:c.3448G&gt;C:p.(Glu1150Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360074.4">PTPRD:ENST00000360074.4:c.3475G&gt;C:p.(Glu1159Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381196.4">PTPRD:ENST00000381196.4:c.3514G&gt;C:p.(Glu1172Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397606.3">PTPRD:ENST00000397606.3:c.2251G&gt;C:p.(Glu751Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397611.3">PTPRD:ENST00000397611.3:c.2272G&gt;C:p.(Glu758Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000397617.3">PTPRD:ENST00000397617.3:c.2251G&gt;C:p.(Glu751Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000486161.1">PTPRD:ENST00000486161.1:c.2281G&gt;C:p.(Glu761Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537002.1">PTPRD:ENST00000537002.1:c.2272G&gt;C:p.(Glu758Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540109.1">PTPRD:ENST00000540109.1:c.3514G&gt;C:p.(Glu1172Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6048526</dd>
                                    
                                    <dd>REVEL: 0.156</dd>
                                    <dd>MVP: 0.605</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NEK2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000117650">NEK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.101</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.925</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.312 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">Retinitis pigmentosa</a> associated with NEK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000602, Ophthalmoplegia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000431, Wide nasal bridge</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEK2%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/entry/614114">Mosaic variegated aneuploidy syndrome 2</a> associated with CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004209, Clinodactyly of the 5th finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEK2%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.314 to mouse mutant of CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003051, curly tail</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004610, small vertebrae</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615565">OMIM:615565</a> ?Retinitis pigmentosa 67 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=791">ORPHA:791</a> Retinitis pigmentosa - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.101</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.925</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-211836799-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202099146">rs202099146</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1022914%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.925</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366999.4">NEK2:ENST00000366999.4:c.1307T&gt;C:p.(Leu436Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540251.1">NEK2:ENST00000540251.1:c.1178T&gt;C:p.(Leu393Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.9332452</dd>
                                    
                                    <dd>REVEL: 0.467</dd>
                                    <dd>MVP: 0.933</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0091%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC NFE: 0.0161%</dd>
                                    <dd>gnomAD_E_NFE: 0.0180%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_NFE: 0.0666%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DCDC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170959">DCDC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.097</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.638</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DCDC1%0DABCA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ABCA5</a> and phenotypic similarity 0.893 to mouse mutant of ABCA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002750, exophthalmos</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0002750, exophthalmos</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.097</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.638</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-31128494-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs772264363">rs772264363</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.638</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597505.1">DCDC1:ENST00000597505.1:c.1601T&gt;C:p.(Ile534Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6395986</dd>
                                    
                                    <dd>MVP: 0.640</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC AMR: 0.0092%</dd>
                                    <dd>gnomAD_E_AMR: 0.0151%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AAMDC" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000087884">AAMDC</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.096</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.637</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AAMDC%0DCOL11A2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL11A2</a> and phenotypic similarity 0.633 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1427">Otospondylomegaepiphyseal dysplasia</a> associated with COL11A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AAMDC%0DCOL11A2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL11A2</a> and phenotypic similarity 0.546 to mouse mutant of COL11A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0006397, disorganized long bone epiphyseal plate</dd><dd>HP:0000486, Strabismus - MP:0000102, abnormal nasal bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000445, short snout</dd><dd>HP:0000520, Proptosis - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0030049, prominent forehead</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AAMDC%0DCOL11A2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COL11A2</a> and phenotypic similarity 0.463 to fish mutant of COL11A2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - ZP:0000121, eye mislocalised, abnormal</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0007203, ventral mandibular arch morphology, abnormal</dd><dd>HP:0000520, Proptosis - ZP:0000121, eye mislocalised, abnormal</dd><dd>HP:0000601, Hypotelorism - ZP:0000121, eye mislocalised, abnormal</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.096</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.637</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-77553595-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138381967">rs138381967</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.637</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304716.8">AAMDC:ENST00000304716.8:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393427.2">AAMDC:ENST00000393427.2:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525034.1">AAMDC:ENST00000525034.1:c.110G&gt;C:p.(Gly37Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525409.1">AAMDC:ENST00000525409.1:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526415.1">AAMDC:ENST00000526415.1:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527134.1">AAMDC:ENST00000527134.1:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000532481.1">AAMDC:ENST00000532481.1:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000533193.1">AAMDC:ENST00000533193.1:c.53G&gt;C:p.(Gly18Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6421952</dd>
                                    
                                    <dd>REVEL: 0.461</dd>
                                    <dd>MVP: 0.642</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0462%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0466%</dd>
                                    <dd>ESP All: 0.0385%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0390%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0447%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0547%</dd>
                                    <dd>gnomAD_E_OTH: 0.0548%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PFKP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000067057">PFKP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.095</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.634</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PFKP%0DZBTB20&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZBTB20</a> and phenotypic similarity 0.728 to <a href="http://www.omim.org/entry/259050">Primrose syndrome</a> associated with ZBTB20.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0008391, Dystrophic fingernails</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PFKP%0DZBTB20&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZBTB20</a> and phenotypic similarity 0.483 to mouse mutant of ZBTB20.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006203, eye hemorrhage</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0006203, eye hemorrhage</dd><dd>HP:0000520, Proptosis - MP:0006203, eye hemorrhage</dd><dd>HP:0000601, Hypotelorism - MP:0006203, eye hemorrhage</dd><dd>HP:0000244, Brachyturricephaly - MP:0000812, abnormal dentate gyrus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.095</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.634</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-3141542-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs899053868">rs899053868</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.634</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381075.2">PFKP:ENST00000381075.2:c.158G&gt;A:p.(Ser53Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381125.4">PFKP:ENST00000381125.4:c.262G&gt;A:p.(Val88Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.63412327</dd>
                                    
                                    <dd>REVEL: 0.180</dd>
                                    <dd>MVP: 0.634</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0019%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AQP6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000086159">AQP6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.089</b>
                    <data>(p=1.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.509</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.616</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.439 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=AQP6">AQP6</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0011964, increased total retina thickness</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0011964, increased total retina thickness</dd><dd>HP:0000520, Proptosis - MP:0011964, increased total retina thickness</dd><dd>HP:0000601, Hypotelorism - MP:0011964, increased total retina thickness</dd><dd>HP:0000244, Brachyturricephaly - MP:0011964, increased total retina thickness</dd></dl><dl><dt>Proximity score 0.509 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AQP6%0DSHPK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SHPK</a> and phenotypic similarity 0.684 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=440713">Isolated sedoheptulokinase deficiency</a> associated with SHPK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000586, Shallow orbits</dd><dd>HP:0000520, Proptosis - HP:0000586, Shallow orbits</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.089</b>
                                    <data>(p=1.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.509</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.616</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-50366997-T-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144813201">rs144813201</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.616</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315520.5">AQP6:ENST00000315520.5:c.41T&gt;G:p.(Met14Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000551733.1">AQP6:ENST00000551733.1:c.-121+679T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6197977</dd>
                                    
                                    <dd>REVEL: 0.206</dd>
                                    <dd>MVP: 0.620</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0147%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0386%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC NFE: 0.0152%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0243%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TOP2A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000131747">TOP2A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.085</b>
                    <data>(p=1.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.508</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.614</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.508 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TOP2A">TOP2A</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=635">ORPHA:635</a> Neuroblastoma (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.085</b>
                                    <data>(p=1.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.614</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-38560731-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs747054164">rs747054164</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.614</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423485.1">TOP2A:ENST00000423485.1:c.2059G&gt;A:p.(Gly687Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6167126</dd>
                                    
                                    <dd>REVEL: 0.410</dd>
                                    <dd>MVP: 0.617</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0049%</dd>
                                    <dd>ExAC AMR: 0.0261%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0303%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAP3K6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000142733">MAP3K6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.068</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.596</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAP3K6%0DLETM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LETM1</a> and phenotypic similarity 0.646 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=280">Wolf-Hirschhorn syndrome</a> associated with LETM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001166, Arachnodactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001362, Calvarial skull defect</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAP3K6%0DLETM1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LETM1</a> and phenotypic similarity 0.198 to mouse mutant of LETM1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0002152, abnormal brain morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.068</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.596</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-27686399-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs905799214">rs905799214</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.596</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357582.2">MAP3K6:ENST00000357582.2:c.2269C&gt;A:p.(Gln757Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374040.3">MAP3K6:ENST00000374040.3:c.2245C&gt;A:p.(Gln749Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000493901.1">MAP3K6:ENST00000493901.1:c.2269C&gt;A:p.(Gln757Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5958713</dd>
                                    
                                    <dd>REVEL: 0.223</dd>
                                    <dd>MVP: 0.596</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EHMT1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181090">EHMT1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.057</b>
                    <data>(p=1.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.634</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.424</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.634 to <a href="http://www.omim.org/entry/610253">Kleefstra syndrome 1</a> associated with EHMT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000303, Mandibular prognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Phenotypic similarity 0.566 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EHMT1">EHMT1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006243, impaired pupillary reflex</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001314, corneal opacity</dd><dd>HP:0000520, Proptosis - MP:0001314, corneal opacity</dd><dd>HP:0000601, Hypotelorism - MP:0001314, corneal opacity</dd><dd>HP:0000244, Brachyturricephaly - MP:0008259, abnormal optic disk morphology</dd></dl><dl><dt>Proximity score 0.576 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EHMT1%0DKDM5C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KDM5C</a> and phenotypic similarity 0.706 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85279">KDM5C-related syndromic X-linked intellectual disability</a> associated with KDM5C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/610253">OMIM:610253</a> Kleefstra syndrome 1 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=261652">ORPHA:261652</a> Kleefstra syndrome due to a point mutation - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=96147">ORPHA:96147</a> Kleefstra syndrome due to 9q34 microdeletion (CNV)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.057</b>
                                    <data>(p=1.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.634</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.424</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.634 to OMIM:610253 Kleefstra syndrome 1</dt>
                                
                                    <dt>Phenotypic similarity 0.610 to ORPHA:96147 Kleefstra syndrome due to 9q34 microdeletion</dt>
                                
                                    <dt>Phenotypic similarity 0.560 to ORPHA:261652 Kleefstra syndrome due to a point mutation</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-140611555-A-G [0/1]</b>
                            
                            
                                
                                    
                                    
                                    
                                    <span>2003930</span>
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2, PP4, BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2051987%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.424</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334856.6">EHMT1:ENST00000334856.6:c.470A&gt;G:p.(Asp157Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000460843.1">EHMT1:ENST00000460843.1:c.563A&gt;G:p.(Asp188Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000462484.1">EHMT1:ENST00000462484.1:c.563A&gt;G:p.(Asp188Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.42446315</dd>
                                    
                                    <dd>REVEL: 0.042</dd>
                                    <dd>MVP: 0.424</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ATXN3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000066427">ATXN3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.054</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.614 to <a href="http://www.omim.org/entry/109150">Machado-Joseph disease</a> associated with ATXN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000641, Dysmetric saccades</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.202 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ATXN3">ATXN3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000877, abnormal Purkinje cell morphology</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATXN3%0DUBE4B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE4B</a> and phenotypic similarity 0.615 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with UBE4B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000307, Pointed chin</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ATXN3%0DUBE4B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE4B</a> and phenotypic similarity 0.217 to mouse mutant of UBE4B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0001917, intraventricular hemorrhage</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/109150">OMIM:109150</a> Machado-Joseph disease - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.054</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_INSERTION</span>
                            <b>INS</b>
                            <b>14-92537354-C-CCTGCTGCTGCTG [1/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs193922928">rs193922928</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=919552%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340660.6">ATXN3:ENST00000340660.6:c.739_750dup:p.(Gln247_Gln250dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393287.5">ATXN3:ENST00000393287.5:c.904_915dup:p.(Gln302_Gln305dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429774.2">ATXN3:ENST00000429774.2:c.883_894dup:p.(Gln295_Gln298dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000502250.1">ATXN3:ENST00000502250.1:c.367_378dup:p.(Gln123_Gln126dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503767.1">ATXN3:ENST00000503767.1:c.859_870dup:p.(Gln287_Gln290dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000532032.1">ATXN3:ENST00000532032.1:c.904_915dup:p.(Gln302_Gln305dup)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545170.1">ATXN3:ENST00000545170.1:c.931_942dup:p.(Gln311_Gln314dup)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF76" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000065029">ZNF76</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.047</b>
                    <data>(p=1.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.510</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.542</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF76%0DDSE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DSE</a> and phenotypic similarity 0.615 to <a href="http://www.omim.org/entry/615539">Ehlers-Danlos syndrome, musculocontractural type 2</a> associated with DSE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001181, Adducted thumb</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000678, Dental crowding</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.510 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF76%0DDSE&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DSE</a> and phenotypic similarity 0.192 to mouse mutant of DSE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000585, kinked tail</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.047</b>
                                    <data>(p=1.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.542</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-35259091-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs759006880">rs759006880</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.542</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339411.5">ZNF76:ENST00000339411.5:c.662G&gt;T:p.(Gly221Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373953.3">ZNF76:ENST00000373953.3:c.662G&gt;T:p.(Gly221Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440666.2">ZNF76:ENST00000440666.2:c.584G&gt;T:p.(Gly195Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5445636</dd>
                                    
                                    <dd>REVEL: 0.459</dd>
                                    <dd>MVP: 0.545</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>ExAC NFE: 0.0052%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0366%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.012</b>
                                    <data>(p=2.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.510</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.388</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-35259091-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs759006880">rs759006880</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.542</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339411.5">ZNF76:ENST00000339411.5:c.662G&gt;T:p.(Gly221Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373953.3">ZNF76:ENST00000373953.3:c.662G&gt;T:p.(Gly221Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440666.2">ZNF76:ENST00000440666.2:c.584G&gt;T:p.(Gly195Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5445636</dd>
                                    
                                    <dd>REVEL: 0.459</dd>
                                    <dd>MVP: 0.545</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>ExAC NFE: 0.0052%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0366%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-35258481-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765023943">rs765023943</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.234</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339411.5">ZNF76:ENST00000339411.5:c.613G&gt;T:p.(Ala205Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000373953.3">ZNF76:ENST00000373953.3:c.613G&gt;T:p.(Ala205Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440666.2">ZNF76:ENST00000440666.2:c.535G&gt;T:p.(Ala179Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.23378578</dd>
                                    
                                    <dd>REVEL: 0.084</dd>
                                    <dd>MVP: 0.234</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SETX" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107290">SETX</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.045</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.439</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.618</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.461 to <a href="http://www.omim.org/entry/606002">Spinocerebellar ataxia, autosomal recessive, with axonal neuropathy 2</a> associated with SETX.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.439 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SETX">SETX</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0011965, decreased total retina thickness</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0011965, decreased total retina thickness</dd><dd>HP:0000520, Proptosis - MP:0011965, decreased total retina thickness</dd><dd>HP:0000601, Hypotelorism - MP:0011965, decreased total retina thickness</dd><dd>HP:0000244, Brachyturricephaly - MP:0011965, decreased total retina thickness</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/602433">OMIM:602433</a> Amyotrophic lateral sclerosis 4, juvenile - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/606002">OMIM:606002</a> Spinocerebellar ataxia, autosomal recessive, with axonal neuropathy 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=357043">ORPHA:357043</a> Amyotrophic lateral sclerosis type 4 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64753">ORPHA:64753</a> Spinocerebellar ataxia with axonal neuropathy type 2 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.045</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.439</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.618</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-135140221-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs376524608">rs376524608</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1008411%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.618</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000224140.5">SETX:ENST00000224140.5:c.7439C&gt;T:p.(Ala2480Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372169.2">SETX:ENST00000372169.2:c.7526C&gt;T:p.(Ala2509Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393220.1">SETX:ENST00000393220.1:c.7340C&gt;T:p.(Ala2447Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.61875004</dd>
                                    
                                    <dd>REVEL: 0.088</dd>
                                    <dd>MVP: 0.619</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0075%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SPAG9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000008294">SPAG9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.045</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.542</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPAG9%0DCDON&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDON</a> and phenotypic similarity 0.694 to <a href="http://www.omim.org/entry/614226">Holoprosencephaly 11</a> associated with CDON.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000175, Cleft palate</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SPAG9%0DCDON&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CDON</a> and phenotypic similarity 0.698 to mouse mutant of CDON.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004462, small basisphenoid bone</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000520, Proptosis - MP:0004462, small basisphenoid bone</dd><dd>HP:0000601, Hypotelorism - MP:0004462, small basisphenoid bone</dd><dd>HP:0000244, Brachyturricephaly - MP:0004462, small basisphenoid bone</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.045</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.542</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-49072486-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1237431261">rs1237431261</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.542</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262013.7">SPAG9:ENST00000262013.7:c.2158G&gt;A:p.(Asp720Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357122.4">SPAG9:ENST00000357122.4:c.2116G&gt;A:p.(Asp706Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505279.1">SPAG9:ENST00000505279.1:c.2128G&gt;A:p.(Asp710Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510283.1">SPAG9:ENST00000510283.1:c.1687G&gt;A:p.(Asp563Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5418759</dd>
                                    
                                    <dd>REVEL: 0.087</dd>
                                    <dd>MVP: 0.542</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC39A5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139540">SLC39A5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.045</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.541</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A5%0DSLC39A13&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC39A13</a> and phenotypic similarity 0.592 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=157965">SLC39A13-related spondylodysplastic Ehlers-Danlos syndrome</a> associated with SLC39A13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001182, Tapered finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0100864, Short femoral neck</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A5%0DSLC39A13&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC39A13</a> and phenotypic similarity 0.797 to mouse mutant of SLC39A13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000133, abnormal long bone metaphysis morphology</dd><dd>HP:0000486, Strabismus - MP:0006198, enophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004540, small maxilla</dd><dd>HP:0000520, Proptosis - MP:0006198, enophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0006198, enophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0000454, abnormal jaw morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615946">OMIM:615946</a> Myopia 24, autosomal dominant - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.045</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.541</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-56629033-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147133811">rs147133811</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2394016%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.541</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000266980.4">SLC39A5:ENST00000266980.4:c.727C&gt;T:p.(Arg243Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000454355.2">SLC39A5:ENST00000454355.2:c.727C&gt;T:p.(Arg243Trp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5464358</dd>
                                    
                                    <dd>REVEL: 0.302</dd>
                                    <dd>MVP: 0.546</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0079%</dd>
                                    <dd>ESP AA: 0.0681%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0391%</dd>
                                    <dd>ExAC NFE: 0.0093%</dd>
                                    <dd>gnomAD_E_AFR: 0.0201%</dd>
                                    <dd>gnomAD_E_AMR: 0.0071%</dd>
                                    <dd>gnomAD_E_NFE: 0.0060%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LAMA3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000053747">LAMA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.039</b>
                    <data>(p=2.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.253</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.813</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.330 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79404">Severe generalized junctional epidermolysis bullosa</a> associated with LAMA3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001211, Abnormal fingertip morphology</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0006297, Enamel hypoplasia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.505 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LAMA3">LAMA3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002576, abnormal enamel morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002576, abnormal enamel morphology</dd><dd>HP:0000520, Proptosis - MP:0002576, abnormal enamel morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002576, abnormal enamel morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002576, abnormal enamel morphology</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LAMA3%0DZEB1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZEB1</a> and phenotypic similarity 0.383 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98973">Posterior polymorphous corneal dystrophy</a> associated with ZEB1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LAMA3%0DZEB1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZEB1</a> and phenotypic similarity 0.757 to mouse mutant of ZEB1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus - MP:0000106, abnormal basisphenoid bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0000106, abnormal basisphenoid bone morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000106, abnormal basisphenoid bone morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002116, abnormal craniofacial bone morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/245660">OMIM:245660</a> Epidermolysis bullosa, junctional 2C, laryngoonychocutaneous - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/619783">OMIM:619783</a> Epidermolysis bullosa, junctional 2A, intermediate - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/619784">OMIM:619784</a> Epidermolysis bullosa, junctional 2B, severe - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79402">ORPHA:79402</a> Intermediate generalized junctional epidermolysis bullosa - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79404">ORPHA:79404</a> Severe generalized junctional epidermolysis bullosa - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.039</b>
                                    <data>(p=2.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.253</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.813</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>18-21494777-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201547061">rs201547061</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.813</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269217.6">LAMA3:ENST00000269217.6:c.2770C&gt;A:p.(Pro924Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313654.9">LAMA3:ENST00000313654.9:c.7597C&gt;A:p.(Pro2533Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399516.3">LAMA3:ENST00000399516.3:c.7429C&gt;A:p.(Pro2477Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000587184.1">LAMA3:ENST00000587184.1:c.2602C&gt;A:p.(Pro868Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8179819</dd>
                                    
                                    <dd>REVEL: 0.555</dd>
                                    <dd>MVP: 0.818</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0015%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SVIL" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197321">SVIL</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.035</b>
                    <data>(p=2.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.253</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.351 to <a href="http://www.omim.org/entry/619040">Myofibrillar myopathy 10</a> associated with SVIL.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001852, Sandal gap</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000303, Mandibular prognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SVIL%0DSH3PXD2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3PXD2B</a> and phenotypic similarity 0.648 to <a href="http://www.omim.org/entry/249420">Frank-ter Haar syndrome</a> associated with SH3PXD2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0009803, Short phalanx of finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SVIL%0DSH3PXD2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3PXD2B</a> and phenotypic similarity 0.862 to mouse mutant of SH3PXD2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0004686, decreased length of long bones</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002750, exophthalmos</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0002750, exophthalmos</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619040">OMIM:619040</a> Myofibrillar myopathy 10 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.035</b>
                                    <data>(p=2.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.253</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>10-29813396-C-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355867.4">SVIL:ENST00000355867.4:c.2586+5G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375398.2">SVIL:ENST00000375398.2:c.2586+5G&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375400.3">SVIL:ENST00000375400.3:c.1308+5G&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RNF113B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139797">RNF113B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.029</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.497</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF113B%0DRNF113A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF113A</a> and phenotypic similarity 0.698 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with RNF113A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001217, Clubbing</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0410219, Hypoplasia of mandible relative to maxilla</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001363, Craniosynostosis</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF113B%0DRNF113A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF113A</a> and phenotypic similarity 0.523 to mouse mutant of RNF113A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000157, abnormal sternum morphology</dd><dd>HP:0000486, Strabismus - MP:0010097, abnormal retina blood vessel morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0010097, abnormal retina blood vessel morphology</dd><dd>HP:0000520, Proptosis - MP:0010097, abnormal retina blood vessel morphology</dd><dd>HP:0000601, Hypotelorism - MP:0010097, abnormal retina blood vessel morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0010097, abnormal retina blood vessel morphology</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RNF113B%0DRNF113A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RNF113A</a> and phenotypic similarity 0.310 to fish mutant of RNF113A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.029</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.497</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>13-98829175-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1272124433">rs1272124433</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.497</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000267291.6">RNF113B:ENST00000267291.6:c.316G&gt;C:p.(Glu106Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000319562.6">RNF113B:ENST00000319562.6:c.-24+33429C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376581.5">RNF113B:ENST00000376581.5:c.-24+33429C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376586.2">RNF113B:ENST00000376586.2:c.-24+33429C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000595437.1">RNF113B:ENST00000595437.1:c.-24+34144C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.49662802</dd>
                                    
                                    <dd>REVEL: 0.102</dd>
                                    <dd>MVP: 0.497</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RBM34" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000188739">RBM34</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.027</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.489</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBM34%0DDKC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DKC1</a> and phenotypic similarity 0.614 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1775">Dyskeratosis congenita</a> associated with DKC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0008404, Nail dystrophy</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBM34%0DDKC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DKC1</a> and phenotypic similarity 0.310 to fish mutant of DKC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.027</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.489</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-235324212-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs184349413">rs184349413</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.524</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366606.3">RBM34:ENST00000366606.3:c.209C&gt;T:p.(Pro70Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408888.3">RBM34:ENST00000408888.3:c.224C&gt;T:p.(Pro75Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.54412895</dd>
                                    
                                    <dd>REVEL: 0.084</dd>
                                    <dd>MVP: 0.544</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1797%</dd>
                                    <dd>TOPMed: 0.0725%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.0785%</dd>
                                    <dd>ESP EA: 0.0364%</dd>
                                    <dd>ESP All: 0.0498%</dd>
                                    <dd>ExAC AFR: 0.0833%</dd>
                                    <dd>ExAC AMR: 0.1649%</dd>
                                    <dd>ExAC NFE: 0.0333%</dd>
                                    <dd>ExAC OTH: 0.1126%</dd>
                                    <dd>gnomAD_E_AFR: 0.0658%</dd>
                                    <dd>gnomAD_E_AMR: 0.1641%</dd>
                                    <dd>gnomAD_E_NFE: 0.0342%</dd>
                                    <dd>gnomAD_E_OTH: 0.2015%</dd>
                                    <dd>gnomAD_G_AFR: 0.0802%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0466%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-235324219-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs181926176">rs181926176</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.454</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366606.3">RBM34:ENST00000366606.3:c.202C&gt;T:p.(Pro68Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408888.3">RBM34:ENST00000408888.3:c.217C&gt;T:p.(Pro73Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.47162008</dd>
                                    
                                    <dd>REVEL: 0.086</dd>
                                    <dd>MVP: 0.472</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1797%</dd>
                                    <dd>TOPMed: 0.1160%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.0779%</dd>
                                    <dd>ESP EA: 0.0364%</dd>
                                    <dd>ESP All: 0.0496%</dd>
                                    <dd>ExAC AFR: 0.0831%</dd>
                                    <dd>ExAC AMR: 0.1649%</dd>
                                    <dd>ExAC NFE: 0.0333%</dd>
                                    <dd>ExAC OTH: 0.1126%</dd>
                                    <dd>gnomAD_E_AFR: 0.0723%</dd>
                                    <dd>gnomAD_E_AMR: 0.1669%</dd>
                                    <dd>gnomAD_E_NFE: 0.0342%</dd>
                                    <dd>gnomAD_E_OTH: 0.2012%</dd>
                                    <dd>gnomAD_G_AFR: 0.0802%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0467%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="USP2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000036672">USP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.026</b>
                    <data>(p=2.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.485</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP2%0DSMO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMO</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2495">Meningioma</a> associated with SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0012246, Oculomotor nerve palsy</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP2%0DSMO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMO</a> and phenotypic similarity 0.467 to mouse mutant of SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000520, Proptosis - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000601, Hypotelorism - MP:0003425, abnormal optic vesicle formation</dd><dd>HP:0000244, Brachyturricephaly - MP:0003425, abnormal optic vesicle formation</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP2%0DSMO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SMO</a> and phenotypic similarity 0.462 to fish mutant of SMO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - ZP:0010276, eye adjacent to eye, abnormal</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001147, mandibular arch skeleton aplastic, abnormal</dd><dd>HP:0000520, Proptosis - ZP:0010276, eye adjacent to eye, abnormal</dd><dd>HP:0000601, Hypotelorism - ZP:0010276, eye adjacent to eye, abnormal</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.026</b>
                                    <data>(p=2.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.485</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-119243493-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs752965596">rs752965596</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.485</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260187.2">USP2:ENST00000260187.2:c.698G&gt;A:p.(Arg233His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455332.2">USP2:ENST00000455332.2:c.45+8690G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48889768</dd>
                                    
                                    <dd>REVEL: 0.115</dd>
                                    <dd>MVP: 0.489</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0038%</dd>
                                    <dd>ExAC EAS: 0.0233%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AMR: 0.0064%</dd>
                                    <dd>gnomAD_E_EAS: 0.0297%</dd>
                                    <dd>gnomAD_E_FIN: 0.0049%</dd>
                                    <dd>gnomAD_E_NFE: 0.0029%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RAB11FIP5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135631">RAB11FIP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.025</b>
                    <data>(p=2.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.480</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RAB11FIP5%0DASAP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ASAP1</a> and phenotypic similarity 0.707 to mouse mutant of ASAP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0005306, abnormal phalanx morphology</dd><dd>HP:0000486, Strabismus - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002820, abnormal premaxilla morphology</dd><dd>HP:0000520, Proptosis - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0008525, decreased cranium height</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.025</b>
                                    <data>(p=2.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.480</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-73315604-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199517071">rs199517071</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.480</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258098.6">RAB11FIP5:ENST00000258098.6:c.1142G&gt;A:p.(Arg381His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48257437</dd>
                                    
                                    <dd>REVEL: 0.042</dd>
                                    <dd>MVP: 0.483</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC NFE: 0.0091%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PARS2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162396">PARS2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.024</b>
                    <data>(p=2.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.623</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.339</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.623 to <a href="http://www.omim.org/entry/618437">Developmental and epileptic encephalopathy 75</a> associated with PARS2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000331, Short chin</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0005484, Secondary microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PARS2%0DCARS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CARS1</a> and phenotypic similarity 0.698 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=33364">Trichothiodystrophy</a> associated with CARS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001217, Clubbing</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0410219, Hypoplasia of mandible relative to maxilla</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001363, Craniosynostosis</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PARS2%0DCARS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CARS1</a> and phenotypic similarity 0.310 to fish mutant of CARS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618437">OMIM:618437</a> Developmental and epileptic encephalopathy 75 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">ORPHA:442835</a> Non-specific early-onset epileptic encephalopathy - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.024</b>
                                    <data>(p=2.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.623</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.339</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.623 to OMIM:618437 Developmental and epileptic encephalopathy 75</dt>
                                
                                    <dt>Phenotypic similarity 0.354 to ORPHA:442835 Non-specific early-onset epileptic encephalopathy</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>1-55224726-CTCT-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs751997497">rs751997497</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM4, PP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=405217%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.602</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371279.3">PARS2:ENST00000371279.3:c.106_108del:p.(Arg36del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.1779%</dd>
                                    <dd>ESP AA: 1.1486%</dd>
                                    <dd>ESP EA: 0.5815%</dd>
                                    <dd>ESP All: 0.7748%</dd>
                                    <dd>ExAC AFR: 0.6141%</dd>
                                    <dd>ExAC AMR: 0.0437%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>ExAC OTH: 0.1131%</dd>
                                    <dd>gnomAD_E_AFR: 0.5118%</dd>
                                    <dd>gnomAD_E_AMR: 0.0387%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.5270%</dd>
                                    <dd>gnomAD_G_AMR: 0.1196%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-55224346-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145967472">rs145967472</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=365357%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.075</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371279.3">PARS2:ENST00000371279.3:c.489G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.3121%</dd>
                                    <dd>ESP AA: 1.0440%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.3767%</dd>
                                    <dd>ExAC AFR: 0.9514%</dd>
                                    <dd>ExAC AMR: 0.0777%</dd>
                                    <dd>ExAC NFE: 0.0075%</dd>
                                    <dd>ExAC OTH: 0.2203%</dd>
                                    <dd>gnomAD_E_AFR: 0.8495%</dd>
                                    <dd>gnomAD_E_AMR: 0.0625%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_G_AFR: 0.9164%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RIPOR2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111913">RIPOR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.022</b>
                    <data>(p=2.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.469</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.304 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RIPOR2">RIPOR2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004521, abnormal cochlear hair cell stereociliary bundle morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004521, abnormal cochlear hair cell stereociliary bundle morphology</dd><dd>HP:0000520, Proptosis - MP:0004521, abnormal cochlear hair cell stereociliary bundle morphology</dd><dd>HP:0000601, Hypotelorism - MP:0004521, abnormal cochlear hair cell stereociliary bundle morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0004521, abnormal cochlear hair cell stereociliary bundle morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIPOR2%0DNTRK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NTRK2</a> and phenotypic similarity 0.354 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=442835">Non-specific early-onset epileptic encephalopathy</a> associated with NTRK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000639, Nystagmus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000668, Hypodontia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RIPOR2%0DNTRK2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NTRK2</a> and phenotypic similarity 0.605 to mouse mutant of NTRK2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002638, abnormal pupillary reflex</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000520, Proptosis - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001340, abnormal eyelid morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0009772, abnormal retina development</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/607017">OMIM:607017</a> Deafness, autosomal dominant 21 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/616515">OMIM:616515</a> ?Deafness, autosomal recessive 104 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.022</b>
                                    <data>(p=2.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.469</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-24843131-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201962345">rs201962345</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2104817%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.469</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000259698.4">RIPOR2:ENST00000259698.4:c.1879G&gt;A:p.(Asp627Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378023.4">RIPOR2:ENST00000378023.4:c.1729G&gt;A:p.(Asp577Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510784.2">RIPOR2:ENST00000510784.2:c.1831G&gt;A:p.(Asp611Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538035.1">RIPOR2:ENST00000538035.1:c.1816G&gt;A:p.(Asp606Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540914.1">RIPOR2:ENST00000540914.1:c.1729G&gt;A:p.(Asp577Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4740744</dd>
                                    
                                    <dd>REVEL: 0.045</dd>
                                    <dd>MVP: 0.474</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0404%</dd>
                                    <dd>UK10K: 0.0661%</dd>
                                    <dd>ESP EA: 0.0732%</dd>
                                    <dd>ESP All: 0.0505%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0438%</dd>
                                    <dd>gnomAD_E_AFR: 0.0071%</dd>
                                    <dd>gnomAD_E_AMR: 0.0559%</dd>
                                    <dd>gnomAD_E_FIN: 0.0057%</dd>
                                    <dd>gnomAD_E_NFE: 0.0612%</dd>
                                    <dd>gnomAD_E_OTH: 0.0307%</dd>
                                    <dd>gnomAD_G_NFE: 0.0333%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NFE2L3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000050344">NFE2L3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.021</b>
                    <data>(p=2.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.461</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NFE2L3%0DMAFB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAFB</a> and phenotypic similarity 0.712 to <a href="http://www.omim.org/entry/166300">Multicentric carpotarsal osteolysis syndrome</a> associated with MAFB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NFE2L3%0DMAFB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MAFB</a> and phenotypic similarity 0.449 to mouse mutant of MAFB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0009913, abnormal hyoid bone greater horn morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009913, abnormal hyoid bone greater horn morphology</dd><dd>HP:0000520, Proptosis - MP:0009913, abnormal hyoid bone greater horn morphology</dd><dd>HP:0000601, Hypotelorism - MP:0009913, abnormal hyoid bone greater horn morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0009913, abnormal hyoid bone greater horn morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.021</b>
                                    <data>(p=2.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.461</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-26224829-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143155770">rs143155770</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.461</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000056233.3">NFE2L3:ENST00000056233.3:c.1511C&gt;G:p.(Pro504Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4626358</dd>
                                    
                                    <dd>REVEL: 0.079</dd>
                                    <dd>MVP: 0.463</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0207%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP EA: 0.0233%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0153%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="COG7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168434">COG7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.020</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.276</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.711</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.458 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79333">COG7-CDG</a> associated with COG7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001181, Adducted thumb</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0011451, Primary microcephaly</dd></dl><dl><dt>Phenotypic similarity 0.552 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=COG7">COG7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006243, impaired pupillary reflex</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005544, corneal deposits</dd><dd>HP:0000520, Proptosis - MP:0005544, corneal deposits</dd><dd>HP:0000601, Hypotelorism - MP:0005544, corneal deposits</dd><dd>HP:0000244, Brachyturricephaly - MP:0005544, corneal deposits</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=COG7%0DCOG4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COG4</a> and phenotypic similarity 0.611 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85172">Microcephalic osteodysplastic dysplasia, Saul-Wilson type</a> associated with COG4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0010230, Cone-shaped epiphyses of the phalanges of the hand</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000272, Malar flattening</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.512 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=COG7%0DCOG4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COG4</a> and phenotypic similarity 0.338 to fish mutant of COG4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001965, mandibular arch skeleton decreased size, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/608779">OMIM:608779</a> Congenital disorder of glycosylation, type IIe - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79333">ORPHA:79333</a> COG7-CDG - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.020</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.276</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.711</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-23456431-G-C [0/1]</b>
                            
                            
                                
                                    
                                    
                                    
                                    <span>1899619</span>
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1960295%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.711</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000307149.5">COG7:ENST00000307149.5:c.373C&gt;G:p.(Leu125Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7109982</dd>
                                    
                                    <dd>REVEL: 0.497</dd>
                                    <dd>MVP: 0.711</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FAT3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165323">FAT3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.019</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.450</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.466 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=FAT3">FAT3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001325, abnormal retina morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001325, abnormal retina morphology</dd><dd>HP:0000520, Proptosis - MP:0001325, abnormal retina morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001325, abnormal retina morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001325, abnormal retina morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAT3%0DDCHS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCHS1</a> and phenotypic similarity 0.744 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=314679">Cerebrofacioarticular syndrome</a> associated with DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001159, Syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FAT3%0DDCHS1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DCHS1</a> and phenotypic similarity 0.504 to mouse mutant of DCHS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0012279, wide sternum</dd><dd>HP:0000486, Strabismus - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004321, short sternum</dd><dd>HP:0000520, Proptosis - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000031, abnormal cochlea morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000031, abnormal cochlea morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.019</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.450</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-92531088-T-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1947218444">rs1947218444</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.450</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298047.6">FAT3:ENST00000298047.6:c.4909T&gt;A:p.(Phe1637Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409404.2">FAT3:ENST00000409404.2:c.4909T&gt;A:p.(Phe1637Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000525166.1">FAT3:ENST00000525166.1:c.4459T&gt;A:p.(Phe1487Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.45</dd>
                                    
                                    <dd>REVEL: 0.450</dd>
                                    <dd>MVP: 0.241</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0015%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ANKZF1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163516">ANKZF1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.018</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.444</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKZF1%0DUBE4B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE4B</a> and phenotypic similarity 0.615 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1606">1p36 deletion syndrome</a> associated with UBE4B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000307, Pointed chin</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ANKZF1%0DUBE4B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UBE4B</a> and phenotypic similarity 0.217 to mouse mutant of UBE4B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0001917, intraventricular hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.018</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.444</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-220097286-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200506719">rs200506719</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=708050%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.451</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323348.5">ANKZF1:ENST00000323348.5:c.439G&gt;A:p.(Glu147Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410034.3">ANKZF1:ENST00000410034.3:c.439G&gt;A:p.(Glu147Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409849.1">ANKZF1:ENST00000409849.1:c.-192G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6708275</dd>
                                    
                                    <dd>REVEL: 0.087</dd>
                                    <dd>MVP: 0.671</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1997%</dd>
                                    <dd>TOPMed: 0.4896%</dd>
                                    <dd>UK10K: 0.7009%</dd>
                                    <dd>ESP AA: 0.2347%</dd>
                                    <dd>ESP EA: 0.6072%</dd>
                                    <dd>ESP All: 0.4889%</dd>
                                    <dd>ExAC AFR: 0.1123%</dd>
                                    <dd>ExAC AMR: 0.3629%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 0.3477%</dd>
                                    <dd>ExAC NFE: 0.7643%</dd>
                                    <dd>ExAC OTH: 0.7795%</dd>
                                    <dd>ExAC SAS: 1.2294%</dd>
                                    <dd>gnomAD_E_AFR: 0.0916%</dd>
                                    <dd>gnomAD_E_AMR: 0.4526%</dd>
                                    <dd>gnomAD_E_EAS: 0.0174%</dd>
                                    <dd>gnomAD_E_FIN: 0.3409%</dd>
                                    <dd>gnomAD_E_NFE: 0.7179%</dd>
                                    <dd>gnomAD_E_OTH: 0.7299%</dd>
                                    <dd>gnomAD_E_SAS: 1.1858%</dd>
                                    <dd>gnomAD_G_AFR: 0.1490%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_FIN: 0.3721%</dd>
                                    <dd>gnomAD_G_NFE: 0.7131%</dd>
                                    <dd>gnomAD_G_OTH: 0.4073%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-220099713-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140395841">rs140395841</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=708052%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.437</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000323348.5">ANKZF1:ENST00000323348.5:c.1370C&gt;T:p.(Thr457Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409849.1">ANKZF1:ENST00000409849.1:c.740C&gt;T:p.(Thr247Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410034.3">ANKZF1:ENST00000410034.3:c.1370C&gt;T:p.(Thr457Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6497504</dd>
                                    
                                    <dd>REVEL: 0.042</dd>
                                    <dd>MVP: 0.650</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1997%</dd>
                                    <dd>TOPMed: 0.4874%</dd>
                                    <dd>UK10K: 0.6612%</dd>
                                    <dd>ESP AA: 0.2217%</dd>
                                    <dd>ESP EA: 0.5978%</dd>
                                    <dd>ESP All: 0.4749%</dd>
                                    <dd>ExAC AFR: 0.1022%</dd>
                                    <dd>ExAC AMR: 0.3631%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 0.3477%</dd>
                                    <dd>ExAC NFE: 0.7622%</dd>
                                    <dd>ExAC OTH: 0.7848%</dd>
                                    <dd>ExAC SAS: 1.2297%</dd>
                                    <dd>gnomAD_E_AFR: 0.0786%</dd>
                                    <dd>gnomAD_E_AMR: 0.4527%</dd>
                                    <dd>gnomAD_E_EAS: 0.0174%</dd>
                                    <dd>gnomAD_E_FIN: 0.3408%</dd>
                                    <dd>gnomAD_E_NFE: 0.7102%</dd>
                                    <dd>gnomAD_E_OTH: 0.7299%</dd>
                                    <dd>gnomAD_E_SAS: 1.1858%</dd>
                                    <dd>gnomAD_G_AFR: 0.1489%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_FIN: 0.3721%</dd>
                                    <dd>gnomAD_G_NFE: 0.6998%</dd>
                                    <dd>gnomAD_G_OTH: 0.4073%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CRY2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121671">CRY2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.018</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.444</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CRY2%0DDYRK1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DYRK1A</a> and phenotypic similarity 0.691 to <a href="http://www.omim.org/entry/614104">Intellectual developmental disorder, autosomal dominant 7</a> associated with DYRK1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001822, Hallux valgus</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CRY2%0DDYRK1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DYRK1A</a> and phenotypic similarity 0.622 to mouse mutant of DYRK1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000433, microcephaly</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000433, microcephaly</dd><dd>HP:0000520, Proptosis - MP:0000433, microcephaly</dd><dd>HP:0000601, Hypotelorism - MP:0000433, microcephaly</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.018</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.444</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-45891977-T-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.444</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000417225.2">CRY2:ENST00000417225.2:c.1260T&gt;G:p.(Ile420Met)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443527.2">CRY2:ENST00000443527.2:c.1506T&gt;G:p.(Ile502Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.444</dd>
                                    
                                    <dd>REVEL: 0.444</dd>
                                    <dd>MVP: 0.345</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CTBP2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000175029">CTBP2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.017</b>
                    <data>(p=2.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.521</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.417</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.466 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CTBP2">CTBP2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001314, corneal opacity</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001314, corneal opacity</dd><dd>HP:0000520, Proptosis - MP:0001314, corneal opacity</dd><dd>HP:0000601, Hypotelorism - MP:0001314, corneal opacity</dd><dd>HP:0000244, Brachyturricephaly - MP:0001314, corneal opacity</dd></dl><dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CTBP2%0DPSAT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSAT1</a> and phenotypic similarity 0.678 to <a href="http://www.omim.org/entry/616038">Neu-Laxova syndrome 2</a> associated with PSAT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CTBP2%0DPSAT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSAT1</a> and phenotypic similarity 0.175 to mouse mutant of PSAT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000914, exencephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.017</b>
                                    <data>(p=2.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.521</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.417</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-126715565-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143281706">rs143281706</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.417</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000309035.6">CTBP2:ENST00000309035.6:c.764G&gt;A:p.(Arg255His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337195.5">CTBP2:ENST00000337195.5:c.58+12001G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000411419.2">CTBP2:ENST00000411419.2:c.58+12001G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000494626.2">CTBP2:ENST00000494626.2:c.58+12001G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000531469.1">CTBP2:ENST00000531469.1:c.58+12001G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.41887692</dd>
                                    
                                    <dd>REVEL: 0.224</dd>
                                    <dd>MVP: 0.419</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0079%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0106%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0108%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_E_SAS: 0.0195%</dd>
                                    <dd>gnomAD_G_NFE: 0.0334%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BAGE2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187172">BAGE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                            <b>DEL</b>
                            <b>21-11029596-AC-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs60459764">rs60459764</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470054.1">BAGE2:ENST00000470054.1:n.1596+1del:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.900</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                            <b>DEL</b>
                            <b>21-11029596-AC-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs60459764">rs60459764</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470054.1">BAGE2:ENST00000470054.1:n.1596+1del:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>INS</b>
                            <b>21-11021215-T-TA [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs71222366">rs71222366</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470054.1">BAGE2:ENST00000470054.1:n.1714-5_1714-4insT:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RP11-293B20.2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000259882">RP11-293B20.2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_ACCEPTOR_VARIANT</span>
                            <b>INS</b>
                            <b>16-33404683-C-CAGG [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs202040953">rs202040953</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000566346.1">RP11-293B20.2:ENST00000566346.1:n.616-3_616-2insAGG:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RP11-683L23.1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173213">RP11-683L23.1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_ELONGATION</span>
                            <b>INS</b>
                            <b>18-49110-A-AGAAGGACGGGGGTCTCACCGCTGGCCTCGTGGTGGTACACGTTG [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000573909.1">RP11-683L23.1:ENST00000573909.1:c.108_109insCAACGTGTACCACCACGAGGCCAGCGGTGAGACCCCCGTCCTTC:p.(Ser37Glnfs*10)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308911.6">RP11-683L23.1:ENST00000308911.6:c.166+18_166+19insCAACGTGTACCACCACGAGGCCAGCGGTGAGACCCCCGTCCTTC:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TMEM255B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184497">TMEM255B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>1.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>1.000</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                            <b>DEL</b>
                            <b>13-114503875-GCCTGCGGGAGGTGAGGGGCACCGGGGACCCCCATATCTACACCTGCGGGAGGTGAGGGGCGCTGGGGACCCCCGTATCTACA-G [0/1]</b>
                            
                            
                                
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/dbvar/variants/nsv3045916">nsv3045916</a>
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 1.000</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375353.3">TMEM255B:ENST00000375353.3:c.500_509+72del:p.(Cys168Glnfs*32)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FAM218A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000250486">FAM218A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.999</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">STOP_GAINED</span>
                            <b>SNV</b>
                            <b>4-165878621-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs767479404">rs767479404</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000513876.2">FAM218A:ENST00000513876.2:c.447G&gt;A:p.(Trp149*)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329314.5">FAM218A:ENST00000329314.5:c.526-2373C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000596751.1">FAM218A:ENST00000596751.1:n.469-3666G&gt;A:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0048%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF675" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197372">ZNF675</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.999</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.999</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_TRUNCATION</span>
                            <b>DEL</b>
                            <b>19-23844937-CA-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1359868666">rs1359868666</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.999</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000596211.1">ZNF675:ENST00000596211.1:c.204del:p.(His68Glnfs*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000599168.1">ZNF675:ENST00000599168.1:c.204del:p.(His68Glnfs*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359788.4">ZNF675:ENST00000359788.4:c.204del:p.(His68Glnfs*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000600313.1">ZNF675:ENST00000600313.1:c.204del:p.(His68Glnfs*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601010.1">ZNF675:ENST00000601010.1:c.204del:p.(His68Glnfs*4)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000601935.1">ZNF675:ENST00000601935.1:c.204del:p.(His68Glnfs*4)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RASGEF1B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138670">RASGEF1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.433</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RASGEF1B%0DHRAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HRAS</a> and phenotypic similarity 0.561 to <a href="http://www.omim.org/entry/218040">Congenital myopathy with excess of muscle spindles</a> associated with HRAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001808, Fragile nails</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RASGEF1B%0DHRAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HRAS</a> and phenotypic similarity 0.704 to mouse mutant of HRAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000107, abnormal frontal bone morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000107, abnormal frontal bone morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.433</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-82369284-T-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs748675949">rs748675949</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.433</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264400.2">RASGEF1B:ENST00000264400.2:c.593A&gt;C:p.(Asp198Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335927.7">RASGEF1B:ENST00000335927.7:c.467A&gt;C:p.(Asp156Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509081.1">RASGEF1B:ENST00000509081.1:c.590A&gt;C:p.(Asp197Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.43371364</dd>
                                    
                                    <dd>REVEL: 0.318</dd>
                                    <dd>MVP: 0.434</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0068%</dd>
                                    <dd>ExAC NFE: 0.0120%</dd>
                                    <dd>gnomAD_E_NFE: 0.0116%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FAM228A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186453">FAM228A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.998</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.998</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">FRAMESHIFT_VARIANT</span>
                            <b>DEL</b>
                            <b>2-24413293-CA-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs745950354">rs745950354</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.998</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295150.3">FAM228A:ENST00000295150.3:c.415del:p.(Ile139Serfs*17)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000584973.1">FAM228A:ENST00000584973.1:c.*1568del:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0023%</dd>
                                    <dd>ESP EA: 0.0128%</dd>
                                    <dd>ESP All: 0.0088%</dd>
                                    <dd>ExAC AMR: 0.0090%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POLR3C" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186141">POLR3C</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.996</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.996</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_DONOR_VARIANT</span>
                            <b>SNV</b>
                            <b>1-145606274-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs782558836">rs782558836</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.996</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334163.3">POLR3C:ENST00000334163.3:c.678+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369294.1">POLR3C:ENST00000369294.1:c.678+1G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0268%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="IGF2BP3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000136231">IGF2BP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.424</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=IGF2BP3%0DHMGA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HMGA2</a> and phenotypic similarity 0.561 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=397590">Silver-Russell syndrome due to a point mutation</a> associated with HMGA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001159, Syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0004482, Relative macrocephaly</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=IGF2BP3%0DHMGA2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HMGA2</a> and phenotypic similarity 0.754 to mouse mutant of HMGA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000435, shortened head</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000435, shortened head</dd><dd>HP:0000520, Proptosis - MP:0000435, shortened head</dd><dd>HP:0000601, Hypotelorism - MP:0000435, shortened head</dd><dd>HP:0000244, Brachyturricephaly - MP:0000435, shortened head</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.424</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-23383339-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150697725">rs150697725</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.552</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258729.3">IGF2BP3:ENST00000258729.3:c.1075A&gt;G:p.(Asn359Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.5730152</dd>
                                    
                                    <dd>REVEL: 0.142</dd>
                                    <dd>MVP: 0.573</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.1511%</dd>
                                    <dd>UK10K: 0.0661%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.1977%</dd>
                                    <dd>ESP All: 0.1384%</dd>
                                    <dd>ExAC AFR: 0.0289%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC NFE: 0.2430%</dd>
                                    <dd>ExAC SAS: 0.1455%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0923%</dd>
                                    <dd>gnomAD_E_NFE: 0.2151%</dd>
                                    <dd>gnomAD_E_OTH: 0.2015%</dd>
                                    <dd>gnomAD_E_SAS: 0.1692%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.1266%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>7-23358880-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs117151817">rs117151817</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=777654%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.296</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258729.3">IGF2BP3:ENST00000258729.3:c.1204-7C&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.6390%</dd>
                                    <dd>TOPMed: 1.0140%</dd>
                                    <dd>UK10K: 1.7323%</dd>
                                    <dd>ESP AA: 0.3858%</dd>
                                    <dd>ESP EA: 1.6860%</dd>
                                    <dd>ESP All: 1.2456%</dd>
                                    <dd>ExAC AFR: 0.2979%</dd>
                                    <dd>ExAC AMR: 1.0886%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 0.3326%</dd>
                                    <dd>ExAC NFE: 1.2455%</dd>
                                    <dd>ExAC OTH: 1.1086%</dd>
                                    <dd>ExAC SAS: 0.6399%</dd>
                                    <dd>gnomAD_E_AFR: 0.2793%</dd>
                                    <dd>gnomAD_E_AMR: 1.1595%</dd>
                                    <dd>gnomAD_E_EAS: 0.0121%</dd>
                                    <dd>gnomAD_E_FIN: 0.4520%</dd>
                                    <dd>gnomAD_E_NFE: 1.3454%</dd>
                                    <dd>gnomAD_E_OTH: 0.5941%</dd>
                                    <dd>gnomAD_E_SAS: 0.5048%</dd>
                                    <dd>gnomAD_G_AFR: 0.3093%</dd>
                                    <dd>gnomAD_G_AMR: 1.0740%</dd>
                                    <dd>gnomAD_G_FIN: 0.2291%</dd>
                                    <dd>gnomAD_G_NFE: 1.3400%</dd>
                                    <dd>gnomAD_G_OTH: 0.7128%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AK9" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000155085">AK9</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.364</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.583</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.364 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98913">Postsynaptic congenital myasthenic syndromes</a> associated with AK9.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000597, Ophthalmoparesis</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000218, High palate</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98913">ORPHA:98913</a> Postsynaptic congenital myasthenic syndromes - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.364</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.583</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.364 to ORPHA:98913 Postsynaptic congenital myasthenic syndromes</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>6-109954383-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201757833">rs201757833</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.790</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285397.5">AK9:ENST00000285397.5:c.1073+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341338.6">AK9:ENST00000341338.6:c.-1691+4A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368948.2">AK9:ENST00000368948.2:c.1073+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424296.2">AK9:ENST00000424296.2:c.1073+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ExAC NFE: 0.0091%</dd>
                                    <dd>ExAC SAS: 0.0783%</dd>
                                    <dd>gnomAD_E_FIN: 0.0046%</dd>
                                    <dd>gnomAD_E_NFE: 0.0095%</dd>
                                    <dd>gnomAD_E_SAS: 0.0853%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-109954427-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200088189">rs200088189</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.375</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285397.5">AK9:ENST00000285397.5:c.1033G&gt;A:p.(Asp345Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368948.2">AK9:ENST00000368948.2:c.1033G&gt;A:p.(Asp345Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424296.2">AK9:ENST00000424296.2:c.1033G&gt;A:p.(Asp345Asn)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341338.6">AK9:ENST00000341338.6:c.-1731G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3798815</dd>
                                    
                                    <dd>REVEL: 0.074</dd>
                                    <dd>MVP: 0.380</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ExAC NFE: 0.0091%</dd>
                                    <dd>ExAC SAS: 0.0732%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0092%</dd>
                                    <dd>gnomAD_E_SAS: 0.0853%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.790</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>6-109954383-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201757833">rs201757833</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.790</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000285397.5">AK9:ENST00000285397.5:c.1073+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000341338.6">AK9:ENST00000341338.6:c.-1691+4A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368948.2">AK9:ENST00000368948.2:c.1073+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424296.2">AK9:ENST00000424296.2:c.1073+4A&gt;G:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ExAC NFE: 0.0091%</dd>
                                    <dd>ExAC SAS: 0.0783%</dd>
                                    <dd>gnomAD_E_FIN: 0.0046%</dd>
                                    <dd>gnomAD_E_NFE: 0.0095%</dd>
                                    <dd>gnomAD_E_SAS: 0.0853%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAH8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000124721">DNAH8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.016</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.254</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.708</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH8%0DDNAH5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNAH5</a> and phenotypic similarity 0.319 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAH5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001217, Clubbing</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0011109, Chronic sinusitis</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAH8%0DDNAH5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DNAH5</a> and phenotypic similarity 0.643 to mouse mutant of DNAH5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002639, micrognathia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0002639, micrognathia</dd><dd>HP:0000601, Hypotelorism - MP:0002639, micrognathia</dd><dd>HP:0000244, Brachyturricephaly - MP:0002639, micrognathia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619095">OMIM:619095</a> Spermatogenic failure 46 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.016</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.254</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.708</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-38810663-A-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.708</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359357.3">DNAH8:ENST00000359357.3:c.4178A&gt;C:p.(His1393Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441566.1">DNAH8:ENST00000441566.1:c.4178A&gt;C:p.(His1393Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449981.2">DNAH8:ENST00000449981.2:c.4829A&gt;C:p.(His1610Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7076186</dd>
                                    
                                    <dd>REVEL: 0.293</dd>
                                    <dd>MVP: 0.708</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.012</b>
                                    <data>(p=2.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.508</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.395</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-38810663-A-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.708</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359357.3">DNAH8:ENST00000359357.3:c.4178A&gt;C:p.(His1393Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441566.1">DNAH8:ENST00000441566.1:c.4178A&gt;C:p.(His1393Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449981.2">DNAH8:ENST00000449981.2:c.4829A&gt;C:p.(His1610Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7076186</dd>
                                    
                                    <dd>REVEL: 0.293</dd>
                                    <dd>MVP: 0.708</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-38690855-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs180723141">rs180723141</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=239906%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.083</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449981.2">DNAH8:ENST00000449981.2:c.270G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.2493%</dd>
                                    <dd>UK10K: 0.2645%</dd>
                                    <dd>ESP AA: 0.0571%</dd>
                                    <dd>ESP EA: 0.2762%</dd>
                                    <dd>ESP All: 0.2093%</dd>
                                    <dd>ExAC AFR: 0.0471%</dd>
                                    <dd>ExAC AMR: 0.2357%</dd>
                                    <dd>ExAC FIN: 0.1362%</dd>
                                    <dd>ExAC NFE: 0.3721%</dd>
                                    <dd>ExAC OTH: 0.8102%</dd>
                                    <dd>ExAC SAS: 0.1353%</dd>
                                    <dd>gnomAD_E_AFR: 0.0424%</dd>
                                    <dd>gnomAD_E_AMR: 0.2748%</dd>
                                    <dd>gnomAD_E_FIN: 0.1391%</dd>
                                    <dd>gnomAD_E_NFE: 0.3788%</dd>
                                    <dd>gnomAD_E_OTH: 0.3704%</dd>
                                    <dd>gnomAD_E_SAS: 0.1735%</dd>
                                    <dd>gnomAD_G_AFR: 0.0687%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.1433%</dd>
                                    <dd>gnomAD_G_NFE: 0.2732%</dd>
                                    <dd>gnomAD_G_OTH: 0.2037%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TNN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000120332">TNN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=2.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.607</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.305</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.607 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TNN">TNN</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000520, Proptosis - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000458, abnormal mandible morphology</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TNN%0DITGB6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ITGB6</a> and phenotypic similarity 0.328 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2850">Alopecia-intellectual disability syndrome</a> associated with ITGB6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TNN%0DITGB6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ITGB6</a> and phenotypic similarity 0.715 to mouse mutant of ITGB6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0002092, abnormal eye morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002092, abnormal eye morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=2.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.607</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.305</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-175049305-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs747487606">rs747487606</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.305</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000239462.4">TNN:ENST00000239462.4:c.791C&gt;A:p.(Thr264Asn)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.30953062</dd>
                                    
                                    <dd>REVEL: 0.059</dd>
                                    <dd>MVP: 0.310</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0268%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AMR: 0.0606%</dd>
                                    <dd>ExAC NFE: 0.0136%</dd>
                                    <dd>gnomAD_E_AMR: 0.0953%</dd>
                                    <dd>gnomAD_E_NFE: 0.0126%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALMS1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000116127">ALMS1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.015</b>
                    <data>(p=2.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.270</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.682</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.519 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64">Alström syndrome</a> associated with ALMS1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001831, Short toe</dd><dd>HP:0000486, Strabismus - HP:0000639, Nystagmus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0009804, Tooth agenesis</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0004438, Hyperostosis frontalis interna</dd></dl><dl><dt>Phenotypic similarity 0.540 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ALMS1">ALMS1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005253, abnormal eye physiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005253, abnormal eye physiology</dd><dd>HP:0000520, Proptosis - MP:0005253, abnormal eye physiology</dd><dd>HP:0000601, Hypotelorism - MP:0005253, abnormal eye physiology</dd><dd>HP:0000244, Brachyturricephaly - MP:0005253, abnormal eye physiology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALMS1%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/entry/614114">Mosaic variegated aneuploidy syndrome 2</a> associated with CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004209, Clinodactyly of the 5th finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALMS1%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.314 to mouse mutant of CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003051, curly tail</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004610, small vertebrae</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/203800">OMIM:203800</a> Alstrom syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=64">ORPHA:64</a> Alström syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.015</b>
                                    <data>(p=2.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.270</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.682</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-73746975-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs779290832">rs779290832</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.682</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000264448.6">ALMS1:ENST00000264448.6:c.9610A&gt;G:p.(Thr3204Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409009.1">ALMS1:ENST00000409009.1:c.9484A&gt;G:p.(Thr3162Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.68238735</dd>
                                    
                                    <dd>REVEL: 0.137</dd>
                                    <dd>MVP: 0.682</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RBM15" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162775">RBM15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.012</b>
                    <data>(p=2.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.402</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBM15%0DANKRD17&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANKRD17</a> and phenotypic similarity 0.628 to <a href="http://www.omim.org/entry/619504">Chopra-Amiel-Gordon syndrome</a> associated with ANKRD17.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000201, Pierre-Robin sequence</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RBM15%0DANKRD17&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ANKRD17</a> and phenotypic similarity 0.222 to mouse mutant of ANKRD17.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0001915, intracranial hemorrhage</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.012</b>
                                    <data>(p=2.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.402</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-110882499-T-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200933258">rs200933258</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.402</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369784.3">RBM15:ENST00000369784.3:c.472T&gt;G:p.(Ser158Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000487146.2">RBM15:ENST00000487146.2:c.472T&gt;G:p.(Ser158Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000602849.1">RBM15:ENST00000602849.1:c.472T&gt;G:p.(Ser158Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.4050702</dd>
                                    
                                    <dd>REVEL: 0.013</dd>
                                    <dd>MVP: 0.405</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0261%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ExAC AFR: 0.0106%</dd>
                                    <dd>ExAC AMR: 0.0261%</dd>
                                    <dd>ExAC NFE: 0.0248%</dd>
                                    <dd>ExAC SAS: 0.0065%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.0300%</dd>
                                    <dd>gnomAD_E_NFE: 0.0300%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ERGIC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000113719">ERGIC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.011</b>
                    <data>(p=2.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.670</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.546 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1143">Neurogenic arthrogryposis multiplex congenita</a> associated with ERGIC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001357, Plagiocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ERGIC1%0DP4HB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to P4HB</a> and phenotypic similarity 0.696 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2050">Cole-Carpenter syndrome</a> associated with P4HB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly - HP:0000262, Turricephaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/208100">OMIM:208100</a> ?Arthrogryposis multiplex congenita 2, neurogenic type (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1143">ORPHA:1143</a> Neurogenic arthrogryposis multiplex congenita - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.011</b>
                                    <data>(p=2.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.670</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-172341813-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369010301">rs369010301</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.670</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000326654.2">ERGIC1:ENST00000326654.2:c.212G&gt;A:p.(Arg71His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393784.3">ERGIC1:ENST00000393784.3:c.347G&gt;A:p.(Arg116His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523291.1">ERGIC1:ENST00000523291.1:c.347G&gt;A:p.(Arg116His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.67209107</dd>
                                    
                                    <dd>REVEL: 0.555</dd>
                                    <dd>MVP: 0.672</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0019%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC FIN: 0.0152%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MND1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121211">MND1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=2.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.379</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MND1%0DRAD51C&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAD51C</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with RAD51C.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=2.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.379</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-154279631-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs951625471">rs951625471</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.379</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000240488.3">MND1:ENST00000240488.3:c.133A&gt;G:p.(Met45Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504860.1">MND1:ENST00000504860.1:c.88A&gt;G:p.(Met30Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.38</dd>
                                    
                                    <dd>REVEL: 0.380</dd>
                                    <dd>MVP: 0.374</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0019%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TSHZ2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182463">TSHZ2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=2.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.377</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.255 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=TSHZ2">TSHZ2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0003107, palatoquadrate cartilage malformed, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSHZ2%0DEYA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EYA1</a> and phenotypic similarity 0.379 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=52429">Branchiootic syndrome</a> associated with EYA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSHZ2%0DEYA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EYA1</a> and phenotypic similarity 0.718 to mouse mutant of EYA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0008148, abnormal sternocostal joint morphology</dd><dd>HP:0000486, Strabismus - MP:0001302, eyelids open at birth</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004539, absent maxilla</dd><dd>HP:0000520, Proptosis - MP:0001302, eyelids open at birth</dd><dd>HP:0000601, Hypotelorism - MP:0001302, eyelids open at birth</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TSHZ2%0DEYA1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to EYA1</a> and phenotypic similarity 0.365 to fish mutant of EYA1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001052, mandibular arch skeleton morphology, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0001049, head circular, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=2.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.377</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-51871502-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45479792">rs45479792</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.582</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329613.6">TSHZ2:ENST00000329613.6:c.1496G&gt;A:p.(Arg499Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371497.5">TSHZ2:ENST00000371497.5:c.1505G&gt;A:p.(Arg502Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603338.2">TSHZ2:ENST00000603338.2:c.1496G&gt;A:p.(Arg499Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7713376</dd>
                                    
                                    <dd>REVEL: 0.184</dd>
                                    <dd>MVP: 0.771</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2995%</dd>
                                    <dd>TOPMed: 0.6430%</dd>
                                    <dd>UK10K: 0.9786%</dd>
                                    <dd>ESP AA: 0.2043%</dd>
                                    <dd>ESP EA: 0.7558%</dd>
                                    <dd>ESP All: 0.5690%</dd>
                                    <dd>ExAC AFR: 0.1258%</dd>
                                    <dd>ExAC AMR: 0.3888%</dd>
                                    <dd>ExAC FIN: 0.3629%</dd>
                                    <dd>ExAC NFE: 1.0357%</dd>
                                    <dd>ExAC OTH: 0.6637%</dd>
                                    <dd>ExAC SAS: 0.1397%</dd>
                                    <dd>gnomAD_E_AFR: 0.1242%</dd>
                                    <dd>gnomAD_E_AMR: 0.4051%</dd>
                                    <dd>gnomAD_E_FIN: 0.4260%</dd>
                                    <dd>gnomAD_E_NFE: 0.9957%</dd>
                                    <dd>gnomAD_E_OTH: 0.6020%</dd>
                                    <dd>gnomAD_E_SAS: 0.1755%</dd>
                                    <dd>gnomAD_G_AFR: 0.1832%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_FIN: 0.5152%</dd>
                                    <dd>gnomAD_G_NFE: 0.8392%</dd>
                                    <dd>gnomAD_G_OTH: 1.0183%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-51870120-A-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs756735462">rs756735462</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.172</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329613.6">TSHZ2:ENST00000329613.6:c.114A&gt;T:p.(Gln38His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371497.5">TSHZ2:ENST00000371497.5:c.123A&gt;T:p.(Gln41His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603338.2">TSHZ2:ENST00000603338.2:c.114A&gt;T:p.(Gln38His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.172</dd>
                                    
                                    <dd>REVEL: 0.172</dd>
                                    <dd>MVP: 0.107</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=5.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.506</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.172</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-51870120-A-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs756735462">rs756735462</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.172</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000329613.6">TSHZ2:ENST00000329613.6:c.114A&gt;T:p.(Gln38His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371497.5">TSHZ2:ENST00000371497.5:c.123A&gt;T:p.(Gln41His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000603338.2">TSHZ2:ENST00000603338.2:c.114A&gt;T:p.(Gln38His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.172</dd>
                                    
                                    <dd>REVEL: 0.172</dd>
                                    <dd>MVP: 0.107</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DELEC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173077">DELEC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=3.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.383</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DELEC1%0DARNT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARNT</a> and phenotypic similarity 0.752 to mouse mutant of ARNT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000435, shortened head</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000435, shortened head</dd><dd>HP:0000520, Proptosis - MP:0000435, shortened head</dd><dd>HP:0000601, Hypotelorism - MP:0000435, shortened head</dd><dd>HP:0000244, Brachyturricephaly - MP:0000435, shortened head</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=3.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.383</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-118162670-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs753445101">rs753445101</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.383</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374016.1">DELEC1:ENST00000374016.1:c.46C&gt;A:p.(Pro16Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.38276124</dd>
                                    
                                    <dd>REVEL: 0.111</dd>
                                    <dd>MVP: 0.383</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC26A6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000225697">SLC26A6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.010</b>
                    <data>(p=3.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.379</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC26A6%0DMORC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MORC2</a> and phenotypic similarity 0.312 to <a href="http://www.omim.org/entry/619090">Developmental delay, impaired growth, dysmorphic facies, and axonal neuropathy</a> associated with MORC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001765, Hammertoe</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC26A6%0DMORC2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MORC2</a> and phenotypic similarity 0.635 to mouse mutant of MORC2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000565, oligodactyly</dd><dd>HP:0000486, Strabismus - MP:0004269, abnormal optic cup morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000454, abnormal jaw morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0005269, abnormal occipital bone morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.010</b>
                                    <data>(p=3.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.379</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-48669454-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1417081775">rs1417081775</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.379</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358747.6">SLC26A6:ENST00000358747.6:c.546C&gt;G:p.(Phe182Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383733.3">SLC26A6:ENST00000383733.3:c.609C&gt;G:p.(Phe203Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395550.2">SLC26A6:ENST00000395550.2:c.609C&gt;G:p.(Phe203Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420764.2">SLC26A6:ENST00000420764.2:c.609C&gt;G:p.(Phe203Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000455886.2">SLC26A6:ENST00000455886.2:c.505C&gt;G:p.(Arg169Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337000.8">SLC26A6:ENST00000337000.8:c.434-224C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3800523</dd>
                                    
                                    <dd>REVEL: 0.185</dd>
                                    <dd>MVP: 0.380</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KIF13A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137177">KIF13A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.009</b>
                    <data>(p=3.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.932</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.009</b>
                                    <data>(p=3.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.932</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-17794837-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201793412">rs201793412</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.932</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000259711.6">KIF13A:ENST00000259711.6:c.3041A&gt;G:p.(Asp1014Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378814.5">KIF13A:ENST00000378814.5:c.3041A&gt;G:p.(Asp1014Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378816.5">KIF13A:ENST00000378816.5:c.3041A&gt;G:p.(Asp1014Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378826.2">KIF13A:ENST00000378826.2:c.3041A&gt;G:p.(Asp1014Gly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378843.2">KIF13A:ENST00000378843.2:c.3041A&gt;G:p.(Asp1014Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.93632835</dd>
                                    
                                    <dd>REVEL: 0.523</dd>
                                    <dd>MVP: 0.936</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0102%</dd>
                                    <dd>ESP EA: 0.0242%</dd>
                                    <dd>ESP All: 0.0165%</dd>
                                    <dd>ExAC AMR: 0.0091%</dd>
                                    <dd>ExAC NFE: 0.0171%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0099%</dd>
                                    <dd>gnomAD_E_OTH: 0.0366%</dd>
                                    <dd>gnomAD_G_NFE: 0.0334%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LY75" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000054219">LY75</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b>
                    <data>(p=3.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.361</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LY75%0DPDCD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDCD1</a> and phenotypic similarity 0.893 to mouse mutant of PDCD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002750, exophthalmos</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0002750, exophthalmos</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.008</b>
                                    <data>(p=3.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.361</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-160706630-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145262865">rs145262865</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.361</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263636.4">LY75:ENST00000263636.4:c.3011C&gt;T:p.(Pro1004Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000553424.1">LY75:ENST00000553424.1:c.3011C&gt;T:p.(Pro1004Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554112.1">LY75:ENST00000554112.1:c.3011C&gt;T:p.(Pro1004Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3617588</dd>
                                    
                                    <dd>REVEL: 0.110</dd>
                                    <dd>MVP: 0.362</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0186%</dd>
                                    <dd>gnomAD_E_SAS: 0.0035%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="USP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000162607">USP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b>
                    <data>(p=3.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.354</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.473 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=USP1">USP1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0003671, abnormal eyelid aperture</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000443, abnormal snout morphology</dd><dd>HP:0000520, Proptosis - MP:0003671, abnormal eyelid aperture</dd><dd>HP:0000601, Hypotelorism - MP:0003671, abnormal eyelid aperture</dd><dd>HP:0000244, Brachyturricephaly - MP:0003671, abnormal eyelid aperture</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP1%0DFANCI&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FANCI</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=84">Fanconi anemia</a> associated with FANCI.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=USP1%0DFANCI&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FANCI</a> and phenotypic similarity 0.685 to mouse mutant of FANCI.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0010772, abnormal pollex morphology</dd><dd>HP:0000486, Strabismus - MP:0009772, abnormal retina development</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000120, malocclusion</dd><dd>HP:0000520, Proptosis - MP:0001297, microphthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001297, microphthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0009772, abnormal retina development</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.008</b>
                                    <data>(p=3.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.504</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.354</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-62910770-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs536162897">rs536162897</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.354</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339950.4">USP1:ENST00000339950.4:c.919A&gt;G:p.(Thr307Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371146.1">USP1:ENST00000371146.1:c.919A&gt;G:p.(Thr307Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.35507384</dd>
                                    
                                    <dd>REVEL: 0.066</dd>
                                    <dd>MVP: 0.355</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF335" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198026">ZNF335</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.008</b>
                    <data>(p=3.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.251</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.636</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.408 to <a href="http://www.omim.org/entry/615095">Microcephaly 10, primary, autosomal recessive</a> associated with ZNF335.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0011451, Primary microcephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF335%0DNCOA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCOA6</a> and phenotypic similarity 0.672 to mouse mutant of NCOA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0001324, abnormal eye pigmentation</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009887, abnormal palatal shelf fusion at midline</dd><dd>HP:0000520, Proptosis - MP:0001324, abnormal eye pigmentation</dd><dd>HP:0000601, Hypotelorism - MP:0001324, abnormal eye pigmentation</dd><dd>HP:0000244, Brachyturricephaly - MP:0001324, abnormal eye pigmentation</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/615095">OMIM:615095</a> Microcephaly 10, primary, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=329228">ORPHA:329228</a> Microcephalic primordial dwarfism due to ZNF335 deficiency - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.008</b>
                                    <data>(p=3.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.636</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>20-44579206-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765035424">rs765035424</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=860640%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.636</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322927.2">ZNF335:ENST00000322927.2:c.3218G&gt;A:p.(Arg1073Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426788.1">ZNF335:ENST00000426788.1:c.2753G&gt;A:p.(Arg918Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6360797</dd>
                                    
                                    <dd>REVEL: 0.306</dd>
                                    <dd>MVP: 0.636</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="BICD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000151746">BICD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.344</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BICD1%0DMMP14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP14</a> and phenotypic similarity 0.484 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=371428">Multicentric osteolysis-nodulosis-arthropathy spectrum</a> associated with MMP14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BICD1%0DMMP14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP14</a> and phenotypic similarity 0.909 to mouse mutant of MMP14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0030245, round head</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=BICD1%0DMMP14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MMP14</a> and phenotypic similarity 0.226 to fish mutant of MMP14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0000126, Meckel's cartilage malformed, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.344</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-32490627-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200867915">rs200867915</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.344</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000281474.5">BICD1:ENST00000281474.5:c.2447T&gt;C:p.(Ile816Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000548411.1">BICD1:ENST00000548411.1:c.2447T&gt;C:p.(Ile816Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.345405</dd>
                                    
                                    <dd>REVEL: 0.076</dd>
                                    <dd>MVP: 0.345</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NIN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000100503">NIN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.674</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.145</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.674 to <a href="http://www.omim.org/entry/614851">?Seckel syndrome 7</a> associated with NIN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NIN%0DCEP135&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP135</a> and phenotypic similarity 0.347 to <a href="http://www.omim.org/entry/614673">Microcephaly 8, primary, autosomal recessive</a> associated with CEP135.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000278, Retrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NIN%0DCEP135&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP135</a> and phenotypic similarity 0.690 to mouse mutant of CEP135.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000562, polydactyly</dd><dd>HP:0000486, Strabismus - MP:0001293, anophthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001293, anophthalmia</dd><dd>HP:0000520, Proptosis - MP:0001293, anophthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001293, anophthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0011495, abnormal head shape</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/614851">OMIM:614851</a> ?Seckel syndrome 7 (unconfirmed)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=319675">ORPHA:319675</a> Microcephalic primordial dwarfism, Dauber type - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.674</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.145</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.674 to OMIM:614851 ?Seckel syndrome 7</dt>
                                
                                    <dt>Phenotypic similarity 0.656 to ORPHA:319675 Microcephalic primordial dwarfism, Dauber type</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-51226983-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147946706">rs147946706</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=429579%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.191</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245441.5">NIN:ENST00000245441.5:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324330.9">NIN:ENST00000324330.9:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382041.3">NIN:ENST00000382041.3:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382043.4">NIN:ENST00000382043.4:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389868.3">NIN:ENST00000389868.3:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453196.1">NIN:ENST00000453196.1:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530997.2">NIN:ENST00000530997.2:c.1991C&gt;T:p.(Thr664Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.19481854</dd>
                                    
                                    <dd>REVEL: 0.034</dd>
                                    <dd>MVP: 0.195</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0242%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AMR: 0.0518%</dd>
                                    <dd>ExAC NFE: 0.0375%</dd>
                                    <dd>ExAC OTH: 0.1109%</dd>
                                    <dd>gnomAD_E_AMR: 0.0447%</dd>
                                    <dd>gnomAD_E_NFE: 0.0108%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-51259463-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146441468">rs146441468</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=725682%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245441.5">NIN:ENST00000245441.5:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324330.9">NIN:ENST00000324330.9:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382041.3">NIN:ENST00000382041.3:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382043.4">NIN:ENST00000382043.4:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389868.3">NIN:ENST00000389868.3:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453196.1">NIN:ENST00000453196.1:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530997.2">NIN:ENST00000530997.2:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0159%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0360%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0349%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>14-51259463-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146441468">rs146441468</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=725682%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000245441.5">NIN:ENST00000245441.5:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324330.9">NIN:ENST00000324330.9:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382041.3">NIN:ENST00000382041.3:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382043.4">NIN:ENST00000382043.4:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389868.3">NIN:ENST00000389868.3:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453196.1">NIN:ENST00000453196.1:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530997.2">NIN:ENST00000530997.2:c.402G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0159%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0360%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0349%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AXIN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000103126">AXIN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.368</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.488</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.735 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=AXIN1">AXIN1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000435, shortened head</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000435, shortened head</dd><dd>HP:0000520, Proptosis - MP:0000435, shortened head</dd><dd>HP:0000601, Hypotelorism - MP:0000435, shortened head</dd><dd>HP:0000244, Brachyturricephaly - MP:0000435, shortened head</dd></dl><dl><dt>Phenotypic similarity 0.191 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=AXIN1">AXIN1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000354, telencephalon aplastic, abnormal</dd></dl><dl><dt>Proximity score 0.522 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AXIN1%0DXPNPEP1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XPNPEP1</a> and phenotypic similarity 0.658 to mouse mutant of XPNPEP1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000433, microcephaly</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000433, microcephaly</dd><dd>HP:0000520, Proptosis - MP:0000433, microcephaly</dd><dd>HP:0000601, Hypotelorism - MP:0000433, microcephaly</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/114550">OMIM:114550</a> Hepatocellular carcinoma, somatic - somatic</dd>
<dd><a href="http://www.omim.org/entry/607864">OMIM:607864</a> ?Caudal duplication anomaly (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.368</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.488</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-341206-C-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.488</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262320.3">AXIN1:ENST00000262320.3:c.2278G&gt;C:p.(Glu760Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354866.3">AXIN1:ENST00000354866.3:c.2187-1599G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48770186</dd>
                                    
                                    <dd>REVEL: 0.091</dd>
                                    <dd>MVP: 0.488</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CDCA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000184661">CDCA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.337</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDCA2%0DNSUN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NSUN2</a> and phenotypic similarity 0.702 to <a href="http://www.omim.org/entry/611091">Intellectual developmental disorder, autosomal recessive 5</a> associated with NSUN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004692, 4-5 toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000331, Short chin</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CDCA2%0DNSUN2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NSUN2</a> and phenotypic similarity 0.590 to mouse mutant of NSUN2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0005296, abnormal humerus morphology</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000438, abnormal cranium morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.337</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-25365148-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1463113093">rs1463113093</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.337</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330560.3">CDCA2:ENST00000330560.3:c.2966C&gt;T:p.(Thr989Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380665.3">CDCA2:ENST00000380665.3:c.2921C&gt;T:p.(Thr974Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3371357</dd>
                                    
                                    <dd>REVEL: 0.040</dd>
                                    <dd>MVP: 0.337</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC39A14" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104635">SLC39A14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.007</b>
                    <data>(p=3.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.626</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.194</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.626 to <a href="http://www.omim.org/entry/144755">?Hyperostosis cranalis interna</a> associated with SLC39A14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0005746, Osteosclerosis of the base of the skull</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly - HP:0004490, Calvarial hyperostosis</dd></dl><dl><dt>Phenotypic similarity 0.337 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SLC39A14">SLC39A14</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0004686, decreased length of long bones</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003109, short femur</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A14%0DSLC39A13&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC39A13</a> and phenotypic similarity 0.592 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=157965">SLC39A13-related spondylodysplastic Ehlers-Danlos syndrome</a> associated with SLC39A13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001182, Tapered finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0100864, Short femoral neck</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC39A14%0DSLC39A13&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SLC39A13</a> and phenotypic similarity 0.797 to mouse mutant of SLC39A13.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000133, abnormal long bone metaphysis morphology</dd><dd>HP:0000486, Strabismus - MP:0006198, enophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004540, small maxilla</dd><dd>HP:0000520, Proptosis - MP:0006198, enophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0006198, enophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0000454, abnormal jaw morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/144755">OMIM:144755</a> ?Hyperostosis cranalis interna (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/617013">OMIM:617013</a> Hypermanganesemia with dystonia 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=521406">ORPHA:521406</a> Dystonia-parkinsonism-hypermanganesemia syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.007</b>
                                    <data>(p=3.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.626</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.194</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.626 to OMIM:144755 ?Hyperostosis cranalis interna</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-22267473-T-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs79856562">rs79856562</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4, BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1978998%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.194</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359741.5">SLC39A14:ENST00000359741.5:c.472T&gt;G:p.(Phe158Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000240095.6">SLC39A14:ENST00000240095.6:c.457+1464T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000289952.5">SLC39A14:ENST00000289952.5:c.457+1464T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381237.1">SLC39A14:ENST00000381237.1:c.457+1464T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.1938658</dd>
                                    
                                    <dd>REVEL: 0.105</dd>
                                    <dd>MVP: 0.194</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0032%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZFYVE16" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000039319">ZFYVE16</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b>
                    <data>(p=3.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.522</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.306</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.522 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZFYVE16%0DTOM1L2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TOM1L2</a> and phenotypic similarity 0.881 to mouse mutant of TOM1L2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001299, abnormal eye distance/ position</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0001299, abnormal eye distance/ position</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.006</b>
                                    <data>(p=3.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.522</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.306</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-79739010-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs893040457">rs893040457</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.306</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338008.5">ZFYVE16:ENST00000338008.5:c.2488A&gt;G:p.(Thr830Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505560.1">ZFYVE16:ENST00000505560.1:c.2488A&gt;G:p.(Thr830Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000510158.1">ZFYVE16:ENST00000510158.1:c.2488A&gt;G:p.(Thr830Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.30605322</dd>
                                    
                                    <dd>REVEL: 0.061</dd>
                                    <dd>MVP: 0.306</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0049%</dd>
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_SAS: 0.0034%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UACA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137831">UACA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b>
                    <data>(p=3.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.331</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UACA%0DNLRP3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NLRP3</a> and phenotypic similarity 0.645 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1451">CINCA syndrome</a> associated with NLRP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0002007, Frontal bossing</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UACA%0DNLRP3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NLRP3</a> and phenotypic similarity 0.377 to mouse mutant of NLRP3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0030825, decreased femur size</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.006</b>
                                    <data>(p=3.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.331</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-70949424-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143220817">rs143220817</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2205260%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.331</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322954.6">UACA:ENST00000322954.6:c.4222A&gt;G:p.(Ile1408Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379983.2">UACA:ENST00000379983.2:c.4183A&gt;G:p.(Ile1395Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539319.1">UACA:ENST00000539319.1:c.3895A&gt;G:p.(Ile1299Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000560441.1">UACA:ENST00000560441.1:c.4177A&gt;G:p.(Ile1393Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.33161932</dd>
                                    
                                    <dd>REVEL: 0.126</dd>
                                    <dd>MVP: 0.332</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0072%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC NFE: 0.0150%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MAG" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105695">MAG</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b>
                    <data>(p=3.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.323</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.250 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=459056">Autosomal recessive spastic paraplegia type 75</a> associated with MAG.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000639, Nystagmus</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.185 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=MAG">MAG</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000953, abnormal oligodendrocyte morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAG%0DCHST14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHST14</a> and phenotypic similarity 0.654 to <a href="http://www.omim.org/entry/601776">Ehlers-Danlos syndrome, musculocontractural type 1</a> associated with CHST14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001181, Adducted thumb</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000308, Microretrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MAG%0DCHST14&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CHST14</a> and phenotypic similarity 0.528 to mouse mutant of CHST14.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000558, abnormal tibia morphology</dd><dd>HP:0000486, Strabismus - MP:0000116, abnormal tooth development</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000116, abnormal tooth development</dd><dd>HP:0000520, Proptosis - MP:0000116, abnormal tooth development</dd><dd>HP:0000601, Hypotelorism - MP:0000116, abnormal tooth development</dd><dd>HP:0000244, Brachyturricephaly - MP:0000116, abnormal tooth development</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616680">OMIM:616680</a> Spastic paraplegia 75, autosomal recessive - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=459056">ORPHA:459056</a> Autosomal recessive spastic paraplegia type 75 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.006</b>
                                    <data>(p=3.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.323</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.250 to ORPHA:459056 Autosomal recessive spastic paraplegia type 75</dt>
                                
                                    <dt>Phenotypic similarity 0.250 to OMIM:616680 Spastic paraplegia 75, autosomal recessive</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-35790493-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144553163">rs144553163</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=411594%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.547</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361922.4">MAG:ENST00000361922.4:c.452C&gt;T:p.(Ala151Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392213.3">MAG:ENST00000392213.3:c.452C&gt;T:p.(Ala151Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537831.2">MAG:ENST00000537831.2:c.377C&gt;T:p.(Ala126Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597035.1">MAG:ENST00000597035.1:c.*36C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.64536357</dd>
                                    
                                    <dd>REVEL: 0.370</dd>
                                    <dd>MVP: 0.645</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2396%</dd>
                                    <dd>TOPMed: 0.3982%</dd>
                                    <dd>UK10K: 0.3042%</dd>
                                    <dd>ESP AA: 0.2497%</dd>
                                    <dd>ESP EA: 0.4302%</dd>
                                    <dd>ESP All: 0.3691%</dd>
                                    <dd>ExAC AFR: 0.1595%</dd>
                                    <dd>ExAC AMR: 0.2183%</dd>
                                    <dd>ExAC EAS: 0.0120%</dd>
                                    <dd>ExAC FIN: 0.1100%</dd>
                                    <dd>ExAC NFE: 0.5755%</dd>
                                    <dd>ExAC OTH: 0.6112%</dd>
                                    <dd>ExAC SAS: 0.5999%</dd>
                                    <dd>gnomAD_E_AFR: 0.1133%</dd>
                                    <dd>gnomAD_E_AMR: 0.1973%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.0685%</dd>
                                    <dd>gnomAD_E_NFE: 0.5923%</dd>
                                    <dd>gnomAD_E_OTH: 0.5425%</dd>
                                    <dd>gnomAD_E_SAS: 0.4572%</dd>
                                    <dd>gnomAD_G_AFR: 0.1609%</dd>
                                    <dd>gnomAD_G_AMR: 0.3589%</dd>
                                    <dd>gnomAD_G_FIN: 0.1146%</dd>
                                    <dd>gnomAD_G_NFE: 0.7555%</dd>
                                    <dd>gnomAD_G_OTH: 0.1025%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-35786682-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199750254">rs199750254</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361922.4">MAG:ENST00000361922.4:c.213G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392213.3">MAG:ENST00000392213.3:c.213G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537831.2">MAG:ENST00000537831.2:c.138G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597035.1">MAG:ENST00000597035.1:c.176+37G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0019%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-35786682-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199750254">rs199750254</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361922.4">MAG:ENST00000361922.4:c.213G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392213.3">MAG:ENST00000392213.3:c.213G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537831.2">MAG:ENST00000537831.2:c.138G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000597035.1">MAG:ENST00000597035.1:c.176+37G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0019%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TFB1M" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000029639">TFB1M</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.006</b>
                    <data>(p=3.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.515</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.303</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.392 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TFB1M">TFB1M</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0011389, absent optic disk</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0011389, absent optic disk</dd><dd>HP:0000520, Proptosis - MP:0011389, absent optic disk</dd><dd>HP:0000601, Hypotelorism - MP:0011389, absent optic disk</dd><dd>HP:0000244, Brachyturricephaly - MP:0011389, absent optic disk</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TFB1M%0DPOLRMT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLRMT</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/entry/619743">Combined oxidative phosphorylation deficiency 55</a> associated with POLRMT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000577, Exotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000218, High palate</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.515 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TFB1M%0DPOLRMT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLRMT</a> and phenotypic similarity 0.213 to mouse mutant of POLRMT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0002152, abnormal brain morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.006</b>
                                    <data>(p=3.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.515</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.303</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-155606327-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs769497533">rs769497533</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.303</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367166.4">TFB1M:ENST00000367166.4:c.631A&gt;G:p.(Thr211Ala)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.30378184</dd>
                                    
                                    <dd>REVEL: 0.141</dd>
                                    <dd>MVP: 0.304</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0057%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC EAS: 0.0231%</dd>
                                    <dd>ExAC NFE: 0.0120%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_NFE: 0.0125%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MDN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000112159">MDN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.311</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.362 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=MDN1">MDN1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001708, mandibular arch skeleton malformed, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MDN1%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.623 to <a href="http://www.omim.org/entry/616462">Acrofacial dysostosis, Cincinnati type</a> associated with POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MDN1%0DPOLR1A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POLR1A</a> and phenotypic similarity 0.403 to fish mutant of POLR1A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0017556, mandibular arch skeleton agenesis, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.311</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-90368431-T-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139030773">rs139030773</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.311</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369393.3">MDN1:ENST00000369393.3:c.14919A&gt;T:p.(Glu4973Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428876.1">MDN1:ENST00000428876.1:c.14919A&gt;T:p.(Glu4973Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.31252968</dd>
                                    
                                    <dd>REVEL: 0.067</dd>
                                    <dd>MVP: 0.313</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0057%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ARVCF" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099889">ARVCF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.005</b>
                    <data>(p=3.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.680</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.680 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=567">22q11.2 deletion syndrome</a> associated with ARVCF.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001829, Foot polydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000262, Turricephaly</dd></dl><dl><dt>Phenotypic similarity 0.491 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ARVCF">ARVCF</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARVCF%0DCOMT&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COMT</a> and phenotypic similarity 0.680 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=567">22q11.2 deletion syndrome</a> associated with COMT.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001829, Foot polydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000262, Turricephaly</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=567">ORPHA:567</a> 22q11.2 deletion syndrome (CNV)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.005</b>
                                    <data>(p=3.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.680</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.680 to ORPHA:567 22q11.2 deletion syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>22-19967453-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs775954994">rs775954994</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263207.3">ARVCF:ENST00000263207.3:c.1209G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344269.3">ARVCF:ENST00000344269.3:c.1020G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401994.1">ARVCF:ENST00000401994.1:c.1020G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406259.1">ARVCF:ENST00000406259.1:c.1209G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406522.1">ARVCF:ENST00000406522.1:c.1020G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ExAC AFR: 0.0214%</dd>
                                    <dd>ExAC EAS: 0.0463%</dd>
                                    <dd>ExAC NFE: 0.0032%</dd>
                                    <dd>gnomAD_E_AFR: 0.0266%</dd>
                                    <dd>gnomAD_E_AMR: 0.0066%</dd>
                                    <dd>gnomAD_E_EAS: 0.0270%</dd>
                                    <dd>gnomAD_E_NFE: 0.0012%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GPRIN2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000204175">GPRIN2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_INSERTION</span>
                            <b>INS</b>
                            <b>10-46999604-A-ATGGAGGTGG [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374314.4">GPRIN2:ENST00000374314.4:c.724_725insTGGAGGTGG:p.(Arg242delinsMetGluValGly)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374317.1">GPRIN2:ENST00000374317.1:c.724_725insTGGAGGTGG:p.(Arg242delinsMetGluValGly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KRTAP4-1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198443">KRTAP4-1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.850</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.850</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">DISRUPTIVE_INFRAME_DELETION</span>
                            <b>DEL</b>
                            <b>17-39340795-ACGGCAGCAGCTGGACATACCACAGCTGGGGTGGCAGGTGGTCTGACAGCAGAGTGGG-A [1/1]</b>
                            
                            
                                
                                    
                                    <a href="https://www.ncbi.nlm.nih.gov/dbvar/variants/nsv5589137">nsv5589137</a>
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.850</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398472.1">KRTAP4-1:ENST00000398472.1:c.255_311del:p.(Pro86_Arg104del)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NEFM" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000104722">NEFM</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.273</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.169 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NEFM">NEFM</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000937, abnormal motor neuron morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NEFM%0DRAC3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAC3</a> and phenotypic similarity 0.667 to <a href="http://www.omim.org/entry/618577">Neurodevelopmental disorder with structural brain anomalies and dysmorphic facies</a> associated with RAC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.273</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-24774812-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61751047">rs61751047</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.459</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221166.5">NEFM:ENST00000221166.5:c.1444G&gt;A:p.(Val482Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000433454.2">NEFM:ENST00000433454.2:c.316G&gt;A:p.(Val106Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437366.2">NEFM:ENST00000437366.2:c.1444G&gt;A:p.(Val482Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518131.1">NEFM:ENST00000518131.1:c.1444G&gt;A:p.(Val482Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.49014416</dd>
                                    
                                    <dd>REVEL: 0.218</dd>
                                    <dd>MVP: 0.490</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.1410%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP AA: 0.3404%</dd>
                                    <dd>ESP EA: 0.0465%</dd>
                                    <dd>ESP All: 0.1461%</dd>
                                    <dd>ExAC AFR: 0.3067%</dd>
                                    <dd>ExAC AMR: 0.0882%</dd>
                                    <dd>ExAC EAS: 0.0357%</dd>
                                    <dd>ExAC NFE: 0.0382%</dd>
                                    <dd>ExAC SAS: 0.0247%</dd>
                                    <dd>gnomAD_E_AFR: 0.3536%</dd>
                                    <dd>gnomAD_E_AMR: 0.1401%</dd>
                                    <dd>gnomAD_E_EAS: 0.0406%</dd>
                                    <dd>gnomAD_E_NFE: 0.0405%</dd>
                                    <dd>gnomAD_E_OTH: 0.1462%</dd>
                                    <dd>gnomAD_E_SAS: 0.0163%</dd>
                                    <dd>gnomAD_G_AFR: 0.3894%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_EAS: 0.0617%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-24771984-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs61735463">rs61735463</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=723022%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.087</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221166.5">NEFM:ENST00000221166.5:c.678G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000437366.2">NEFM:ENST00000437366.2:c.678G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000518131.1">NEFM:ENST00000518131.1:c.678G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1997%</dd>
                                    <dd>TOPMed: 0.2150%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.6582%</dd>
                                    <dd>ESP EA: 0.0465%</dd>
                                    <dd>ESP All: 0.2537%</dd>
                                    <dd>ExAC AFR: 0.5403%</dd>
                                    <dd>ExAC AMR: 0.1047%</dd>
                                    <dd>ExAC NFE: 0.0380%</dd>
                                    <dd>gnomAD_E_AFR: 0.5899%</dd>
                                    <dd>gnomAD_E_AMR: 0.1490%</dd>
                                    <dd>gnomAD_E_NFE: 0.0404%</dd>
                                    <dd>gnomAD_E_OTH: 0.1460%</dd>
                                    <dd>gnomAD_G_AFR: 0.5157%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="POLR2A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181222">POLR2A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.652</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.652 to <a href="http://www.omim.org/entry/618603">Neurodevelopmental disorder with hypotonia and variable intellectual and behavioral abnormalities</a> associated with POLR2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000692, Tooth malposition</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618603">OMIM:618603</a> Neurodevelopmental disorder with hypotonia and variable intellectual and behavioral abnormalities - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.652</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.652 to OMIM:618603 Neurodevelopmental disorder with hypotonia and variable intellectual and behavioral abnormalities</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-7406476-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs770126817">rs770126817</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000322644.6">POLR2A:ENST00000322644.6:c.2793G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0038%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KLHL28" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179454">KLHL28</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.004</b>
                    <data>(p=4.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.835</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.004</b>
                                    <data>(p=4.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.835</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-45400737-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs761233044">rs761233044</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2220439%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.835</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355081.2">KLHL28:ENST00000355081.2:c.1393C&gt;T:p.(Arg465Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396128.4">KLHL28:ENST00000396128.4:c.1351C&gt;T:p.(Arg451Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8357329</dd>
                                    
                                    <dd>REVEL: 0.620</dd>
                                    <dd>MVP: 0.836</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FRMPD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000070601">FRMPD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.830</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.830</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-37711382-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140966026">rs140966026</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.830</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377765.3">FRMPD1:ENST00000377765.3:c.398G&gt;C:p.(Arg133Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539465.1">FRMPD1:ENST00000539465.1:c.398G&gt;C:p.(Arg133Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540557.1">FRMPD1:ENST00000540557.1:c.*1011+16543C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.829944</dd>
                                    
                                    <dd>REVEL: 0.352</dd>
                                    <dd>MVP: 0.830</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF462" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000148143">ZNF462</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.578</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.171</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.296 to <a href="http://www.omim.org/entry/618619">Weiss-Kruszka syndrome</a> associated with ZNF462.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0009623, Proximal placement of thumb</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0002263, Exaggerated cupid's bow</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.474 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZNF462">ZNF462</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001302, eyelids open at birth</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001302, eyelids open at birth</dd><dd>HP:0000520, Proptosis - MP:0001302, eyelids open at birth</dd><dd>HP:0000601, Hypotelorism - MP:0001302, eyelids open at birth</dd><dd>HP:0000244, Brachyturricephaly - MP:0001302, eyelids open at birth</dd></dl><dl><dt>Proximity score 0.578 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF462%0DVAX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VAX1</a> and phenotypic similarity 0.217 to <a href="http://www.omim.org/entry/614402">?Microphthalmia, syndromic 11</a> associated with VAX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000175, Cleft palate</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.578 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF462%0DVAX1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to VAX1</a> and phenotypic similarity 0.684 to mouse mutant of VAX1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004448, abnormal presphenoid bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000455, abnormal maxilla morphology</dd><dd>HP:0000520, Proptosis - MP:0004448, abnormal presphenoid bone morphology</dd><dd>HP:0000601, Hypotelorism - MP:0004448, abnormal presphenoid bone morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0005352, small cranium</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618619">OMIM:618619</a> Weiss-Kruszka syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.578</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.171</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.296 to OMIM:618619 Weiss-Kruszka syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-109692959-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs549946790">rs549946790</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.171</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000277225.5">ZNF462:ENST00000277225.5:c.6001G&gt;C:p.(Ala2001Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441147.2">ZNF462:ENST00000441147.2:c.2536G&gt;C:p.(Ala846Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457913.1">ZNF462:ENST00000457913.1:c.6001G&gt;C:p.(Ala2001Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.17122021</dd>
                                    
                                    <dd>REVEL: 0.150</dd>
                                    <dd>MVP: 0.171</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0170%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.289</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.115</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-109692959-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs549946790">rs549946790</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.171</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000277225.5">ZNF462:ENST00000277225.5:c.6001G&gt;C:p.(Ala2001Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000441147.2">ZNF462:ENST00000441147.2:c.2536G&gt;C:p.(Ala846Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457913.1">ZNF462:ENST00000457913.1:c.6001G&gt;C:p.(Ala2001Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.17122021</dd>
                                    
                                    <dd>REVEL: 0.150</dd>
                                    <dd>MVP: 0.171</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0170%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-109688482-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs72741799">rs72741799</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1710108%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.058</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000277225.5">ZNF462:ENST00000277225.5:c.2289A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457913.1">ZNF462:ENST00000457913.1:c.2289A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5791%</dd>
                                    <dd>TOPMed: 0.7681%</dd>
                                    <dd>UK10K: 1.2827%</dd>
                                    <dd>ESP AA: 0.2043%</dd>
                                    <dd>ESP EA: 1.1047%</dd>
                                    <dd>ESP All: 0.7996%</dd>
                                    <dd>ExAC AFR: 0.1827%</dd>
                                    <dd>ExAC AMR: 0.6651%</dd>
                                    <dd>ExAC FIN: 1.1340%</dd>
                                    <dd>ExAC NFE: 1.1553%</dd>
                                    <dd>ExAC OTH: 0.6608%</dd>
                                    <dd>ExAC SAS: 0.0727%</dd>
                                    <dd>gnomAD_E_AFR: 0.1568%</dd>
                                    <dd>gnomAD_E_AMR: 0.6670%</dd>
                                    <dd>gnomAD_E_FIN: 1.2825%</dd>
                                    <dd>gnomAD_E_NFE: 1.1153%</dd>
                                    <dd>gnomAD_E_OTH: 0.6562%</dd>
                                    <dd>gnomAD_E_SAS: 0.1364%</dd>
                                    <dd>gnomAD_G_AFR: 0.2524%</dd>
                                    <dd>gnomAD_G_AMR: 0.7160%</dd>
                                    <dd>gnomAD_G_FIN: 1.4040%</dd>
                                    <dd>gnomAD_G_NFE: 1.1397%</dd>
                                    <dd>gnomAD_G_OTH: 0.6110%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LHFPL3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187416">LHFPL3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.253</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LHFPL3%0DTNN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TNN</a> and phenotypic similarity 0.607 to mouse mutant of TNN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000520, Proptosis - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000458, abnormal mandible morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000458, abnormal mandible morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.253</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>7-103969537-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs190167677">rs190167677</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=699723%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.423</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401970.2">LHFPL3:ENST00000401970.2:c.268G&gt;T:p.(Ala90Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424859.1">LHFPL3:ENST00000424859.1:c.268G&gt;T:p.(Ala90Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535008.1">LHFPL3:ENST00000535008.1:c.310G&gt;T:p.(Ala104Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543266.1">LHFPL3:ENST00000543266.1:c.310G&gt;T:p.(Ala104Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.50326246</dd>
                                    
                                    <dd>REVEL: 0.446</dd>
                                    <dd>MVP: 0.503</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0774%</dd>
                                    <dd>UK10K: 0.0926%</dd>
                                    <dd>ESP EA: 0.0949%</dd>
                                    <dd>ESP All: 0.0641%</dd>
                                    <dd>ExAC AFR: 0.0204%</dd>
                                    <dd>ExAC AMR: 0.2938%</dd>
                                    <dd>ExAC NFE: 0.0795%</dd>
                                    <dd>ExAC OTH: 0.7812%</dd>
                                    <dd>ExAC SAS: 0.0545%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.1697%</dd>
                                    <dd>gnomAD_E_NFE: 0.0904%</dd>
                                    <dd>gnomAD_E_OTH: 0.2191%</dd>
                                    <dd>gnomAD_E_SAS: 0.0487%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0734%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>7-103969525-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs185070516">rs185070516</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=699722%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.084</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401970.2">LHFPL3:ENST00000401970.2:c.256C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424859.1">LHFPL3:ENST00000424859.1:c.256C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000535008.1">LHFPL3:ENST00000535008.1:c.298C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543266.1">LHFPL3:ENST00000543266.1:c.298C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0778%</dd>
                                    <dd>UK10K: 0.0793%</dd>
                                    <dd>ESP EA: 0.0952%</dd>
                                    <dd>ESP All: 0.0643%</dd>
                                    <dd>ExAC AFR: 0.0204%</dd>
                                    <dd>ExAC AMR: 0.2938%</dd>
                                    <dd>ExAC NFE: 0.0840%</dd>
                                    <dd>ExAC OTH: 0.7812%</dd>
                                    <dd>ExAC SAS: 0.0545%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.1697%</dd>
                                    <dd>gnomAD_E_NFE: 0.0940%</dd>
                                    <dd>gnomAD_E_OTH: 0.2372%</dd>
                                    <dd>gnomAD_E_SAS: 0.0487%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0734%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FBXO33" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000165355">FBXO33</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.246</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FBXO33%0DGUSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GUSB</a> and phenotypic similarity 0.343 to <a href="http://www.omim.org/entry/253220">Mucopolysaccharidosis VII</a> associated with GUSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000687, Widely spaced teeth</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FBXO33%0DGUSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GUSB</a> and phenotypic similarity 0.791 to mouse mutant of GUSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000547, short limbs</dd><dd>HP:0000486, Strabismus - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000520, Proptosis - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000074, abnormal neurocranium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000435, shortened head</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.246</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-39868794-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs776602432">rs776602432</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.246</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000298097.7">FBXO33:ENST00000298097.7:c.1594G&gt;A:p.(Glu532Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554190.1">FBXO33:ENST00000554190.1:c.*143G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.247</dd>
                                    
                                    <dd>REVEL: 0.247</dd>
                                    <dd>MVP: 0.093</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0042%</dd>
                                    <dd>ExAC AMR: 0.0186%</dd>
                                    <dd>ExAC NFE: 0.0032%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NCAPH2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000025770">NCAPH2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.650</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.067</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.650 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NCAPH2">NCAPH2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000433, microcephaly</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000433, microcephaly</dd><dd>HP:0000520, Proptosis - MP:0000433, microcephaly</dd><dd>HP:0000601, Hypotelorism - MP:0000433, microcephaly</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.650</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.067</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-50956675-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139403039">rs139403039</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.067</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000299821.11">NCAPH2:ENST00000299821.11:c.614C&gt;T:p.(Ala205Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395698.3">NCAPH2:ENST00000395698.3:c.614C&gt;T:p.(Ala205Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395701.3">NCAPH2:ENST00000395701.3:c.614C&gt;T:p.(Ala205Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000420993.2">NCAPH2:ENST00000420993.2:c.614C&gt;T:p.(Ala205Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.06752429</dd>
                                    
                                    <dd>REVEL: 0.043</dd>
                                    <dd>MVP: 0.068</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0117%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0393%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>ExAC FIN: 0.0155%</dd>
                                    <dd>ExAC NFE: 0.0046%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_EAS: 0.0174%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MST1P2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186301">MST1P2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>1-16972920-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs4661882">rs4661882</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000334429.5">MST1P2:ENST00000334429.5:n.158C&gt;T:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418421.2">MST1P2:ENST00000418421.2:n.123C&gt;T:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000457982.3">MST1P2:ENST00000457982.3:n.114C&gt;T:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RP11-764K9.4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000215548">RP11-764K9.4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>9-68433462-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs111392331">rs111392331</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000376334.3">RP11-764K9.4:ENST00000376334.3:n.695+3A&gt;G:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF285CP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000018607">ZNF285CP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.800</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.800</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>2-133074683-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs7355726">rs7355726</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.800</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000424130.2">ZNF285CP:ENST00000424130.2:n.144T&gt;C:</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438300.1">ZNF285CP:ENST00000438300.1:n.148T&gt;C:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="STKLD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198870">STKLD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.799</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.799</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>9-136245998-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs376960568">rs376960568</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.799</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371955.1">STKLD1:ENST00000371955.1:c.-952+5G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371957.3">STKLD1:ENST00000371957.3:c.174+5G&gt;A:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426926.2">STKLD1:ENST00000426926.2:c.174+5G&gt;A:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0023%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLAMF8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158714">SLAMF8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.003</b>
                    <data>(p=4.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.797</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.003</b>
                                    <data>(p=4.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.797</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>1-159804970-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs924324896">rs924324896</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.797</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000289707.5">SLAMF8:ENST00000289707.5:c.782-4C&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368104.4">SLAMF8:ENST00000368104.4:c.455-4C&gt;G:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368102.1">SLAMF8:ENST00000368102.1:c.*977G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_AFR: 0.0230%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ARSD" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000006756">ARSD</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.211</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARSD%0DARSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARSB</a> and phenotypic similarity 0.391 to <a href="http://www.omim.org/entry/253200">Mucopolysaccharidosis type VI (Maroteaux-Lamy)</a> associated with ARSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000670, Carious teeth</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARSD%0DARSB&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ARSB</a> and phenotypic similarity 0.828 to mouse mutant of ARSB.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000097, short maxilla</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0001300, ocular hypertelorism</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>X_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.211</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>X-2833643-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs115332247">rs115332247</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.211</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381154.1">ARSD:ENST00000381154.1:c.954G&gt;T:p.(Gln318His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.219</dd>
                                    
                                    <dd>REVEL: 0.219</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>gnomAD_E_FIN: 0.0091%</dd>
                                    <dd>gnomAD_E_SAS: 0.0106%</dd>
                                    <dd>gnomAD_G_AFR: 0.0800%</dd>
                                    <dd>gnomAD_G_AMR: 0.2347%</dd>
                                    <dd>gnomAD_G_FIN: 0.1067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
                <div class="row">
                    <div class="col-sm-12">
                        <b>Other passed variants:</b>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col-sm-12">
                        <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                        <b>SNV</b>
                        <b>X-2835964-G-A [0/1]</b>
                        
                        
                            <a href="https://www.ncbi.nlm.nih.gov/snp/rs73632972">rs73632972</a>
                            
                            
                            
                            
                        
                    </div>
                    <!--                    <div class="col-sm-12">-->
                    <!--                        <b>Exomiser ACMG: </b>-->
                    <!--                        <th:bloc  th:with="moi =${geneScore.getModeOfInheritance()} , acmgClassification = ${variantEvaluation.acmgAssignmentForMoi(${moi})}"-->
                    <!--                                  th:switch="${acmgClassification.toString()}">-->
                    <!--                                    <span th:case="'BENIGN'" class="label label-success"-->
                    <!--                                          th:text="${acmgClassification}">BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_BENIGN'" class="label label-info"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_BENIGN</span>-->
                    <!--                            <span th:case="'LIKELY_PATHOGENIC'" class="label label-warning"-->
                    <!--                                  th:text="${acmgClassification}">LIKELY_PATHOGENIC</span>-->
                    <!--                            <span th:case="'PATHOGENIC'" class="label label-danger"-->
                    <!--                                  th:text="${acmgClassification}">PATHOGENIC</span>-->
                    <!--                            <span th:case="'UNCERTAIN_SIGNIFICANCE'" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">UNCERTAIN_SIGNIFICANCE</span>-->
                    <!--                            <span th:case="*" class="label label-default"-->
                    <!--                                  th:text="${acmgClassification}">NOT_AVAILABLE</span>-->
                    <!--                            <b-->
                    <!--                                    th:text="${acmgClassification.acmgCriteria()}">-->
                    <!--                                [PVS1, BP2]-->
                    <!--                            </b>-->
                    <!--                        </th:bloc>-->
                    <!--                    </div>-->
                    
                    <div class="col-sm-6">
                        <b>Variant score: 0.096</b>
                        
                        <dl class="list-unstyled">
                            <dt>Transcripts:</dt>
                            <dd>
                                
                                <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381154.1">ARSD:ENST00000381154.1:c.744C&gt;T:p.(=)</a>
                                
                            </dd>
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Pathogenicity Score: 1.00</b>-->
                        <dl class="list-unstyled">
                            <dt>Pathogenicity Data:</dt>
                            
                            <dd>No
                                pathogenicity data
                            </dd>
                            
                        </dl>
                    </div>
                    <div class="col-sm-3">
                        <!--<b>Frequency Score: 0.95</b>-->
                        <dl>
                            <dt>Frequency Data:</dt>
                            
                            <dd>TOPMed: 0.0008%</dd>
                            <dd>gnomAD_G_AFR: 0.0806%</dd>
                            <dd>gnomAD_G_AMR: 0.2387%</dd>
                            <dd>gnomAD_G_FIN: 0.1067%</dd>
                            <dd>gnomAD_G_NFE: 0.0293%</dd>
                        </dl>
                    </div>
                </div>
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PRRC2B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000288701">PRRC2B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.536</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.182</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.536 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PRRC2B">PRRC2B</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006203, eye hemorrhage</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002092, abnormal eye morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0013826, absent hypoglossal canal</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRRC2B%0DNXN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NXN</a> and phenotypic similarity 0.631 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1507">Autosomal recessive Robinow syndrome</a> associated with NXN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PRRC2B%0DNXN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NXN</a> and phenotypic similarity 0.561 to mouse mutant of NXN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000157, abnormal sternum morphology</dd><dd>HP:0000486, Strabismus - MP:0004377, small frontal bone</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004377, small frontal bone</dd><dd>HP:0000520, Proptosis - MP:0004377, small frontal bone</dd><dd>HP:0000601, Hypotelorism - MP:0004377, small frontal bone</dd><dd>HP:0000244, Brachyturricephaly - MP:0004377, small frontal bone</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.536</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.182</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-134308126-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs41302675">rs41302675</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.276</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357304.4">PRRC2B:ENST00000357304.4:c.238G&gt;A:p.(Val80Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405995.1">PRRC2B:ENST00000405995.1:c.238G&gt;A:p.(Val80Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458550.1">PRRC2B:ENST00000458550.1:c.238G&gt;A:p.(Val80Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.317</dd>
                                    
                                    <dd>REVEL: 0.317</dd>
                                    <dd>MVP: 0.103</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.2784%</dd>
                                    <dd>UK10K: 0.6744%</dd>
                                    <dd>ESP AA: 0.1207%</dd>
                                    <dd>ESP EA: 0.2959%</dd>
                                    <dd>ESP All: 0.2382%</dd>
                                    <dd>ExAC AFR: 0.0919%</dd>
                                    <dd>ExAC AMR: 0.4061%</dd>
                                    <dd>ExAC FIN: 0.3024%</dd>
                                    <dd>ExAC NFE: 0.5402%</dd>
                                    <dd>ExAC OTH: 0.3341%</dd>
                                    <dd>ExAC SAS: 0.1453%</dd>
                                    <dd>gnomAD_E_AFR: 0.0720%</dd>
                                    <dd>gnomAD_E_AMR: 0.3186%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.2152%</dd>
                                    <dd>gnomAD_E_NFE: 0.5622%</dd>
                                    <dd>gnomAD_E_OTH: 0.4931%</dd>
                                    <dd>gnomAD_E_SAS: 0.1494%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.1431%</dd>
                                    <dd>gnomAD_G_NFE: 0.4331%</dd>
                                    <dd>gnomAD_G_OTH: 0.4073%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-134349958-A-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200268888">rs200268888</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.088</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357304.4">PRRC2B:ENST00000357304.4:c.2442A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405995.1">PRRC2B:ENST00000405995.1:c.2324+847A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000458550.1">PRRC2B:ENST00000458550.1:c.2324+847A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372249.1">PRRC2B:ENST00000372249.1:c.-3268A&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1597%</dd>
                                    <dd>TOPMed: 0.4073%</dd>
                                    <dd>UK10K: 0.6215%</dd>
                                    <dd>ESP AA: 0.1595%</dd>
                                    <dd>ESP EA: 0.5711%</dd>
                                    <dd>ESP All: 0.4420%</dd>
                                    <dd>ExAC AFR: 0.1226%</dd>
                                    <dd>ExAC AMR: 0.3375%</dd>
                                    <dd>ExAC FIN: 0.0302%</dd>
                                    <dd>ExAC NFE: 0.5113%</dd>
                                    <dd>ExAC OTH: 0.3348%</dd>
                                    <dd>ExAC SAS: 0.1333%</dd>
                                    <dd>gnomAD_E_AFR: 0.1178%</dd>
                                    <dd>gnomAD_E_AMR: 0.4556%</dd>
                                    <dd>gnomAD_E_FIN: 0.1076%</dd>
                                    <dd>gnomAD_E_NFE: 0.5541%</dd>
                                    <dd>gnomAD_E_OTH: 0.5657%</dd>
                                    <dd>gnomAD_E_SAS: 0.1072%</dd>
                                    <dd>gnomAD_G_AFR: 0.1147%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_FIN: 0.0572%</dd>
                                    <dd>gnomAD_G_NFE: 0.4401%</dd>
                                    <dd>gnomAD_G_OTH: 0.2037%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC9B1P3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000233867">SLC9B1P3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.789</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.789</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>10-38934323-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs552153337">rs552153337</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.789</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452667.2">SLC9B1P3:ENST00000452667.2:n.1119-6A&gt;G:</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0998%</dd>
                                    <dd>TOPMed: 0.0998%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ULK3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000140474">ULK3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.208</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.474 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ULK3">ULK3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001304, cataract</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001304, cataract</dd><dd>HP:0000520, Proptosis - MP:0001304, cataract</dd><dd>HP:0000601, Hypotelorism - MP:0001304, cataract</dd><dd>HP:0000244, Brachyturricephaly - MP:0001304, cataract</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ULK3%0DPFAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PFAS</a> and phenotypic similarity 0.772 to mouse mutant of PFAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0005175, non-pigmented tail tip</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030466, alveolar process atrophy</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0011862, decreased cranium length</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.208</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-75131031-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs753772870">rs753772870</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.208</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440863.2">ULK3:ENST00000440863.2:c.1055A&gt;G:p.(Asn352Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000568667.1">ULK3:ENST00000568667.1:c.1088A&gt;G:p.(Asn363Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000569437.1">ULK3:ENST00000569437.1:c.1055A&gt;G:p.(Asn352Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20881669</dd>
                                    
                                    <dd>REVEL: 0.012</dd>
                                    <dd>MVP: 0.209</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0045%</dd>
                                    <dd>ExAC AMR: 0.0103%</dd>
                                    <dd>ExAC EAS: 0.0264%</dd>
                                    <dd>ExAC NFE: 0.0035%</dd>
                                    <dd>ExAC SAS: 0.0073%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0233%</dd>
                                    <dd>gnomAD_E_FIN: 0.0046%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0368%</dd>
                                    <dd>gnomAD_E_SAS: 0.0098%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR13J1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168828">OR13J1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.002</b>
                    <data>(p=4.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.192</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR13J1%0DSTAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STAG2</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93925">Alobar holoprosencephaly</a> associated with STAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0006315, Solitary median maxillary central incisor</dd><dd>HP:0000520, Proptosis - HP:0009914, Cyclopia</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.002</b>
                                    <data>(p=4.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.192</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-35869498-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146834712">rs146834712</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2365224%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.192</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377981.2">OR13J1:ENST00000377981.2:c.901G&gt;A:p.(Ala301Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.19290502</dd>
                                    
                                    <dd>REVEL: 0.028</dd>
                                    <dd>MVP: 0.193</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0311%</dd>
                                    <dd>ESP EA: 0.0465%</dd>
                                    <dd>ESP All: 0.0308%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0346%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.0180%</dd>
                                    <dd>ExAC SAS: 0.0485%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0387%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0206%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                    <dd>gnomAD_G_NFE: 0.0468%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CELSR1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000075275">CELSR1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.158</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.478 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CELSR1">CELSR1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002699, abnormal vitreous body morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002699, abnormal vitreous body morphology</dd><dd>HP:0000520, Proptosis - MP:0002699, abnormal vitreous body morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002699, abnormal vitreous body morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002699, abnormal vitreous body morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CELSR1%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.820 to mouse mutant of PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002544, brachydactyly</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030475, fused lower incisors</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0030049, prominent forehead</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/619319">OMIM:619319</a> Lymphatic malformation 9 - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.158</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-46804943-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201049711">rs201049711</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.158</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262738.3">CELSR1:ENST00000262738.3:c.5176A&gt;G:p.(Ser1726Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.15854086</dd>
                                    
                                    <dd>REVEL: 0.091</dd>
                                    <dd>MVP: 0.159</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0068%</dd>
                                    <dd>ExAC NFE: 0.0077%</dd>
                                    <dd>gnomAD_E_AMR: 0.0120%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_E_OTH: 0.0367%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.283</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-46932032-T-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142090496">rs142090496</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1687480%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.408</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262738.3">CELSR1:ENST00000262738.3:c.1036A&gt;T:p.(Thr346Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395964.1">CELSR1:ENST00000395964.1:c.1036A&gt;T:p.(Thr346Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.42559242</dd>
                                    
                                    <dd>REVEL: 0.189</dd>
                                    <dd>MVP: 0.426</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.1360%</dd>
                                    <dd>UK10K: 0.1058%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP EA: 0.2674%</dd>
                                    <dd>ESP All: 0.1923%</dd>
                                    <dd>ExAC AFR: 0.0293%</dd>
                                    <dd>ExAC AMR: 0.0780%</dd>
                                    <dd>ExAC FIN: 0.0305%</dd>
                                    <dd>ExAC NFE: 0.1707%</dd>
                                    <dd>ExAC SAS: 0.0364%</dd>
                                    <dd>gnomAD_E_AFR: 0.0591%</dd>
                                    <dd>gnomAD_E_AMR: 0.1251%</dd>
                                    <dd>gnomAD_E_FIN: 0.0182%</dd>
                                    <dd>gnomAD_E_NFE: 0.2012%</dd>
                                    <dd>gnomAD_E_OTH: 0.1281%</dd>
                                    <dd>gnomAD_E_SAS: 0.0520%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_FIN: 0.0859%</dd>
                                    <dd>gnomAD_G_NFE: 0.1601%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>22-46804943-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201049711">rs201049711</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.158</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262738.3">CELSR1:ENST00000262738.3:c.5176A&gt;G:p.(Ser1726Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.15854086</dd>
                                    
                                    <dd>REVEL: 0.091</dd>
                                    <dd>MVP: 0.159</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0068%</dd>
                                    <dd>ExAC NFE: 0.0077%</dd>
                                    <dd>gnomAD_E_AMR: 0.0120%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_E_OTH: 0.0367%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PCDHA6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000081842">PCDHA6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.722</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.722</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-140209597-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs782454436">rs782454436</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.722</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529310.1">PCDHA6:ENST00000529310.1:c.1921G&gt;T:p.(Asp641Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527624.1">PCDHA6:ENST00000527624.1:c.1602+319G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529619.1">PCDHA6:ENST00000529619.1:c.2352+5885G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529859.1">PCDHA6:ENST00000529859.1:c.2352+5885G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512229.2">PCDHA6:ENST00000512229.2:c.2385+20440G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530339.1">PCDHA6:ENST00000530339.1:c.2385+20440G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526136.1">PCDHA6:ENST00000526136.1:c.2388+32660G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394633.3">PCDHA6:ENST00000394633.3:c.1602+42120G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504120.2">PCDHA6:ENST00000504120.2:c.2394+41328G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522353.2">PCDHA6:ENST00000522353.2:c.2394+26421G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7243917</dd>
                                    
                                    <dd>REVEL: 0.333</dd>
                                    <dd>MVP: 0.724</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.711</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-140209597-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs782454436">rs782454436</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.722</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529310.1">PCDHA6:ENST00000529310.1:c.1921G&gt;T:p.(Asp641Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527624.1">PCDHA6:ENST00000527624.1:c.1602+319G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529619.1">PCDHA6:ENST00000529619.1:c.2352+5885G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529859.1">PCDHA6:ENST00000529859.1:c.2352+5885G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512229.2">PCDHA6:ENST00000512229.2:c.2385+20440G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530339.1">PCDHA6:ENST00000530339.1:c.2385+20440G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526136.1">PCDHA6:ENST00000526136.1:c.2388+32660G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394633.3">PCDHA6:ENST00000394633.3:c.1602+42120G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504120.2">PCDHA6:ENST00000504120.2:c.2394+41328G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522353.2">PCDHA6:ENST00000522353.2:c.2394+26421G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7243917</dd>
                                    
                                    <dd>REVEL: 0.333</dd>
                                    <dd>MVP: 0.724</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>5-140209598-A-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs782671408">rs782671408</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.700</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529310.1">PCDHA6:ENST00000529310.1:c.1922A&gt;T:p.(Asp641Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527624.1">PCDHA6:ENST00000527624.1:c.1602+320A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529619.1">PCDHA6:ENST00000529619.1:c.2352+5886A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000529859.1">PCDHA6:ENST00000529859.1:c.2352+5886A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000512229.2">PCDHA6:ENST00000512229.2:c.2385+20441A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530339.1">PCDHA6:ENST00000530339.1:c.2385+20441A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000526136.1">PCDHA6:ENST00000526136.1:c.2388+32661A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394633.3">PCDHA6:ENST00000394633.3:c.1602+42121A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504120.2">PCDHA6:ENST00000504120.2:c.2394+41329A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000522353.2">PCDHA6:ENST00000522353.2:c.2394+26422A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.7022398</dd>
                                    
                                    <dd>REVEL: 0.323</dd>
                                    <dd>MVP: 0.702</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC AMR: 0.0260%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SETD2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181555">SETD2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.547</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.547 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=821">Sotos syndrome</a> associated with SETD2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004691, 2-3 toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0400000, Tall chin</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Phenotypic similarity 0.296 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SETD2">SETD2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000784, forebrain hypoplasia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/616831">OMIM:616831</a> Luscan-Lumish syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=821">ORPHA:821</a> Sotos syndrome - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.547</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.547 to ORPHA:821 Sotos syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.388 to OMIM:616831 Luscan-Lumish syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-47088028-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs780412960">rs780412960</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409792.3">SETD2:ENST00000409792.3:c.7047C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SWSAP1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000173928">SWSAP1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.147</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SWSAP1%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.699 to <a href="http://www.omim.org/entry/616541">Short stature, microcephaly, and endocrine dysfunction</a> associated with XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus - HP:0007772, Impaired smooth pursuit</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000331, Short chin</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SWSAP1%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.248 to mouse mutant of XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000788, abnormal cerebral cortex morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.147</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-11486643-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150033461">rs150033461</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2234775%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.147</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000312423.2">SWSAP1:ENST00000312423.2:c.641C&gt;G:p.(Ala214Gly)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.14800313</dd>
                                    
                                    <dd>REVEL: 0.077</dd>
                                    <dd>MVP: 0.148</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0204%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AFR: 0.0253%</dd>
                                    <dd>ExAC AMR: 0.0289%</dd>
                                    <dd>ExAC NFE: 0.0254%</dd>
                                    <dd>gnomAD_E_AFR: 0.0071%</dd>
                                    <dd>gnomAD_E_AMR: 0.0150%</dd>
                                    <dd>gnomAD_E_NFE: 0.0298%</dd>
                                    <dd>gnomAD_E_OTH: 0.0378%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NBEAL1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000144426">NBEAL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.716</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.716</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-204053267-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs745409621">rs745409621</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.716</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000449802.1">NBEAL1:ENST00000449802.1:c.6691G&gt;C:p.(Ala2231Pro)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.71958745</dd>
                                    
                                    <dd>REVEL: 0.570</dd>
                                    <dd>MVP: 0.720</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0140%</dd>
                                    <dd>ExAC AMR: 0.0346%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0364%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_E_OTH: 0.0184%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="NCOA7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111912">NCOA7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.146</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.248 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=NCOA7">NCOA7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000788, abnormal cerebral cortex morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=NCOA7%0DPOU4F2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POU4F2</a> and phenotypic similarity 0.821 to mouse mutant of POU4F2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006156, abnormal visual pursuit</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002003, miotic pupils</dd><dd>HP:0000520, Proptosis - MP:0002003, miotic pupils</dd><dd>HP:0000601, Hypotelorism - MP:0002003, miotic pupils</dd><dd>HP:0000244, Brachyturricephaly - MP:0006221, optic nerve hypoplasia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.146</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-126210903-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1183526634">rs1183526634</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.146</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000229634.9">NCOA7:ENST00000229634.9:c.1358G&gt;T:p.(Gly453Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368357.3">NCOA7:ENST00000368357.3:c.1703G&gt;T:p.(Gly568Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392477.2">NCOA7:ENST00000392477.2:c.1703G&gt;T:p.(Gly568Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.146</dd>
                                    
                                    <dd>REVEL: 0.146</dd>
                                    <dd>MVP: 0.082</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KIAA0753" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000198920">KIAA0753</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.628</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.628 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">Orofaciodigital syndrome type 6</a> associated with KIAA0753.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001159, Syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0004422, Biparietal narrowing</dd></dl><dl><dt>Phenotypic similarity 0.335 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=KIAA0753">KIAA0753</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617127">OMIM:617127</a> ?Orofaciodigital syndrome XV (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/619476">OMIM:619476</a> ?Joubert syndrome 38 (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/619479">OMIM:619479</a> Short-rib thoracic dysplasia 21 without polydactyly - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">ORPHA:2754</a> Orofaciodigital syndrome type 6 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.167</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.197</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-6483131-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs746702772">rs746702772</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.197</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000361413.3">KIAA0753:ENST00000361413.3:c.2840A&gt;G:p.(Gln947Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542606.1">KIAA0753:ENST00000542606.1:c.1943A&gt;G:p.(Gln648Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000572370.1">KIAA0753:ENST00000572370.1:c.1943A&gt;G:p.(Gln648Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000589033.1">KIAA0753:ENST00000589033.1:c.1208A&gt;G:p.(Gln403Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.19762549</dd>
                                    
                                    <dd>REVEL: 0.037</dd>
                                    <dd>MVP: 0.198</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0034%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GLRX3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000108010">GLRX3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.519</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.123</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.232 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GLRX3">GLRX3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0001915, intracranial hemorrhage</dd></dl><dl><dt>Proximity score 0.519 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLRX3%0DTP53RK&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TP53RK</a> and phenotypic similarity 0.701 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2065">Galloway-Mowat syndrome</a> associated with TP53RK.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001181, Adducted thumb</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.519</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.123</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-131965213-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs760738747">rs760738747</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.123</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331244.5">GLRX3:ENST00000331244.5:c.682A&gt;G:p.(Ile228Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368644.1">GLRX3:ENST00000368644.1:c.682A&gt;G:p.(Ile228Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.123314135</dd>
                                    
                                    <dd>REVEL: 0.015</dd>
                                    <dd>MVP: 0.123</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>gnomAD_E_AMR: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RFX7" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181827">RFX7</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.140</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.452 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=RFX7">RFX7</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001325, abnormal retina morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001325, abnormal retina morphology</dd><dd>HP:0000520, Proptosis - MP:0001325, abnormal retina morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001325, abnormal retina morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001325, abnormal retina morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=RFX7%0DRARA&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RARA</a> and phenotypic similarity 0.698 to mouse mutant of RARA.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000571, interdigital webbing</dd><dd>HP:0000486, Strabismus - MP:0003056, abnormal hyoid bone morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003056, abnormal hyoid bone morphology</dd><dd>HP:0000520, Proptosis - MP:0003056, abnormal hyoid bone morphology</dd><dd>HP:0000601, Hypotelorism - MP:0003056, abnormal hyoid bone morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0003056, abnormal hyoid bone morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.140</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>15-56385886-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs375177046">rs375177046</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.140</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317318.6">RFX7:ENST00000317318.6:c.4040A&gt;G:p.(Asn1347Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000422057.1">RFX7:ENST00000422057.1:c.3749A&gt;G:p.(Asn1250Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000423270.1">RFX7:ENST00000423270.1:c.4040A&gt;G:p.(Asn1347Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000559447.2">RFX7:ENST00000559447.2:c.3749A&gt;G:p.(Asn1250Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.141</dd>
                                    
                                    <dd>REVEL: 0.141</dd>
                                    <dd>MVP: 0.068</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0053%</dd>
                                    <dd>ESP AA: 0.0262%</dd>
                                    <dd>ESP All: 0.0083%</dd>
                                    <dd>ExAC AFR: 0.0306%</dd>
                                    <dd>ExAC NFE: 0.0120%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALB" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163631">ALB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.706</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/615999">OMIM:615999</a> ?[Dysalbuminemic hypertriiodothyroninemia] (unconfirmed)</dd>
<dd><a href="http://www.omim.org/entry/616000">OMIM:616000</a> Analbuminemia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=86816">ORPHA:86816</a> Congenital analbuminemia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.706</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>4-74285363-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141131597">rs141131597</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=294171%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.707</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295897.4">ALB:ENST00000295897.4:c.1785+7C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401494.3">ALB:ENST00000401494.3:c.1440+7C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415165.2">ALB:ENST00000415165.2:c.1209+7C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503124.1">ALB:ENST00000503124.1:c.1335+7C&gt;T:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509063.1">ALB:ENST00000509063.1:c.1785+7C&gt;T:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.2441%</dd>
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ESP AA: 0.4539%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.1615%</dd>
                                    <dd>ExAC AFR: 0.5575%</dd>
                                    <dd>ExAC AMR: 0.1649%</dd>
                                    <dd>ExAC NFE: 0.0360%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.5228%</dd>
                                    <dd>gnomAD_E_AMR: 0.1878%</dd>
                                    <dd>gnomAD_E_NFE: 0.0376%</dd>
                                    <dd>gnomAD_E_OTH: 0.1825%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.6200%</dd>
                                    <dd>gnomAD_G_AMR: 0.3597%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SPLICE_REGION_VARIANT</span>
                            <b>SNV</b>
                            <b>4-74283242-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs57705126">rs57705126</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=295692%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.705</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000295897.4">ALB:ENST00000295897.4:c.1290-6T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000401494.3">ALB:ENST00000401494.3:c.945-6T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000415165.2">ALB:ENST00000415165.2:c.714-6T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503124.1">ALB:ENST00000503124.1:c.840-6T&gt;C:p.?</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000509063.1">ALB:ENST00000509063.1:c.1290-6T&gt;C:p.?</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.2441%</dd>
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ESP AA: 0.4312%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.1538%</dd>
                                    <dd>ExAC AFR: 0.5809%</dd>
                                    <dd>ExAC AMR: 0.1650%</dd>
                                    <dd>ExAC NFE: 0.0360%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.5296%</dd>
                                    <dd>gnomAD_E_AMR: 0.1879%</dd>
                                    <dd>gnomAD_E_NFE: 0.0376%</dd>
                                    <dd>gnomAD_E_OTH: 0.1825%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.6302%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_NFE: 0.0266%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR9Q1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000186509">OR9Q1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.140</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR9Q1%0DSTAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STAG2</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93925">Alobar holoprosencephaly</a> associated with STAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0006315, Solitary median maxillary central incisor</dd><dd>HP:0000520, Proptosis - HP:0009914, Cyclopia</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.140</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-57947223-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1565098605">rs1565098605</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.140</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335397.3">OR9Q1:ENST00000335397.3:c.307C&gt;A:p.(Leu103Met)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.13967828</dd>
                                    
                                    <dd>REVEL: 0.101</dd>
                                    <dd>MVP: 0.140</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0015%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DDX60L" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000181381">DDX60L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.120</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX60L%0DSKIC3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SKIC3</a> and phenotypic similarity 0.618 to <a href="http://www.omim.org/entry/222470">Trichohepatoenteric syndrome 1</a> associated with SKIC3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0009891, Underdeveloped supraorbital ridges</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0009891, Underdeveloped supraorbital ridges</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DDX60L%0DTTC37&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TTC37</a> and phenotypic similarity 0.474 to mouse mutant of TTC37.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001304, cataract</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001304, cataract</dd><dd>HP:0000520, Proptosis - MP:0001304, cataract</dd><dd>HP:0000601, Hypotelorism - MP:0001304, cataract</dd><dd>HP:0000244, Brachyturricephaly - MP:0001304, cataract</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.120</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-169393005-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1426968548">rs1426968548</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.120</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000260184.7">DDX60L:ENST00000260184.7:c.157G&gt;A:p.(Val53Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505890.1">DDX60L:ENST00000505890.1:c.157G&gt;A:p.(Val53Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000511577.1">DDX60L:ENST00000511577.1:c.157G&gt;A:p.(Val53Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.12</dd>
                                    
                                    <dd>REVEL: 0.120</dd>
                                    <dd>MVP: 0.067</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EPB41L5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115109">EPB41L5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.523</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.452 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EPB41L5">EPB41L5</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001325, abnormal retina morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001325, abnormal retina morphology</dd><dd>HP:0000520, Proptosis - MP:0001325, abnormal retina morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001325, abnormal retina morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001325, abnormal retina morphology</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EPB41L5%0DNXN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NXN</a> and phenotypic similarity 0.631 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1507">Autosomal recessive Robinow syndrome</a> associated with NXN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.523 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EPB41L5%0DNXN&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NXN</a> and phenotypic similarity 0.561 to mouse mutant of NXN.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000157, abnormal sternum morphology</dd><dd>HP:0000486, Strabismus - MP:0004377, small frontal bone</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004377, small frontal bone</dd><dd>HP:0000520, Proptosis - MP:0004377, small frontal bone</dd><dd>HP:0000601, Hypotelorism - MP:0004377, small frontal bone</dd><dd>HP:0000244, Brachyturricephaly - MP:0004377, small frontal bone</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.523</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-120849134-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1255545247">rs1255545247</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263713.5">EPB41L5:ENST00000263713.5:c.1047G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331393.4">EPB41L5:ENST00000331393.4:c.1047G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443124.1">EPB41L5:ENST00000443124.1:c.1047G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000443902.2">EPB41L5:ENST00000443902.2:c.1047G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000452780.1">EPB41L5:ENST00000452780.1:c.1047G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="EP400" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000183495">EP400</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.520</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.157 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=EP400">EP400</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000930, wavy neural tube</dd></dl><dl><dt>Proximity score 0.520 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EP400%0DZMYND11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZMYND11</a> and phenotypic similarity 0.600 to <a href="http://www.omim.org/entry/616083">Intellectual developmental disorder, autosomal dominant 30</a> associated with ZMYND11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.520 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=EP400%0DZMYND11&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ZMYND11</a> and phenotypic similarity 0.483 to mouse mutant of ZMYND11.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0003686, abnormal eye muscle morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000111, cleft palate</dd><dd>HP:0000520, Proptosis - MP:0003686, abnormal eye muscle morphology</dd><dd>HP:0000601, Hypotelorism - MP:0003686, abnormal eye muscle morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000783, abnormal forebrain morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.520</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-132490838-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1051046685">rs1051046685</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330386.6">EP400:ENST00000330386.6:c.3117C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000332482.4">EP400:ENST00000332482.4:c.3006C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333577.4">EP400:ENST00000333577.4:c.3225C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389561.2">EP400:ENST00000389561.2:c.3117C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389562.2">EP400:ENST00000389562.2:c.3114C&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="THRAP3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000054118">THRAP3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.502</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.120</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=THRAP3%0DMED12&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MED12</a> and phenotypic similarity 0.662 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=776">Lujan-Fryns syndrome</a> associated with MED12.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=THRAP3%0DMED12&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MED12</a> and phenotypic similarity 0.169 to mouse mutant of MED12.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0003400, kinked neural tube</dd></dl><dl><dt>Proximity score 0.502 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=THRAP3%0DMED12&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to MED12</a> and phenotypic similarity 0.272 to fish mutant of MED12.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001052, mandibular arch skeleton morphology, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.502</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.120</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-36755247-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs778663232">rs778663232</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.120</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354618.5">THRAP3:ENST00000354618.5:c.1627G&gt;A:p.(Glu543Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000469141.2">THRAP3:ENST00000469141.2:c.1627G&gt;A:p.(Glu543Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.12</dd>
                                    
                                    <dd>REVEL: 0.120</dd>
                                    <dd>MVP: 0.093</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SMARCA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000080503">SMARCA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.513</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.513 to <a href="http://www.omim.org/entry/619293">Blepharophimosis-impaired intellectual development syndrome</a> associated with SMARCA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001182, Tapered finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000687, Widely spaced teeth</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0001357, Plagiocephaly</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMARCA2%0DPOU4F2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to POU4F2</a> and phenotypic similarity 0.821 to mouse mutant of POU4F2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0006156, abnormal visual pursuit</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002003, miotic pupils</dd><dd>HP:0000520, Proptosis - MP:0002003, miotic pupils</dd><dd>HP:0000601, Hypotelorism - MP:0002003, miotic pupils</dd><dd>HP:0000244, Brachyturricephaly - MP:0006221, optic nerve hypoplasia</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/601358">OMIM:601358</a> Nicolaides-Baraitser syndrome - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/619293">OMIM:619293</a> Blepharophimosis-impaired intellectual development syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2728">ORPHA:2728</a> Blepharophimosis-intellectual disability syndrome, Ohdo type - unknown</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3051">ORPHA:3051</a> Nicolaides-Baraitser syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.513</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.513 to OMIM:619293 Blepharophimosis-impaired intellectual development syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.384 to OMIM:601358 Nicolaides-Baraitser syndrome</dt>
                                
                                    <dt>Phenotypic similarity 0.374 to ORPHA:3051 Nicolaides-Baraitser syndrome</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-2039575-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1160691232">rs1160691232</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349721.2">SMARCA2:ENST00000349721.2:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357248.2">SMARCA2:ENST00000357248.2:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382194.1">SMARCA2:ENST00000382194.1:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382203.1">SMARCA2:ENST00000382203.1:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.257</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-2039575-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1160691232">rs1160691232</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349721.2">SMARCA2:ENST00000349721.2:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357248.2">SMARCA2:ENST00000357248.2:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382194.1">SMARCA2:ENST00000382194.1:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382203.1">SMARCA2:ENST00000382203.1:c.465C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-2077644-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs149890563">rs149890563</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1898163%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349721.2">SMARCA2:ENST00000349721.2:c.2052C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357248.2">SMARCA2:ENST00000357248.2:c.2052C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382194.1">SMARCA2:ENST00000382194.1:c.2052C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000382203.1">SMARCA2:ENST00000382203.1:c.2052C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0015%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GOLGA3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000090615">GOLGA3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GOLGA3%0DCOG4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COG4</a> and phenotypic similarity 0.611 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85172">Microcephalic osteodysplastic dysplasia, Saul-Wilson type</a> associated with COG4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0010230, Cone-shaped epiphyses of the phalanges of the hand</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000272, Malar flattening</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.511 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GOLGA3%0DCOG4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to COG4</a> and phenotypic similarity 0.338 to fish mutant of COG4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001965, mandibular arch skeleton decreased size, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>12-133384698-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs745645465">rs745645465</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000204726.3">GOLGA3:ENST00000204726.3:c.957G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450791.2">GOLGA3:ENST00000450791.2:c.957G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456883.2">GOLGA3:ENST00000456883.2:c.957G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537452.1">GOLGA3:ENST00000537452.1:c.957G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545875.1">GOLGA3:ENST00000545875.1:c.957G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0038%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0092%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0055%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="JMJD4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000081692">JMJD4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.511</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.511 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=JMJD4">JMJD4</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005358, abnormal incisor morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005358, abnormal incisor morphology</dd><dd>HP:0000520, Proptosis - MP:0005358, abnormal incisor morphology</dd><dd>HP:0000601, Hypotelorism - MP:0005358, abnormal incisor morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0005358, abnormal incisor morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.511</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-227921376-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs754677509">rs754677509</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366758.3">JMJD4:ENST00000366758.3:c.699G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000438896.2">JMJD4:ENST00000438896.2:c.699G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366760.1">JMJD4:ENST00000366760.1:c.-95+4889C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0019%</dd>
                                    <dd>ExAC EAS: 0.0117%</dd>
                                    <dd>ExAC NFE: 0.0031%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AFAP1L1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157510">AFAP1L1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.507</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AFAP1L1%0DSH3PXD2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3PXD2B</a> and phenotypic similarity 0.648 to <a href="http://www.omim.org/entry/249420">Frank-ter Haar syndrome</a> associated with SH3PXD2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0009803, Short phalanx of finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.507 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=AFAP1L1%0DSH3PXD2B&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to SH3PXD2B</a> and phenotypic similarity 0.862 to mouse mutant of SH3PXD2B.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0004686, decreased length of long bones</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002750, exophthalmos</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0002750, exophthalmos</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.507</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>5-148689617-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139614291">rs139614291</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000296721.4">AFAP1L1:ENST00000296721.4:c.846C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000515000.1">AFAP1L1:ENST00000515000.1:c.846C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0272%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0384%</dd>
                                    <dd>ExAC AFR: 0.0481%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0180%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_NFE: 0.0161%</dd>
                                    <dd>gnomAD_E_OTH: 0.0547%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNM2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000079805">DNM2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.323 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169189">Autosomal dominant centronuclear myopathy</a> associated with DNM2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000544, External ophthalmoplegia</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0004488, Macrocephaly at birth</dd></dl><dl><dt>Phenotypic similarity 0.475 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DNM2">DNM2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001297, microphthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000111, cleft palate</dd><dd>HP:0000520, Proptosis - MP:0001297, microphthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001297, microphthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0012260, encephalomeningocele</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNM2%0DNOSIP&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NOSIP</a> and phenotypic similarity 0.847 to mouse mutant of NOSIP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005163, cyclopia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030268, agnathia</dd><dd>HP:0000520, Proptosis - MP:0006197, ocular hypotelorism</dd><dd>HP:0000601, Hypotelorism - MP:0006197, ocular hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0006290, proboscis</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/160150">OMIM:160150</a> Centronuclear myopathy 1 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/606482">OMIM:606482</a> Charcot-Marie-Tooth disease, axonal type 2M - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/615368">OMIM:615368</a> Lethal congenital contracture syndrome 5 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=169189">ORPHA:169189</a> Autosomal dominant centronuclear myopathy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.323 to ORPHA:169189 Autosomal dominant centronuclear myopathy</dt>
                                
                                    <dt>Phenotypic similarity 0.264 to OMIM:160150 Centronuclear myopathy 1</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-10939816-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs770491381">rs770491381</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2, BP6]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1148384%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000314646.5">DNM2:ENST00000314646.5:c.2163C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000355667.6">DNM2:ENST00000355667.6:c.2163C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000359692.6">DNM2:ENST00000359692.6:c.2151C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389253.4">DNM2:ENST00000389253.4:c.2163C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000408974.4">DNM2:ENST00000408974.4:c.2151C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585892.1">DNM2:ENST00000585892.1:c.2163C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ACOX3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000087008">ACOX3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACOX3%0DHSD17B4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HSD17B4</a> and phenotypic similarity 0.580 to <a href="http://www.omim.org/entry/261515">D-bifunctional protein deficiency</a> associated with HSD17B4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001765, Hammertoe</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACOX3%0DHSD17B4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HSD17B4</a> and phenotypic similarity 0.601 to mouse mutant of HSD17B4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000157, abnormal sternum morphology</dd><dd>HP:0000486, Strabismus - MP:0002092, abnormal eye morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002100, abnormal tooth morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002092, abnormal eye morphology</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ACOX3%0DHSD17B4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to HSD17B4</a> and phenotypic similarity 0.310 to fish mutant of HSD17B4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>4-8401310-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765251206">rs765251206</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356406.5">ACOX3:ENST00000356406.5:c.846C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413009.2">ACOX3:ENST00000413009.2:c.846C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000503233.1">ACOX3:ENST00000503233.1:c.846C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MSH6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000116062">MSH6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.505</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MSH6%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.699 to <a href="http://www.omim.org/entry/616541">Short stature, microcephaly, and endocrine dysfunction</a> associated with XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0030084, Clinodactyly</dd><dd>HP:0000486, Strabismus - HP:0007772, Impaired smooth pursuit</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000331, Short chin</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.505 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MSH6%0DXRCC4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to XRCC4</a> and phenotypic similarity 0.248 to mouse mutant of XRCC4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000788, abnormal cerebral cortex morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/608089">OMIM:608089</a> Endometrial cancer, familial (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/614350">OMIM:614350</a> Lynch syndrome 5 - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/619097">OMIM:619097</a> Mismatch repair cancer syndrome 3 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=144">ORPHA:144</a> Lynch syndrome - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=587">ORPHA:587</a> Muir-Torre syndrome - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.505</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-48026799-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs63749893">rs63749893</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=94690%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000234420.5">MSH6:ENST00000234420.5:c.1677C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538136.1">MSH6:ENST00000538136.1:c.771C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540021.1">MSH6:ENST00000540021.1:c.1287C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405808.1">MSH6:ENST00000405808.1:c.169+8535G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PIAS4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105229">PIAS4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PIAS4%0DTCF4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCF4</a> and phenotypic similarity 0.539 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2896">Pitt-Hopkins syndrome</a> associated with TCF4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0011300, Broad fingertip</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000692, Tooth malposition</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PIAS4%0DTCF4&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TCF4</a> and phenotypic similarity 0.648 to mouse mutant of TCF4.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000433, microcephaly</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000433, microcephaly</dd><dd>HP:0000520, Proptosis - MP:0000433, microcephaly</dd><dd>HP:0000601, Hypotelorism - MP:0000433, microcephaly</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-4028798-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs755908472">rs755908472</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262971.2">PIAS4:ENST00000262971.2:c.753G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0040%</dd>
                                    <dd>ExAC AMR: 0.0174%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CEP131" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000141577">CEP131</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEP131%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/entry/614114">Mosaic variegated aneuploidy syndrome 2</a> associated with CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0004209, Clinodactyly of the 5th finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000268, Dolichocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEP131%0DCEP57&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP57</a> and phenotypic similarity 0.314 to mouse mutant of CEP57.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003051, curly tail</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004610, small vertebrae</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-79164752-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs765497684">rs765497684</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000269392.4">CEP131:ENST00000269392.4:c.2907G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374782.3">CEP131:ENST00000374782.3:c.2790G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000450824.2">CEP131:ENST00000450824.2:c.2898G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000575907.1">CEP131:ENST00000575907.1:c.2799G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                    <dd>ExAC NFE: 0.0016%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ALDH1B1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000137124">ALDH1B1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDH1B1%0DGAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GAD1</a> and phenotypic similarity 0.492 to <a href="http://www.omim.org/entry/619124">Developmental and epileptic encephalopathy 89</a> associated with GAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000308, Microretrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ALDH1B1%0DGAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GAD1</a> and phenotypic similarity 0.675 to mouse mutant of GAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0030248, narrow maxilla</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030248, narrow maxilla</dd><dd>HP:0000520, Proptosis - MP:0030248, narrow maxilla</dd><dd>HP:0000601, Hypotelorism - MP:0030248, narrow maxilla</dd><dd>HP:0000244, Brachyturricephaly - MP:0030248, narrow maxilla</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-38395910-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs745749940">rs745749940</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377698.3">ALDH1B1:ENST00000377698.3:c.165G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0042%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>gnomAD_E_AMR: 0.0238%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CELSR3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000008300">CELSR3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.313 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CELSR3">CELSR3</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000788, abnormal cerebral cortex morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CELSR3%0DPRICKLE1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PRICKLE1</a> and phenotypic similarity 0.820 to mouse mutant of PRICKLE1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002544, brachydactyly</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030475, fused lower incisors</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0030049, prominent forehead</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-48699636-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs781777002">rs781777002</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000164024.4">CELSR3:ENST00000164024.4:c.432G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544264.1">CELSR3:ENST00000544264.1:c.432G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0129%</dd>
                                    <dd>ExAC AMR: 0.0435%</dd>
                                    <dd>ExAC NFE: 0.0047%</dd>
                                    <dd>gnomAD_E_AMR: 0.0507%</dd>
                                    <dd>gnomAD_E_NFE: 0.0037%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.503</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.087</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-48699636-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs781777002">rs781777002</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000164024.4">CELSR3:ENST00000164024.4:c.432G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544264.1">CELSR3:ENST00000544264.1:c.432G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0129%</dd>
                                    <dd>ExAC AMR: 0.0435%</dd>
                                    <dd>ExAC NFE: 0.0047%</dd>
                                    <dd>gnomAD_E_AMR: 0.0507%</dd>
                                    <dd>gnomAD_E_NFE: 0.0037%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-48694471-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145229058">rs145229058</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=698158%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.075</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000164024.4">CELSR3:ENST00000164024.4:c.4059G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000544264.1">CELSR3:ENST00000544264.1:c.4059G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1597%</dd>
                                    <dd>TOPMed: 0.6013%</dd>
                                    <dd>UK10K: 1.0579%</dd>
                                    <dd>ESP AA: 0.1136%</dd>
                                    <dd>ESP EA: 0.8843%</dd>
                                    <dd>ESP All: 0.6234%</dd>
                                    <dd>ExAC AFR: 0.0936%</dd>
                                    <dd>ExAC AMR: 0.2948%</dd>
                                    <dd>ExAC FIN: 0.5584%</dd>
                                    <dd>ExAC NFE: 0.9413%</dd>
                                    <dd>ExAC OTH: 0.3529%</dd>
                                    <dd>ExAC SAS: 0.1467%</dd>
                                    <dd>gnomAD_E_AFR: 0.1135%</dd>
                                    <dd>gnomAD_E_AMR: 0.2330%</dd>
                                    <dd>gnomAD_E_FIN: 0.5641%</dd>
                                    <dd>gnomAD_E_NFE: 0.9618%</dd>
                                    <dd>gnomAD_E_OTH: 0.6315%</dd>
                                    <dd>gnomAD_E_SAS: 0.1432%</dd>
                                    <dd>gnomAD_G_AFR: 0.1492%</dd>
                                    <dd>gnomAD_G_AMR: 0.1193%</dd>
                                    <dd>gnomAD_G_FIN: 0.5158%</dd>
                                    <dd>gnomAD_G_NFE: 0.9810%</dd>
                                    <dd>gnomAD_G_OTH: 0.5102%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="MTIF2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000085760">MTIF2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTIF2%0DDKC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DKC1</a> and phenotypic similarity 0.614 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1775">Dyskeratosis congenita</a> associated with DKC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0008404, Nail dystrophy</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=MTIF2%0DDKC1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DKC1</a> and phenotypic similarity 0.310 to fish mutant of DKC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-55476582-T-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263629.4">MTIF2:ENST00000263629.4:c.930A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394600.3">MTIF2:ENST00000394600.3:c.930A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000403721.1">MTIF2:ENST00000403721.1:c.930A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="APOA5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110243">APOA5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=APOA5%0DNCOA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to NCOA6</a> and phenotypic similarity 0.672 to mouse mutant of NCOA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0001324, abnormal eye pigmentation</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009887, abnormal palatal shelf fusion at midline</dd><dd>HP:0000520, Proptosis - MP:0001324, abnormal eye pigmentation</dd><dd>HP:0000601, Hypotelorism - MP:0001324, abnormal eye pigmentation</dd><dd>HP:0000244, Brachyturricephaly - MP:0001324, abnormal eye pigmentation</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/144650">OMIM:144650</a> Hyperchylomicronemia, late-onset - autosomal dominant</dd>
<dd><a href="http://www.omim.org/entry/145750">OMIM:145750</a> Hypertriglyceridemia, susceptibility to (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-116661330-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs752477156">rs752477156</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000227665.4">APOA5:ENST00000227665.4:c.615C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542499.1">APOA5:ENST00000542499.1:c.615C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC AMR: 0.0092%</dd>
                                    <dd>ExAC NFE: 0.0017%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FRK" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111816">FRK</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FRK%0DDLG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLG1</a> and phenotypic similarity 0.598 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=199306">Cleft lip/palate</a> associated with DLG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=FRK%0DDLG1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to DLG1</a> and phenotypic similarity 0.667 to mouse mutant of DLG1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000585, kinked tail</dd><dd>HP:0000486, Strabismus - MP:0003237, abnormal lens epithelium morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004871, premaxilla hypoplasia</dd><dd>HP:0000520, Proptosis - MP:0003237, abnormal lens epithelium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0003237, abnormal lens epithelium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000440, domed cranium</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-116325143-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs777585898">rs777585898</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606080.1">FRK:ENST00000606080.1:c.363C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0038%</dd>
                                    <dd>ExAC AFR: 0.0098%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GRID1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182771">GRID1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.331 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=GRID1">GRID1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004411, decreased endocochlear potential</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004411, decreased endocochlear potential</dd><dd>HP:0000520, Proptosis - MP:0004411, decreased endocochlear potential</dd><dd>HP:0000601, Hypotelorism - MP:0004411, decreased endocochlear potential</dd><dd>HP:0000244, Brachyturricephaly - MP:0004411, decreased endocochlear potential</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GRID1%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.717 to <a href="http://www.omim.org/entry/617798">Intellectual developmental disorder, autosomal dominant 53</a> associated with CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0003097, Short femur</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GRID1%0DCAMK2A&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CAMK2A</a> and phenotypic similarity 0.251 to mouse mutant of CAMK2A.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000812, abnormal dentate gyrus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-87373299-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs150033140">rs150033140</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000327946.7">GRID1:ENST00000327946.7:c.2466C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000536331.1">GRID1:ENST00000536331.1:c.1179C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0242%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC AMR: 0.0346%</dd>
                                    <dd>ExAC EAS: 0.0464%</dd>
                                    <dd>ExAC NFE: 0.0211%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0119%</dd>
                                    <dd>gnomAD_E_EAS: 0.0174%</dd>
                                    <dd>gnomAD_E_NFE: 0.0323%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0130%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CEACAM5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105388">CEACAM5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEACAM5%0DUFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UFD1</a> and phenotypic similarity 0.680 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=567">22q11.2 deletion syndrome</a> associated with UFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001829, Foot polydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000262, Turricephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CEACAM5%0DUFD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to UFD1</a> and phenotypic similarity 0.389 to fish mutant of UFD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001052, mandibular arch skeleton morphology, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-42213898-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs781988963">rs781988963</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000221992.6">CEACAM5:ENST00000221992.6:c.364C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000398599.4">CEACAM5:ENST00000398599.4:c.364C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000405816.1">CEACAM5:ENST00000405816.1:c.364C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000598976.1">CEACAM5:ENST00000598976.1:c.364C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0038%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>ExAC SAS: 0.0065%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="INA" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000148798">INA</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=INA%0DRPS6KA6&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RPS6KA6</a> and phenotypic similarity 0.667 to mouse mutant of RPS6KA6.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000433, microcephaly</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000433, microcephaly</dd><dd>HP:0000520, Proptosis - MP:0000433, microcephaly</dd><dd>HP:0000601, Hypotelorism - MP:0000433, microcephaly</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-105037868-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs964523911">rs964523911</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000369849.4">INA:ENST00000369849.4:c.900C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0034%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC9A2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000115616">SLC9A2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC9A2%0DCLCN3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CLCN3</a> and phenotypic similarity 0.696 to <a href="http://www.omim.org/entry/619512">Neurodevelopmental disorder with hypotonia and brain abnormalities</a> associated with CLCN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0100807, Long fingers</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC9A2%0DCLCN3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CLCN3</a> and phenotypic similarity 0.535 to mouse mutant of CLCN3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005253, abnormal eye physiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002092, abnormal eye morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000807, abnormal hippocampus morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-103300756-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs201428753">rs201428753</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000233969.2">SLC9A2:ENST00000233969.2:c.1386G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_NFE: 0.0045%</dd>
                                    <dd>gnomAD_G_AFR: 0.0114%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PER1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000179094">PER1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PER1%0DTPO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TPO</a> and phenotypic similarity 0.164 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=95716">Familial thyroid dyshormonogenesis</a> associated with TPO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PER1%0DTPO&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TPO</a> and phenotypic similarity 0.908 to mouse mutant of TPO.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002750, exophthalmos</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002750, exophthalmos</dd><dd>HP:0000520, Proptosis - MP:0002750, exophthalmos</dd><dd>HP:0000601, Hypotelorism - MP:0002750, exophthalmos</dd><dd>HP:0000244, Brachyturricephaly - MP:0030049, prominent forehead</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-8046779-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs766667752">rs766667752</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000317276.4">PER1:ENST00000317276.4:c.2877C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000581082.1">PER1:ENST00000581082.1:c.2808C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>ExAC EAS: 0.0379%</dd>
                                    <dd>ExAC NFE: 0.0017%</dd>
                                    <dd>gnomAD_E_EAS: 0.0176%</dd>
                                    <dd>gnomAD_E_NFE: 0.0028%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CST4" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000101441">CST4</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CST4%0DTRAPPC10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAPPC10</a> and phenotypic similarity 0.297 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2512">Autosomal recessive primary microcephaly</a> associated with TRAPPC10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000219, Thin upper lip vermilion</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CST4%0DTRAPPC10&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAPPC10</a> and phenotypic similarity 0.676 to mouse mutant of TRAPPC10.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001297, microphthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0001297, microphthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001297, microphthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0001297, microphthalmia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>20-23669406-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs191660510">rs191660510</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000217423.3">CST4:ENST00000217423.3:c.201G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0599%</dd>
                                    <dd>TOPMed: 0.0219%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0605%</dd>
                                    <dd>ExAC NFE: 0.0105%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0596%</dd>
                                    <dd>gnomAD_E_NFE: 0.0063%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0201%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ARRDC2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000105643">ARRDC2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ARRDC2%0DKLF2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to KLF2</a> and phenotypic similarity 0.667 to mouse mutant of KLF2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000433, microcephaly</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000433, microcephaly</dd><dd>HP:0000520, Proptosis - MP:0000433, microcephaly</dd><dd>HP:0000601, Hypotelorism - MP:0000433, microcephaly</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-18123757-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140570800">rs140570800</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000222250.4">ARRDC2:ENST00000222250.4:c.1206G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000379656.3">ARRDC2:ENST00000379656.3:c.1191G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0166%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AMR: 0.0349%</dd>
                                    <dd>ExAC FIN: 0.0154%</dd>
                                    <dd>ExAC NFE: 0.0332%</dd>
                                    <dd>gnomAD_E_AMR: 0.0509%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0287%</dd>
                                    <dd>gnomAD_E_OTH: 0.0732%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                    <dd>gnomAD_G_NFE: 0.0200%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SYNE2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000054654">SYNE2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.273</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.354</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.545 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SYNE2">SYNE2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004022, abnormal cone electrophysiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002864, abnormal ocular fundus morphology</dd><dd>HP:0000520, Proptosis - MP:0002864, abnormal ocular fundus morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002864, abnormal ocular fundus morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0011965, decreased total retina thickness</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SYNE2%0DTMEM67&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TMEM67</a> and phenotypic similarity 0.560 to <a href="http://www.omim.org/entry/602152">?RHYNS syndrome</a> associated with TMEM67.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000577, Exotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0100864, Short femoral neck</dd><dd>HP:0000520, Proptosis - HP:0000490, Deeply set eye</dd><dd>HP:0000601, Hypotelorism - HP:0000490, Deeply set eye</dd><dd>HP:0000244, Brachyturricephaly - HP:0002684, Thickened calvaria</dd></dl><dl><dt>Proximity score 0.508 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SYNE2%0DTMEM67&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TMEM67</a> and phenotypic similarity 0.632 to mouse mutant of TMEM67.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000460, mandible hypoplasia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000460, mandible hypoplasia</dd><dd>HP:0000520, Proptosis - MP:0000460, mandible hypoplasia</dd><dd>HP:0000601, Hypotelorism - MP:0000460, mandible hypoplasia</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/612999">OMIM:612999</a> Emery-Dreifuss muscular dystrophy 5, autosomal dominant - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=98853">ORPHA:98853</a> Autosomal dominant Emery-Dreifuss muscular dystrophy - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.273</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.354</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-64610629-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142206385">rs142206385</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=841637%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.383</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344113.4">SYNE2:ENST00000344113.4:c.15446G&gt;A:p.(Arg5149His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357395.3">SYNE2:ENST00000357395.3:c.4601G&gt;A:p.(Arg1534His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358025.3">SYNE2:ENST00000358025.3:c.15446G&gt;A:p.(Arg5149His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394768.2">SYNE2:ENST00000394768.2:c.4601G&gt;A:p.(Arg1534His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554584.1">SYNE2:ENST00000554584.1:c.15197G&gt;A:p.(Arg5066His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555002.1">SYNE2:ENST00000555002.1:c.5348G&gt;A:p.(Arg1783His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542956.1">SYNE2:ENST00000542956.1:c.1407-58859C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.39006042</dd>
                                    
                                    <dd>REVEL: 0.106</dd>
                                    <dd>MVP: 0.390</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0189%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AMR: 0.0518%</dd>
                                    <dd>ExAC FIN: 0.0454%</dd>
                                    <dd>ExAC NFE: 0.0150%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.1256%</dd>
                                    <dd>gnomAD_E_NFE: 0.0125%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_FIN: 0.0859%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-64604650-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147640636">rs147640636</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=195843%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.326</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000344113.4">SYNE2:ENST00000344113.4:c.14792A&gt;G:p.(Lys4931Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357395.3">SYNE2:ENST00000357395.3:c.3947A&gt;G:p.(Lys1316Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358025.3">SYNE2:ENST00000358025.3:c.14792A&gt;G:p.(Lys4931Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394768.2">SYNE2:ENST00000394768.2:c.3947A&gt;G:p.(Lys1316Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000554584.1">SYNE2:ENST00000554584.1:c.14543A&gt;G:p.(Lys4848Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555002.1">SYNE2:ENST00000555002.1:c.4694A&gt;G:p.(Lys1565Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542956.1">SYNE2:ENST00000542956.1:c.1407-52880T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.33786896</dd>
                                    
                                    <dd>REVEL: 0.039</dd>
                                    <dd>MVP: 0.338</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.1084%</dd>
                                    <dd>UK10K: 0.2380%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.1395%</dd>
                                    <dd>ESP All: 0.1000%</dd>
                                    <dd>ExAC AFR: 0.0289%</dd>
                                    <dd>ExAC AMR: 0.0347%</dd>
                                    <dd>ExAC NFE: 0.1187%</dd>
                                    <dd>ExAC OTH: 0.1104%</dd>
                                    <dd>ExAC SAS: 0.0182%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0506%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.1317%</dd>
                                    <dd>gnomAD_E_OTH: 0.1825%</dd>
                                    <dd>gnomAD_E_SAS: 0.0195%</dd>
                                    <dd>gnomAD_G_AFR: 0.0459%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.1265%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OR9I1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000172377">OR9I1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=OR9I1%0DSTAG2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to STAG2</a> and phenotypic similarity 0.659 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93925">Alobar holoprosencephaly</a> associated with STAG2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0006315, Solitary median maxillary central incisor</dd><dd>HP:0000520, Proptosis - HP:0009914, Cyclopia</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.500</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-57886441-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139300657">rs139300657</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2335363%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.114</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302610.1">OR9I1:ENST00000302610.1:c.476G&gt;A:p.(Arg159His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335397.3">OR9I1:ENST00000335397.3:c.-14-60462C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.116</dd>
                                    
                                    <dd>REVEL: 0.116</dd>
                                    <dd>MVP: 0.043</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0333%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0582%</dd>
                                    <dd>ESP All: 0.0462%</dd>
                                    <dd>ExAC AFR: 0.0097%</dd>
                                    <dd>ExAC AMR: 0.0347%</dd>
                                    <dd>ExAC EAS: 0.0236%</dd>
                                    <dd>ExAC NFE: 0.0420%</dd>
                                    <dd>ExAC SAS: 0.0303%</dd>
                                    <dd>gnomAD_E_AMR: 0.0626%</dd>
                                    <dd>gnomAD_E_EAS: 0.0175%</dd>
                                    <dd>gnomAD_E_NFE: 0.0441%</dd>
                                    <dd>gnomAD_E_OTH: 0.0367%</dd>
                                    <dd>gnomAD_E_SAS: 0.0455%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_EAS: 0.0620%</dd>
                                    <dd>gnomAD_G_NFE: 0.0534%</dd>
                                    <dd>gnomAD_G_OTH: 0.1025%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>11-57886280-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs113763198">rs113763198</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.074</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000302610.1">OR9I1:ENST00000302610.1:c.637G&gt;A:p.(Val213Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000335397.3">OR9I1:ENST00000335397.3:c.-14-60623C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.124</dd>
                                    
                                    <dd>REVEL: 0.124</dd>
                                    <dd>MVP: 0.043</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3594%</dd>
                                    <dd>TOPMed: 0.6528%</dd>
                                    <dd>UK10K: 1.3885%</dd>
                                    <dd>ESP AA: 0.2045%</dd>
                                    <dd>ESP EA: 1.1057%</dd>
                                    <dd>ESP All: 0.8004%</dd>
                                    <dd>ExAC AFR: 0.2126%</dd>
                                    <dd>ExAC AMR: 0.2778%</dd>
                                    <dd>ExAC EAS: 0.0470%</dd>
                                    <dd>ExAC FIN: 0.7268%</dd>
                                    <dd>ExAC NFE: 0.9374%</dd>
                                    <dd>ExAC OTH: 0.5531%</dd>
                                    <dd>ExAC SAS: 0.3275%</dd>
                                    <dd>gnomAD_E_AFR: 0.1965%</dd>
                                    <dd>gnomAD_E_AMR: 0.3729%</dd>
                                    <dd>gnomAD_E_EAS: 0.0409%</dd>
                                    <dd>gnomAD_E_FIN: 0.5785%</dd>
                                    <dd>gnomAD_E_NFE: 1.0142%</dd>
                                    <dd>gnomAD_E_OTH: 0.6606%</dd>
                                    <dd>gnomAD_E_SAS: 0.2958%</dd>
                                    <dd>gnomAD_G_AFR: 0.2748%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_FIN: 1.1162%</dd>
                                    <dd>gnomAD_G_NFE: 0.8062%</dd>
                                    <dd>gnomAD_G_OTH: 0.6110%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="OPLAH" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178814">OPLAH</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=5.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.491</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.094</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.491 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=OPLAH">OPLAH</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/260005">OMIM:260005</a> 5-oxoprolinase deficiency - autosomal dominant/recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=5.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.491</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.094</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-145112442-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs781841448">rs781841448</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1893738%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.094</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426825.1">OPLAH:ENST00000426825.1:c.1331C&gt;T:p.(Pro444Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.094</dd>
                                    
                                    <dd>REVEL: 0.094</dd>
                                    <dd>MVP: 0.055</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0064%</dd>
                                    <dd>ExAC AMR: 0.0146%</dd>
                                    <dd>ExAC NFE: 0.0022%</dd>
                                    <dd>ExAC SAS: 0.0109%</dd>
                                    <dd>gnomAD_E_AMR: 0.0066%</dd>
                                    <dd>gnomAD_E_EAS: 0.0061%</dd>
                                    <dd>gnomAD_E_NFE: 0.0010%</dd>
                                    <dd>gnomAD_E_OTH: 0.0208%</dd>
                                    <dd>gnomAD_E_SAS: 0.0074%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HCAR2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182782">HCAR2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.063</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HCAR2%0DCCR3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCR3</a> and phenotypic similarity 0.688 to mouse mutant of CCR3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0002110, abnormal digit morphology</dd><dd>HP:0000486, Strabismus - MP:0001312, abnormal cornea morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001312, abnormal cornea morphology</dd><dd>HP:0000520, Proptosis - MP:0001312, abnormal cornea morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001312, abnormal cornea morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001312, abnormal cornea morphology</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.501</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.063</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-123187313-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs144376493">rs144376493</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.070</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328880.5">HCAR2:ENST00000328880.5:c.518G&gt;C:p.(Gly173Ala)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356987.2">HCAR2:ENST00000356987.2:c.*294+6983G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.07629995</dd>
                                    
                                    <dd>REVEL: 0.056</dd>
                                    <dd>MVP: 0.076</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.2023%</dd>
                                    <dd>ExAC AFR: 0.0384%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC FIN: 0.2419%</dd>
                                    <dd>ExAC NFE: 0.2907%</dd>
                                    <dd>ExAC SAS: 0.0424%</dd>
                                    <dd>gnomAD_E_AFR: 0.0327%</dd>
                                    <dd>gnomAD_E_AMR: 0.0715%</dd>
                                    <dd>gnomAD_E_FIN: 0.1526%</dd>
                                    <dd>gnomAD_E_NFE: 0.2993%</dd>
                                    <dd>gnomAD_E_OTH: 0.0730%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                    <dd>gnomAD_G_AFR: 0.0573%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_FIN: 0.0859%</dd>
                                    <dd>gnomAD_G_NFE: 0.2666%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-123187314-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs141052038">rs141052038</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.056</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000328880.5">HCAR2:ENST00000328880.5:c.517G&gt;A:p.(Gly173Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000356987.2">HCAR2:ENST00000356987.2:c.*294+6982G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.061188463</dd>
                                    
                                    <dd>REVEL: 0.048</dd>
                                    <dd>MVP: 0.061</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1198%</dd>
                                    <dd>TOPMed: 0.2071%</dd>
                                    <dd>ExAC AFR: 0.0673%</dd>
                                    <dd>ExAC AMR: 0.0173%</dd>
                                    <dd>ExAC FIN: 0.2419%</dd>
                                    <dd>ExAC NFE: 0.2952%</dd>
                                    <dd>ExAC SAS: 0.0424%</dd>
                                    <dd>gnomAD_E_AFR: 0.0654%</dd>
                                    <dd>gnomAD_E_AMR: 0.0715%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_FIN: 0.1481%</dd>
                                    <dd>gnomAD_E_NFE: 0.3029%</dd>
                                    <dd>gnomAD_E_OTH: 0.0730%</dd>
                                    <dd>gnomAD_E_SAS: 0.0325%</dd>
                                    <dd>gnomAD_G_AFR: 0.0687%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_FIN: 0.0859%</dd>
                                    <dd>gnomAD_G_NFE: 0.2666%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AUNIP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000127423">AUNIP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.625</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.625</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-26161677-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs764073268">rs764073268</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.625</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000374298.3">AUNIP:ENST00000374298.3:c.881G&gt;A:p.(Gly294Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000538789.1">AUNIP:ENST00000538789.1:c.881G&gt;A:p.(Gly294Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.62612754</dd>
                                    
                                    <dd>REVEL: 0.446</dd>
                                    <dd>MVP: 0.626</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0019%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0060%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0054%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF597" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167981">ZNF597</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.001</b>
                    <data>(p=6.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.459</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.459 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=ZNF597">ZNF597</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004222, iris synechia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004222, iris synechia</dd><dd>HP:0000520, Proptosis - MP:0004222, iris synechia</dd><dd>HP:0000601, Hypotelorism - MP:0004222, iris synechia</dd><dd>HP:0000244, Brachyturricephaly - MP:0004222, iris synechia</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.001</b>
                                    <data>(p=6.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.459</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-3487470-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369212733">rs369212733</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301744.4">ZNF597:ENST00000301744.4:c.229T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0077%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GLI1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000111087">GLI1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.260</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.320</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.491 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=289">Ellis Van Creveld syndrome</a> associated with GLI1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001829, Foot polydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000691, Microdontia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.376 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=GLI1">GLI1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - ZP:0000620, eye mislocalised medially, abnormal</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - ZP:0000620, eye mislocalised medially, abnormal</dd><dd>HP:0000601, Hypotelorism - ZP:0000620, eye mislocalised medially, abnormal</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLI1%0DRAB23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB23</a> and phenotypic similarity 0.656 to <a href="http://www.omim.org/entry/201000">Carpenter syndrome</a> associated with RAB23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.521 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=GLI1%0DRAB23&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to RAB23</a> and phenotypic similarity 0.653 to mouse mutant of RAB23.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000562, polydactyly</dd><dd>HP:0000486, Strabismus - MP:0002092, abnormal eye morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002092, abnormal eye morphology</dd><dd>HP:0000520, Proptosis - MP:0002092, abnormal eye morphology</dd><dd>HP:0000601, Hypotelorism - MP:0002092, abnormal eye morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0002092, abnormal eye morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/174400">OMIM:174400</a> Polydactyly, preaxial I - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618123">OMIM:618123</a> Polydactyly, postaxial, type A8 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=289">ORPHA:289</a> Ellis Van Creveld syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.260</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.320</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-57863479-C-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.320</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000228682.2">GLI1:ENST00000228682.2:c.1574C&gt;G:p.(Thr525Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543426.1">GLI1:ENST00000543426.1:c.1190C&gt;G:p.(Thr397Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000546141.1">GLI1:ENST00000546141.1:c.1451C&gt;G:p.(Thr484Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.31997085</dd>
                                    
                                    <dd>REVEL: 0.055</dd>
                                    <dd>MVP: 0.320</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LAMA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196569">LAMA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.542</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.299 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=258">Laminin subunit alpha 2-related congenital muscular dystrophy</a> associated with LAMA2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0010754, Abnormality of the temporomandibular joint</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.542 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=LAMA2">LAMA2</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000520, Proptosis - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000441, increased cranium width</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LAMA2%0DLAMA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMA5</a> and phenotypic similarity 0.351 to <a href="http://www.omim.org/entry/620076">?Bent bone dysplasia syndrome 2</a> associated with LAMA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0010034, Short 1st metacarpal</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000773, Short ribs</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0005474, Decreased calvarial ossification</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=LAMA2%0DLAMA5&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LAMA5</a> and phenotypic similarity 0.744 to mouse mutant of LAMA5.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000564, syndactyly</dd><dd>HP:0000486, Strabismus - MP:0001293, anophthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001293, anophthalmia</dd><dd>HP:0000520, Proptosis - MP:0001293, anophthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001293, anophthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0001293, anophthalmia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/607855">OMIM:607855</a> Muscular dystrophy, congenital, merosin deficient or partially deficient - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/618138">OMIM:618138</a> Muscular dystrophy, limb-girdle, autosomal recessive 23 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=258">ORPHA:258</a> Laminin subunit alpha 2-related congenital muscular dystrophy - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=258">ORPHA:258</a> Laminin subunit alpha 2-related congenital muscular dystrophy - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.271</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-129813620-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs950138001">rs950138001</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421865.2">LAMA2:ENST00000421865.2:c.8236C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNAAF3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167646">DNAAF3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.518</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.319 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with DNAAF3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001217, Clubbing</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0011109, Chronic sinusitis</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAAF3%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.319 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">Primary ciliary dyskinesia</a> associated with CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001217, Clubbing</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0011109, Chronic sinusitis</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNAAF3%0DCCDC39&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CCDC39</a> and phenotypic similarity 0.647 to mouse mutant of CCDC39.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002639, micrognathia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0002639, micrognathia</dd><dd>HP:0000601, Hypotelorism - MP:0002639, micrognathia</dd><dd>HP:0000244, Brachyturricephaly - MP:0002639, micrognathia</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/606763">OMIM:606763</a> Ciliary dyskinesia, primary, 2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=244">ORPHA:244</a> Primary ciliary dyskinesia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.259</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.217</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-55677895-A-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs748529039">rs748529039</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.217</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391720.4">DNAAF3:ENST00000391720.4:c.122T&gt;G:p.(Ile41Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000527223.2">DNAAF3:ENST00000527223.2:c.122T&gt;G:p.(Ile41Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000524407.2">DNAAF3:ENST00000524407.2:c.-9T&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.21737058</dd>
                                    
                                    <dd>REVEL: 0.035</dd>
                                    <dd>MVP: 0.217</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="XYLT2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000015532">XYLT2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.513</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.483 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85194">Spondylo-ocular syndrome</a> associated with XYLT2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000639, Nystagmus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000343, Long philtrum</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=XYLT2%0DB3GAT3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B3GAT3</a> and phenotypic similarity 0.653 to <a href="http://www.omim.org/entry/245600">Multiple joint dislocations, short stature, craniofacial dysmorphism, with or without congenital heart defects</a> associated with B3GAT3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001852, Sandal gap</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000308, Microretrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=XYLT2%0DB3GAT3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B3GAT3</a> and phenotypic similarity 0.317 to mouse mutant of B3GAT3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003109, short femur</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0003109, short femur</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.513 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=XYLT2%0DB3GAT3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to B3GAT3</a> and phenotypic similarity 0.433 to fish mutant of B3GAT3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0002322, mandibular arch skeleton decreased length, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/264800">OMIM:264800</a> Pseudoxanthoma elasticum, modifier of severity of (susceptibility)</dd>
<dd><a href="http://www.omim.org/entry/605822">OMIM:605822</a> Spondyloocular syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=85194">ORPHA:85194</a> Spondylo-ocular syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.256</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.202</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-48435851-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369581548">rs369581548</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2122155%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.202</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000017003.2">XYLT2:ENST00000017003.2:c.2225G&gt;A:p.(Arg742His)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000507602.1">XYLT2:ENST00000507602.1:c.1941+1238G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20208623</dd>
                                    
                                    <dd>REVEL: 0.037</dd>
                                    <dd>MVP: 0.202</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0091%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC FIN: 0.0154%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0208%</dd>
                                    <dd>gnomAD_E_EAS: 0.0174%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PKHD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170927">PKHD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.506</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.358 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=731">Autosomal recessive polycystic kidney disease</a> associated with PKHD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.198 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PKHD1">PKHD1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0002152, abnormal brain morphology</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKHD1%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.647 to mouse mutant of PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0002639, micrognathia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0002639, micrognathia</dd><dd>HP:0000601, Hypotelorism - MP:0002639, micrognathia</dd><dd>HP:0000244, Brachyturricephaly - MP:0002639, micrognathia</dd></dl><dl><dt>Proximity score 0.506 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=PKHD1%0DPKD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PKD1</a> and phenotypic similarity 0.266 to fish mutant of PKD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001708, mandibular arch skeleton malformed, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/263200">OMIM:263200</a> Polycystic kidney disease 4, with or without hepatic disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=53035">ORPHA:53035</a> Caroli disease - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=731">ORPHA:731</a> Autosomal recessive polycystic kidney disease - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.253</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-51893037-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1581827926">rs1581827926</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=686884%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000340994.4">PKHD1:ENST00000340994.4:c.3477C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371117.3">PKHD1:ENST00000371117.3:c.3477C&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF526" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167625">ZNF526</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.504 to <a href="http://www.omim.org/entry/619877">Dentici-Novelli neurodevelopmental syndrome</a> associated with ZNF526.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000307, Pointed chin</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF526%0DTRAF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAF7</a> and phenotypic similarity 0.605 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2495">Meningioma</a> associated with TRAF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0012246, Oculomotor nerve palsy</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000520, Proptosis</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ZNF526%0DTRAF7&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to TRAF7</a> and phenotypic similarity 0.491 to mouse mutant of TRAF7.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/619877">OMIM:619877</a> Dentici-Novelli neurodevelopmental syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.271</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-42729069-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143231579">rs143231579</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1780396%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.271</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301215.3">ZNF526:ENST00000301215.3:c.514C&gt;A:p.(Pro172Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.27220586</dd>
                                    
                                    <dd>REVEL: 0.042</dd>
                                    <dd>MVP: 0.272</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0098%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AMR: 0.0357%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0162%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="UFC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143222">UFC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.504</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.362 to <a href="http://www.omim.org/entry/618076">Neurodevelopmental disorder with spasticity and poor growth</a> associated with UFC1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000657, Oculomotor apraxia</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UFC1%0DUSP9X&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to USP9X</a> and phenotypic similarity 0.688 to <a href="http://www.omim.org/entry/300968">Intellectual developmental disorder, X-linked 99, syndromic, female-restricted</a> associated with USP9X.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0100259, Postaxial polydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000193, Bifid uvula</dd><dd>HP:0000520, Proptosis - HP:0000601, Hypotelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000601, Hypotelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000248, Brachycephaly</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=UFC1%0DUSP9X&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to USP9X</a> and phenotypic similarity 0.150 to mouse mutant of USP9X.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000929, open neural tube</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618076">OMIM:618076</a> Neurodevelopmental disorder with spasticity and poor growth - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-161126745-G-A [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368003.5">UFC1:ENST00000368003.5:c.129G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DPYS" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000147647">DPYS</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.503</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.382 to <a href="http://www.omim.org/entry/222748">Dihydropyrimidinuria</a> associated with DPYS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0009803, Short phalanx of finger</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0009803, Short phalanx of finger</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0001357, Plagiocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DPYS%0DPFAS&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PFAS</a> and phenotypic similarity 0.772 to mouse mutant of PFAS.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0005175, non-pigmented tail tip</dd><dd>HP:0000486, Strabismus - MP:0001300, ocular hypertelorism</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030466, alveolar process atrophy</dd><dd>HP:0000520, Proptosis - MP:0001300, ocular hypertelorism</dd><dd>HP:0000601, Hypotelorism - MP:0001300, ocular hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - MP:0011862, decreased cranium length</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/222748">OMIM:222748</a> Dihydropyrimidinuria - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-105459708-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs757754514">rs757754514</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351513.2">DPYS:ENST00000351513.2:c.447T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SMARCAL1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138375">SMARCAL1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.501</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.308 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1830">Schimke immuno-osseous dysplasia</a> associated with SMARCAL1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000691, Microdontia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.392 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=SMARCAL1">SMARCAL1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003055, abnormal long bone epiphyseal plate morphology</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004704, short vertebral column</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.373 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=SMARCAL1">SMARCAL1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0000062, palatoquadrate cartilage decreased size, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMARCAL1%0DATR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATR</a> and phenotypic similarity 0.517 to <a href="http://www.omim.org/entry/210600">Seckel syndrome 1</a> associated with ATR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0006143, Abnormal finger flexion creases</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMARCAL1%0DATR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATR</a> and phenotypic similarity 0.680 to mouse mutant of ATR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000120, malocclusion</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0000120, malocclusion</dd><dd>HP:0000601, Hypotelorism - MP:0000120, malocclusion</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dl><dt>Proximity score 0.501 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SMARCAL1%0DATR&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ATR</a> and phenotypic similarity 0.310 to fish mutant of ATR.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/242900">OMIM:242900</a> Schimke immunoosseous dysplasia - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=1830">ORPHA:1830</a> Schimke immuno-osseous dysplasia - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.251</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-217280117-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs757584821">rs757584821</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=781164%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000357276.4">SMARCAL1:ENST00000357276.4:c.690C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358207.5">SMARCAL1:ENST00000358207.5:c.690C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TBCE" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000284770">TBCE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.486 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93324">Autosomal recessive Kenny-Caffey syndrome</a> associated with TBCE.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000670, Carious teeth</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Phenotypic similarity 0.491 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=TBCE">TBCE</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001303, abnormal lens morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001303, abnormal lens morphology</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001303, abnormal lens morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBCE%0DPSAT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSAT1</a> and phenotypic similarity 0.678 to <a href="http://www.omim.org/entry/616038">Neu-Laxova syndrome 2</a> associated with PSAT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001770, Toe syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TBCE%0DPSAT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSAT1</a> and phenotypic similarity 0.175 to mouse mutant of PSAT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000914, exencephaly</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/241410">OMIM:241410</a> Hypoparathyroidism-retardation-dysmorphism syndrome - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/244460">OMIM:244460</a> Kenny-Caffey syndrome, type 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/617207">OMIM:617207</a> Encephalopathy, progressive, with amyotrophy and optic atrophy - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2323">ORPHA:2323</a> Sanjad-Sakati syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=496756">ORPHA:496756</a> Early-onset progressive encephalopathy-spastic ataxia-distal spinal muscular atrophy syndrome - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93324">ORPHA:93324</a> Autosomal recessive Kenny-Caffey syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.268</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-235602095-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs751036181">rs751036181</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.268</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000366601.3">TBCE:ENST00000366601.3:c.1128G&gt;C:p.(Glu376Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000406207.1">TBCE:ENST00000406207.1:c.1128G&gt;C:p.(Glu376Asp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543662.1">TBCE:ENST00000543662.1:c.1281G&gt;C:p.(Glu427Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.26911122</dd>
                                    
                                    <dd>REVEL: 0.036</dd>
                                    <dd>MVP: 0.269</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0024%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0036%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DNMBP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107554">DNMBP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.500</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.000</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.400 to <a href="http://www.omim.org/entry/618415">Cataract 48</a> associated with DNMBP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - HP:0000577, Exotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Phenotypic similarity 0.470 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=DNMBP">DNMBP</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000520, Proptosis - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000601, Hypotelorism - MP:0001289, persistence of hyaloid vascular system</dd><dd>HP:0000244, Brachyturricephaly - MP:0008259, abnormal optic disk morphology</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=DNMBP%0DFAT1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to FAT1</a> and phenotypic similarity 0.751 to mouse mutant of FAT1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0005163, cyclopia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001293, anophthalmia</dd><dd>HP:0000520, Proptosis - MP:0005163, cyclopia</dd><dd>HP:0000601, Hypotelorism - MP:0005163, cyclopia</dd><dd>HP:0000244, Brachyturricephaly - MP:0006290, proboscis</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618415">OMIM:618415</a> Cataract 48 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.180</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-101639648-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs199968530">rs199968530</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.180</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000324109.4">DNMBP:ENST00000324109.4:c.4468G&gt;A:p.(Val1490Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000342239.3">DNMBP:ENST00000342239.3:c.4540G&gt;A:p.(Val1514Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000540316.1">DNMBP:ENST00000540316.1:c.1276G&gt;A:p.(Val426Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000543621.1">DNMBP:ENST00000543621.1:c.2206G&gt;A:p.(Val736Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.18167952</dd>
                                    
                                    <dd>REVEL: 0.015</dd>
                                    <dd>MVP: 0.182</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0333%</dd>
                                    <dd>UK10K: 0.0793%</dd>
                                    <dd>ESP EA: 0.0349%</dd>
                                    <dd>ESP All: 0.0231%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC NFE: 0.0494%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AFR: 0.0196%</dd>
                                    <dd>gnomAD_E_AMR: 0.0417%</dd>
                                    <dd>gnomAD_E_NFE: 0.0654%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_E_SAS: 0.0130%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                    <dd>gnomAD_G_NFE: 0.0333%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TAPBP" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000231925">TAPBP</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=6.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.193</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.316</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.193 to <a href="http://www.omim.org/entry/604571">Bare lymphocyte syndrome, type I</a> associated with TAPBP.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0011109, Chronic sinusitis</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/604571">OMIM:604571</a> Bare lymphocyte syndrome, type I - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=6.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.193</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.316</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            
                            <dl class="list-group">
                                <!--                            th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}">-->
                                <dt>
                                    Phenotype matches to diseases consistent with this MOI:
                                </dt>
                                <!--                            <dt th:each="diseaseModel: ${geneScore.getCompatibleDiseaseMatches()}"-->
                                <!--                                th:text="Phenotypic similarity ${diseaseModel.getScore()} to ${diseaseModel.getModel().getDiseaseName()}">-->
                                <!--                                Phenotypic similarity 0.95 to APERT SYNDROME-->
                                <!--                            </dt>-->
                                
                                    <dt>Phenotypic similarity 0.193 to OMIM:604571 Bare lymphocyte syndrome, type I</dt>
                                
                            </dl>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-33281504-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45583737">rs45583737</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=721987%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.552</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426633.2">TAPBP:ENST00000426633.2:c.175G&gt;T:p.(Asp59Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434618.2">TAPBP:ENST00000434618.2:c.175G&gt;T:p.(Asp59Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456592.2">TAPBP:ENST00000456592.2:c.175G&gt;T:p.(Asp59Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000475304.1">TAPBP:ENST00000475304.1:c.175G&gt;T:p.(Asp59Tyr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000489157.1">TAPBP:ENST00000489157.1:c.175G&gt;T:p.(Asp59Tyr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.6935195</dd>
                                    
                                    <dd>REVEL: 0.333</dd>
                                    <dd>MVP: 0.694</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.4484%</dd>
                                    <dd>UK10K: 0.5554%</dd>
                                    <dd>ESP AA: 0.1365%</dd>
                                    <dd>ESP EA: 0.7572%</dd>
                                    <dd>ESP All: 0.5471%</dd>
                                    <dd>ExAC AFR: 0.1755%</dd>
                                    <dd>ExAC AMR: 0.2263%</dd>
                                    <dd>ExAC FIN: 0.1524%</dd>
                                    <dd>ExAC NFE: 0.7848%</dd>
                                    <dd>ExAC OTH: 0.3472%</dd>
                                    <dd>ExAC SAS: 0.2761%</dd>
                                    <dd>gnomAD_E_AFR: 0.1355%</dd>
                                    <dd>gnomAD_E_AMR: 0.3189%</dd>
                                    <dd>gnomAD_E_EAS: 0.0175%</dd>
                                    <dd>gnomAD_E_FIN: 0.1712%</dd>
                                    <dd>gnomAD_E_NFE: 0.7598%</dd>
                                    <dd>gnomAD_E_OTH: 0.7353%</dd>
                                    <dd>gnomAD_E_SAS: 0.2406%</dd>
                                    <dd>gnomAD_G_AFR: 0.1150%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_FIN: 0.1720%</dd>
                                    <dd>gnomAD_G_NFE: 0.5418%</dd>
                                    <dd>gnomAD_G_OTH: 0.9184%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>6-33281505-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs45501592">rs45501592</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP6_Strong]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=721988%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.080</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000426633.2">TAPBP:ENST00000426633.2:c.174G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434618.2">TAPBP:ENST00000434618.2:c.174G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000456592.2">TAPBP:ENST00000456592.2:c.174G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000475304.1">TAPBP:ENST00000475304.1:c.174G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000489157.1">TAPBP:ENST00000489157.1:c.174G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.3395%</dd>
                                    <dd>TOPMed: 0.4500%</dd>
                                    <dd>UK10K: 0.5554%</dd>
                                    <dd>ESP AA: 0.1366%</dd>
                                    <dd>ESP EA: 0.7572%</dd>
                                    <dd>ESP All: 0.5472%</dd>
                                    <dd>ExAC AFR: 0.1756%</dd>
                                    <dd>ExAC AMR: 0.2263%</dd>
                                    <dd>ExAC FIN: 0.1523%</dd>
                                    <dd>ExAC NFE: 0.7853%</dd>
                                    <dd>ExAC OTH: 0.3472%</dd>
                                    <dd>ExAC SAS: 0.2759%</dd>
                                    <dd>gnomAD_E_AFR: 0.1357%</dd>
                                    <dd>gnomAD_E_AMR: 0.3189%</dd>
                                    <dd>gnomAD_E_EAS: 0.0175%</dd>
                                    <dd>gnomAD_E_FIN: 0.1711%</dd>
                                    <dd>gnomAD_E_NFE: 0.7614%</dd>
                                    <dd>gnomAD_E_OTH: 0.7358%</dd>
                                    <dd>gnomAD_E_SAS: 0.2406%</dd>
                                    <dd>gnomAD_G_AFR: 0.1150%</dd>
                                    <dd>gnomAD_G_AMR: 0.4773%</dd>
                                    <dd>gnomAD_G_FIN: 0.1720%</dd>
                                    <dd>gnomAD_G_NFE: 0.5416%</dd>
                                    <dd>gnomAD_G_OTH: 0.9184%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SH2D4B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000178217">SH2D4B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.495</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.495</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-82363337-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs773754216">rs773754216</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.495</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000313455.4">SH2D4B:ENST00000313455.4:c.499C&gt;T:p.(Arg167Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000339284.2">SH2D4B:ENST00000339284.2:c.646C&gt;T:p.(Arg216Cys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000470604.2">SH2D4B:ENST00000470604.2:c.643C&gt;T:p.(Arg215Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.49590495</dd>
                                    
                                    <dd>REVEL: 0.132</dd>
                                    <dd>MVP: 0.496</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_FIN: 0.0066%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AKR1C1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000187134">AKR1C1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.177</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.274</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.177 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=AKR1C1">AKR1C1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0009808, decreased oligodendrocyte number</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.177</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.274</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>10-5011081-A-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs760575583">rs760575583</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.274</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380859.1">AKR1C1:ENST00000380859.1:c.521A&gt;T:p.(Gln174Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000380872.4">AKR1C1:ENST00000380872.4:c.515A&gt;T:p.(Gln172Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000434459.2">AKR1C1:ENST00000434459.2:c.515A&gt;T:p.(Gln172Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.27436614</dd>
                                    
                                    <dd>REVEL: 0.185</dd>
                                    <dd>MVP: 0.274</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0106%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0107%</dd>
                                    <dd>gnomAD_E_OTH: 0.0182%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ADCY10" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000143199">ADCY10</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.144</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY10%0DPDE6D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDE6D</a> and phenotypic similarity 0.628 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2754">Orofaciodigital syndrome type 6</a> associated with PDE6D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001159, Syndactyly</dd><dd>HP:0000486, Strabismus - HP:0000565, Esotropia</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0004422, Biparietal narrowing</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=ADCY10%0DPDE6D&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PDE6D</a> and phenotypic similarity 0.548 to mouse mutant of PDE6D.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0004022, abnormal cone electrophysiology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000520, Proptosis - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000601, Hypotelorism - MP:0005551, abnormal eye electrophysiology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001326, retina degeneration</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/143870">OMIM:143870</a> Hypercalciuria, absorptive, susceptibility to (susceptibility)</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2197">ORPHA:2197</a> Idiopathic hypercalciuria - autosomal dominant</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.144</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-167787479-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs41270737">rs41270737</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=696239%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.280</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367848.1">ADCY10:ENST00000367848.1:c.4037A&gt;G:p.(Asn1346Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367851.4">ADCY10:ENST00000367851.4:c.4313A&gt;G:p.(Asn1438Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545172.1">ADCY10:ENST00000545172.1:c.3854A&gt;G:p.(Asn1285Ser)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.46478412</dd>
                                    
                                    <dd>REVEL: 0.055</dd>
                                    <dd>MVP: 0.465</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 0.7783%</dd>
                                    <dd>UK10K: 1.0315%</dd>
                                    <dd>ESP AA: 0.3404%</dd>
                                    <dd>ESP EA: 1.3721%</dd>
                                    <dd>ESP All: 1.0226%</dd>
                                    <dd>ExAC AFR: 0.2316%</dd>
                                    <dd>ExAC AMR: 0.7949%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 0.6501%</dd>
                                    <dd>ExAC NFE: 1.0626%</dd>
                                    <dd>ExAC OTH: 0.7743%</dd>
                                    <dd>ExAC SAS: 0.7268%</dd>
                                    <dd>gnomAD_E_AFR: 0.2157%</dd>
                                    <dd>gnomAD_E_AMR: 0.6581%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_FIN: 0.6906%</dd>
                                    <dd>gnomAD_E_NFE: 1.1047%</dd>
                                    <dd>gnomAD_E_OTH: 1.1848%</dd>
                                    <dd>gnomAD_E_SAS: 0.6660%</dd>
                                    <dd>gnomAD_G_AFR: 0.2290%</dd>
                                    <dd>gnomAD_G_AMR: 0.7160%</dd>
                                    <dd>gnomAD_G_FIN: 0.8014%</dd>
                                    <dd>gnomAD_G_NFE: 1.1454%</dd>
                                    <dd>gnomAD_G_OTH: 1.2220%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-167825449-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs112466731">rs112466731</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                <span class="label label-success">BENIGN_OR_LIKELY_BENIGN</span>
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1153348%5Balleleid%5D">(criteria provided, multiple submitters, no conflicts)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.008</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367848.1">ADCY10:ENST00000367848.1:c.1849A&gt;G:p.(Ile617Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000367851.4">ADCY10:ENST00000367851.4:c.2125A&gt;G:p.(Ile709Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000545172.1">ADCY10:ENST00000545172.1:c.1666A&gt;G:p.(Ile556Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.029</dd>
                                    
                                    <dd>REVEL: 0.029</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5391%</dd>
                                    <dd>TOPMed: 0.8531%</dd>
                                    <dd>UK10K: 1.4679%</dd>
                                    <dd>ESP AA: 0.3177%</dd>
                                    <dd>ESP EA: 1.4535%</dd>
                                    <dd>ESP All: 1.0687%</dd>
                                    <dd>ExAC AFR: 0.2018%</dd>
                                    <dd>ExAC AMR: 0.5096%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC FIN: 1.7236%</dd>
                                    <dd>ExAC NFE: 1.4416%</dd>
                                    <dd>ExAC OTH: 0.7709%</dd>
                                    <dd>ExAC SAS: 0.5875%</dd>
                                    <dd>gnomAD_E_AFR: 0.1503%</dd>
                                    <dd>gnomAD_E_AMR: 0.5122%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 1.7580%</dd>
                                    <dd>gnomAD_E_NFE: 1.5098%</dd>
                                    <dd>gnomAD_E_OTH: 1.0953%</dd>
                                    <dd>gnomAD_E_SAS: 0.6075%</dd>
                                    <dd>gnomAD_G_AFR: 0.2065%</dd>
                                    <dd>gnomAD_G_AMR: 0.5981%</dd>
                                    <dd>gnomAD_G_FIN: 1.8603%</dd>
                                    <dd>gnomAD_G_NFE: 1.8385%</dd>
                                    <dd>gnomAD_G_OTH: 1.0225%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PLCB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000182621">PLCB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.285</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.570 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=PLCB1">PLCB1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0003723, abnormal long bone morphology</dd><dd>HP:0000486, Strabismus - MP:0000439, enlarged cranium</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000439, enlarged cranium</dd><dd>HP:0000520, Proptosis - MP:0000439, enlarged cranium</dd><dd>HP:0000601, Hypotelorism - MP:0000439, enlarged cranium</dd><dd>HP:0000244, Brachyturricephaly - MP:0000441, increased cranium width</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/613722">OMIM:613722</a> Developmental and epileptic encephalopathy 12 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3451">ORPHA:3451</a> Infantile spasms syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.285</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>20-8745817-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs776762609">rs776762609</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=689236%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000338037.6">PLCB1:ENST00000338037.6:c.2742G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378637.2">PLCB1:ENST00000378637.2:c.2742G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000378641.3">PLCB1:ENST00000378641.3:c.2742G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0053%</dd>
                                    <dd>ExAC AMR: 0.0175%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AMR: 0.0298%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ABCC1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000103222">ABCC1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.413</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618915">OMIM:618915</a> ?Deafness, autosomal dominant 77 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.413</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-16208928-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs74985930">rs74985930</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.742</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345148.5">ABCC1:ENST00000345148.5:c.3385G&gt;A:p.(Val1129Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346370.5">ABCC1:ENST00000346370.5:c.3217G&gt;A:p.(Val1073Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349029.5">ABCC1:ENST00000349029.5:c.3040G&gt;A:p.(Val1014Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351154.5">ABCC1:ENST00000351154.5:c.3208G&gt;A:p.(Val1070Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399408.2">ABCC1:ENST00000399408.2:c.3415G&gt;A:p.(Val1139Ile)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399410.3">ABCC1:ENST00000399410.3:c.3385G&gt;A:p.(Val1129Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.8093051</dd>
                                    
                                    <dd>REVEL: 0.436</dd>
                                    <dd>MVP: 0.809</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0790%</dd>
                                    <dd>UK10K: 0.0397%</dd>
                                    <dd>ESP EA: 0.1299%</dd>
                                    <dd>ESP All: 0.0868%</dd>
                                    <dd>ExAC AFR: 0.0309%</dd>
                                    <dd>ExAC AMR: 0.0695%</dd>
                                    <dd>ExAC NFE: 0.1407%</dd>
                                    <dd>ExAC SAS: 0.4767%</dd>
                                    <dd>gnomAD_E_AFR: 0.0392%</dd>
                                    <dd>gnomAD_E_AMR: 0.0685%</dd>
                                    <dd>gnomAD_E_FIN: 0.0135%</dd>
                                    <dd>gnomAD_E_NFE: 0.1080%</dd>
                                    <dd>gnomAD_E_OTH: 0.1645%</dd>
                                    <dd>gnomAD_E_SAS: 0.4817%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0799%</dd>
                                    <dd>gnomAD_G_OTH: 0.1018%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-16208714-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs191017838">rs191017838</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    <span class="label label-success">BENIGN</span>
                                
                                
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=703516%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.084</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000345148.5">ABCC1:ENST00000345148.5:c.3171G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000346370.5">ABCC1:ENST00000346370.5:c.3003G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349029.5">ABCC1:ENST00000349029.5:c.2826G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000351154.5">ABCC1:ENST00000351154.5:c.2994G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399408.2">ABCC1:ENST00000399408.2:c.3201G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000399410.3">ABCC1:ENST00000399410.3:c.3171G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.4080%</dd>
                                    <dd>UK10K: 0.7670%</dd>
                                    <dd>ESP AA: 0.1157%</dd>
                                    <dd>ESP EA: 0.7705%</dd>
                                    <dd>ESP All: 0.5509%</dd>
                                    <dd>ExAC AFR: 0.0517%</dd>
                                    <dd>ExAC AMR: 0.3333%</dd>
                                    <dd>ExAC FIN: 0.1093%</dd>
                                    <dd>ExAC NFE: 0.5068%</dd>
                                    <dd>ExAC OTH: 0.4556%</dd>
                                    <dd>ExAC SAS: 0.1172%</dd>
                                    <dd>gnomAD_E_AFR: 0.0721%</dd>
                                    <dd>gnomAD_E_AMR: 0.3369%</dd>
                                    <dd>gnomAD_E_FIN: 0.1888%</dd>
                                    <dd>gnomAD_E_NFE: 0.5739%</dd>
                                    <dd>gnomAD_E_OTH: 0.5309%</dd>
                                    <dd>gnomAD_E_SAS: 0.1140%</dd>
                                    <dd>gnomAD_G_AFR: 0.0917%</dd>
                                    <dd>gnomAD_G_FIN: 0.1718%</dd>
                                    <dd>gnomAD_G_NFE: 0.5064%</dd>
                                    <dd>gnomAD_G_OTH: 0.5092%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TEX15" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000133863">TEX15</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.107</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEX15%0DALX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALX3</a> and phenotypic similarity 0.705 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=391474">Frontorhiny</a> associated with ALX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0001156, Brachydactyly</dd><dd>HP:0000486, Strabismus - HP:0000486, Strabismus</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000327, Hypoplasia of the maxilla</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.504 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEX15%0DALX3&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to ALX3</a> and phenotypic similarity 0.432 to mouse mutant of ALX3.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0009264, failure of eyelid fusion</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0009264, failure of eyelid fusion</dd><dd>HP:0000520, Proptosis - MP:0009264, failure of eyelid fusion</dd><dd>HP:0000601, Hypotelorism - MP:0009264, failure of eyelid fusion</dd><dd>HP:0000244, Brachyturricephaly - MP:0009264, failure of eyelid fusion</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617960">OMIM:617960</a> Spermatogenic failure 25 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.107</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>8-30706055-G-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.107</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000256246.2">TEX15:ENST00000256246.2:c.479C&gt;G:p.(Ser160Cys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.10739988</dd>
                                    
                                    <dd>REVEL: 0.070</dd>
                                    <dd>MVP: 0.107</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TEX14" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121101">TEX14</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.259</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEX14%0DCEP55&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP55</a> and phenotypic similarity 0.537 to <a href="http://www.omim.org/entry/236500">Multinucleated neurons, anhydramnios, renal dysplasia, cerebellar hypoplasia, and hydranencephaly</a> associated with CEP55.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - HP:0012725, Cutaneous syndactyly</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000308, Microretrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEX14%0DCEP55&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP55</a> and phenotypic similarity 0.655 to mouse mutant of CEP55.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000520, Proptosis - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000601, Hypotelorism - MP:0000438, abnormal cranium morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0000433, microcephaly</dd></dl><dl><dt>Proximity score 0.518 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=TEX14%0DCEP55&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to CEP55</a> and phenotypic similarity 0.266 to fish mutant of CEP55.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - ZP:0001708, mandibular arch skeleton malformed, abnormal</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/617707">OMIM:617707</a> Spermatogenic failure 23 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=399805">ORPHA:399805</a> Male infertility with azoospermia or oligozoospermia due to single gene mutation - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.259</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>17-56688605-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs192936672">rs192936672</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=740908%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000240361.8">TEX14:ENST00000240361.8:c.1119C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000349033.5">TEX14:ENST00000349033.5:c.1101C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389934.3">TEX14:ENST00000389934.3:c.1101C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0189%</dd>
                                    <dd>UK10K: 0.0661%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC AMR: 0.0086%</dd>
                                    <dd>ExAC NFE: 0.0270%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0296%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0467%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CUBN" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000107611">CUBN</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.252</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.480 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=CUBN">CUBN</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0001322, abnormal iris morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0001322, abnormal iris morphology</dd><dd>HP:0000520, Proptosis - MP:0001322, abnormal iris morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001322, abnormal iris morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0001322, abnormal iris morphology</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.622 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=2143">Donnai-Barrow syndrome</a> associated with LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0005280, Depressed nasal bridge</dd><dd>HP:0000520, Proptosis - HP:0000520, Proptosis</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.681 to mouse mutant of LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000585, kinked tail</dd><dd>HP:0000486, Strabismus - MP:0001293, anophthalmia</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0002639, micrognathia</dd><dd>HP:0000520, Proptosis - MP:0001293, anophthalmia</dd><dd>HP:0000601, Hypotelorism - MP:0001293, anophthalmia</dd><dd>HP:0000244, Brachyturricephaly - MP:0001293, anophthalmia</dd></dl><dl><dt>Proximity score 0.503 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=CUBN%0DLRP2&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to LRP2</a> and phenotypic similarity 0.544 to fish mutant of LRP2.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - ZP:0006772, eye protruding, abnormal</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis - ZP:0006772, eye protruding, abnormal</dd><dd>HP:0000601, Hypotelorism - ZP:0006772, eye protruding, abnormal</dd><dd>HP:0000244, Brachyturricephaly -</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/261100">OMIM:261100</a> Imerslund-Grasbeck syndrome 1 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=35858">ORPHA:35858</a> Imerslund-Gräsbeck syndrome - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.252</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-17083149-A-G [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000377833.4">CUBN:ENST00000377833.4:c.3900T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="HOGA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000241935">HOGA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HOGA1%0DPSPH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSPH</a> and phenotypic similarity 0.410 to <a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=79350">3-phosphoserine phosphatase deficiency, infantile/juvenile form</a> associated with PSPH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000347, Micrognathia</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - HP:0000252, Microcephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=HOGA1%0DPSPH&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to PSPH</a> and phenotypic similarity 0.616 to mouse mutant of PSPH.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly - MP:0000565, oligodactyly</dd><dd>HP:0000486, Strabismus - MP:0004269, abnormal optic cup morphology</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0004539, absent maxilla</dd><dd>HP:0000520, Proptosis - MP:0001303, abnormal lens morphology</dd><dd>HP:0000601, Hypotelorism - MP:0001303, abnormal lens morphology</dd><dd>HP:0000244, Brachyturricephaly - MP:0004268, abnormal optic stalk morphology</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/613616">OMIM:613616</a> Hyperoxaluria, primary, type III - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=93600">ORPHA:93600</a> Primary hyperoxaluria type 3 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>10-99358629-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs772566254">rs772566254</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                <span class="label label-info">LIKELY_BENIGN</span>
                                
                                
                                
                                
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=737670%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370646.4">HOGA1:ENST00000370646.4:c.309C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000555577.1">HOGA1:ENST00000555577.1:c.212-2985C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370647.4">HOGA1:ENST00000370647.4:c.212-2985C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000370649.3">HOGA1:ENST00000370649.3:c.212-2985C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC NFE: 0.0045%</dd>
                                    <dd>gnomAD_E_AMR: 0.0149%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC13A3" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000158296">SLC13A3</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.250</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC13A3%0DGAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GAD1</a> and phenotypic similarity 0.492 to <a href="http://www.omim.org/entry/619124">Developmental and epileptic encephalopathy 89</a> associated with GAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla - HP:0000308, Microretrognathia</dd><dd>HP:0000520, Proptosis - HP:0000316, Hypertelorism</dd><dd>HP:0000601, Hypotelorism - HP:0000316, Hypertelorism</dd><dd>HP:0000244, Brachyturricephaly - HP:0000256, Macrocephaly</dd></dl><dl><dt>Proximity score 0.500 in <a href="http://version10.string-db.org/newstring_cgi/show_network_section.pl?identifiers=SLC13A3%0DGAD1&required_score=700&network_flavor=evidence&species=9606&limit=20">interactome to GAD1</a> and phenotypic similarity 0.675 to mouse mutant of GAD1.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus - MP:0030248, narrow maxilla</dd><dd>HP:0000327, Hypoplasia of the maxilla - MP:0030248, narrow maxilla</dd><dd>HP:0000520, Proptosis - MP:0030248, narrow maxilla</dd><dd>HP:0000601, Hypotelorism - MP:0030248, narrow maxilla</dd><dd>HP:0000244, Brachyturricephaly - MP:0030248, narrow maxilla</dd></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618384">OMIM:618384</a> Leukoencephalopathy, acute reversible, with increased urinary alpha-ketoglutarate - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.250</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>20-45221159-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs139819116">rs139819116</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000279027.4">SLC13A3:ENST00000279027.4:c.804G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000290317.5">SLC13A3:ENST00000290317.5:c.663G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396360.1">SLC13A3:ENST00000396360.1:c.663G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000472148.1">SLC13A3:ENST00000472148.1:c.663G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000495082.1">SLC13A3:ENST00000495082.1:c.663G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372121.1">SLC13A3:ENST00000372121.1:c.728-74G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413164.2">SLC13A3:ENST00000413164.2:c.728-74G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000435032.1">SLC13A3:ENST00000435032.1:c.-329G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0132%</dd>
                                    <dd>UK10K: 0.0264%</dd>
                                    <dd>ESP AA: 0.0454%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0486%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0523%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_AFR: 0.0229%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CEP170B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000099814">CEP170B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.381</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.381</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-105353232-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs373327078">rs373327078</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.381</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000414716.3">CEP170B:ENST00000414716.3:c.2656A&gt;G:p.(Ile886Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418279.1">CEP170B:ENST00000418279.1:c.2446A&gt;G:p.(Ile816Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453495.1">CEP170B:ENST00000453495.1:c.2659A&gt;G:p.(Ile887Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000556508.1">CEP170B:ENST00000556508.1:c.2446A&gt;G:p.(Ile816Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3836093</dd>
                                    
                                    <dd>REVEL: 0.165</dd>
                                    <dd>MVP: 0.384</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0302%</dd>
                                    <dd>ESP EA: 0.0243%</dd>
                                    <dd>ESP All: 0.0165%</dd>
                                    <dd>ExAC NFE: 0.0079%</dd>
                                    <dd>gnomAD_E_AMR: 0.0418%</dd>
                                    <dd>gnomAD_E_NFE: 0.0046%</dd>
                                    <dd>gnomAD_E_OTH: 0.0554%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="INKA1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185614">INKA1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.310</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.029</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>Phenotypic similarity 0.175 to mouse mutant involving <a href="http://www.informatics.jax.org/searchtool/Search.do?query=INKA1">INKA1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - MP:0000914, exencephaly</dd></dl><dl><dt>Phenotypic similarity 0.310 to zebrafish mutant involving <a href="http://zfin.org/action/quicksearch/query?query=INKA1">INKA1</a>.</dt><dt>Best Phenotype Matches:</dt><dd>HP:0001159, Syndactyly -</dd><dd>HP:0000486, Strabismus -</dd><dd>HP:0000327, Hypoplasia of the maxilla -</dd><dd>HP:0000520, Proptosis -</dd><dd>HP:0000601, Hypotelorism -</dd><dd>HP:0000244, Brachyturricephaly - ZP:0000054, head decreased size, abnormal</dd></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.310</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.029</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-49841903-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs571203506">rs571203506</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.029</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333323.4">INKA1:ENST00000333323.4:c.347G&gt;A:p.(Arg116His)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.029</dd>
                                    
                                    <dd>REVEL: 0.029</dd>
                                    <dd>MVP: 0.014</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LY75-CD302" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000248672">LY75-CD302</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=7.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.361</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=7.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.361</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-160706630-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs145262865">rs145262865</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.361</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000504764.1">LY75-CD302:ENST00000504764.1:c.3011C&gt;T:p.(Pro1004Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000505052.1">LY75-CD302:ENST00000505052.1:c.3011C&gt;T:p.(Pro1004Leu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3617588</dd>
                                    
                                    <dd>REVEL: 0.110</dd>
                                    <dd>MVP: 0.362</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0030%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_FIN: 0.0045%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_OTH: 0.0186%</dd>
                                    <dd>gnomAD_E_SAS: 0.0035%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FNDC8" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000073598">FNDC8</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.348</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.348</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-33457364-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs774238390">rs774238390</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.348</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000158009.5">FNDC8:ENST00000158009.5:c.886C&gt;T:p.(Arg296Trp)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000586869.1">FNDC8:ENST00000586869.1:c.*2092G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.34810194</dd>
                                    
                                    <dd>REVEL: 0.093</dd>
                                    <dd>MVP: 0.348</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0034%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>ExAC SAS: 0.0121%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="AHNAK2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000185567">AHNAK2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.0E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.343</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.0E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.343</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>14-105413326-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs369899617">rs369899617</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.343</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333244.5">AHNAK2:ENST00000333244.5:c.8462C&gt;T:p.(Pro2821Leu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000557457.1">AHNAK2:ENST00000557457.1:c.-220-6011C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3447107</dd>
                                    
                                    <dd>REVEL: 0.162</dd>
                                    <dd>MVP: 0.345</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ESP AA: 0.0254%</dd>
                                    <dd>ESP All: 0.0082%</dd>
                                    <dd>ExAC AFR: 0.0408%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0417%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                    <dd>gnomAD_E_OTH: 0.0365%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SLC2A6" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000160326">SLC2A6</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.1E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.335</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.1E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.335</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>9-136340625-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs147854160">rs147854160</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.584</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371897.4">SLC2A6:ENST00000371897.4:c.671G&gt;A:p.(Arg224Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371899.4">SLC2A6:ENST00000371899.4:c.671G&gt;A:p.(Arg224Gln)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.68332887</dd>
                                    
                                    <dd>REVEL: 0.101</dd>
                                    <dd>MVP: 0.683</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.3786%</dd>
                                    <dd>UK10K: 0.7273%</dd>
                                    <dd>ESP AA: 0.0910%</dd>
                                    <dd>ESP EA: 0.5117%</dd>
                                    <dd>ESP All: 0.3693%</dd>
                                    <dd>ExAC AFR: 0.0990%</dd>
                                    <dd>ExAC AMR: 0.2546%</dd>
                                    <dd>ExAC FIN: 0.0874%</dd>
                                    <dd>ExAC NFE: 0.4092%</dd>
                                    <dd>ExAC OTH: 0.2525%</dd>
                                    <dd>ExAC SAS: 0.0063%</dd>
                                    <dd>gnomAD_E_AFR: 0.0805%</dd>
                                    <dd>gnomAD_E_AMR: 0.3852%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.1042%</dd>
                                    <dd>gnomAD_E_NFE: 0.4702%</dd>
                                    <dd>gnomAD_E_OTH: 0.1846%</dd>
                                    <dd>gnomAD_E_SAS: 0.0065%</dd>
                                    <dd>gnomAD_G_AFR: 0.0687%</dd>
                                    <dd>gnomAD_G_FIN: 0.1433%</dd>
                                    <dd>gnomAD_G_NFE: 0.3335%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>9-136337164-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142001028">rs142001028</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.086</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371897.4">SLC2A6:ENST00000371897.4:c.1317G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000371899.4">SLC2A6:ENST00000371899.4:c.1503G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0799%</dd>
                                    <dd>TOPMed: 0.3672%</dd>
                                    <dd>UK10K: 0.7273%</dd>
                                    <dd>ESP AA: 0.1135%</dd>
                                    <dd>ESP EA: 0.5349%</dd>
                                    <dd>ESP All: 0.3921%</dd>
                                    <dd>ExAC AFR: 0.0874%</dd>
                                    <dd>ExAC AMR: 0.2510%</dd>
                                    <dd>ExAC FIN: 0.0908%</dd>
                                    <dd>ExAC NFE: 0.3884%</dd>
                                    <dd>ExAC OTH: 0.2247%</dd>
                                    <dd>gnomAD_E_AFR: 0.0720%</dd>
                                    <dd>gnomAD_E_AMR: 0.3873%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_FIN: 0.1077%</dd>
                                    <dd>gnomAD_E_NFE: 0.4638%</dd>
                                    <dd>gnomAD_E_OTH: 0.1828%</dd>
                                    <dd>gnomAD_E_SAS: 0.0032%</dd>
                                    <dd>gnomAD_G_AFR: 0.0689%</dd>
                                    <dd>gnomAD_G_FIN: 0.1434%</dd>
                                    <dd>gnomAD_G_NFE: 0.3346%</dd>
                                    <dd>gnomAD_G_OTH: 0.1022%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF420" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197050">ZNF420</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.318</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.318</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-37619496-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs757785456">rs757785456</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.318</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000337995.3">ZNF420:ENST00000337995.3:c.1603G&gt;C:p.(Glu535Gln)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000304239.7">ZNF420:ENST00000304239.7:c.1411+192G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.3176678</dd>
                                    
                                    <dd>REVEL: 0.072</dd>
                                    <dd>MVP: 0.318</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TEKT5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000153060">TEKT5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.2E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.311</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.2E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.311</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-10788200-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs148185751">rs148185751</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.429</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283025.2">TEKT5:ENST00000283025.2:c.531G&gt;C:p.(Glu177Asp)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.48198804</dd>
                                    
                                    <dd>REVEL: 0.192</dd>
                                    <dd>MVP: 0.482</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2596%</dd>
                                    <dd>UK10K: 0.4099%</dd>
                                    <dd>ESP AA: 0.1138%</dd>
                                    <dd>ESP EA: 0.3372%</dd>
                                    <dd>ESP All: 0.2617%</dd>
                                    <dd>ExAC AFR: 0.1454%</dd>
                                    <dd>ExAC AMR: 0.3469%</dd>
                                    <dd>ExAC FIN: 0.3175%</dd>
                                    <dd>ExAC NFE: 0.3449%</dd>
                                    <dd>ExAC OTH: 0.5924%</dd>
                                    <dd>ExAC SAS: 0.0394%</dd>
                                    <dd>gnomAD_E_AFR: 0.1570%</dd>
                                    <dd>gnomAD_E_AMR: 0.2761%</dd>
                                    <dd>gnomAD_E_FIN: 0.2277%</dd>
                                    <dd>gnomAD_E_NFE: 0.3410%</dd>
                                    <dd>gnomAD_E_OTH: 0.4801%</dd>
                                    <dd>gnomAD_E_SAS: 0.0199%</dd>
                                    <dd>gnomAD_G_AFR: 0.1489%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_FIN: 0.2862%</dd>
                                    <dd>gnomAD_G_NFE: 0.2132%</dd>
                                    <dd>gnomAD_G_OTH: 0.2041%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-10775932-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138988373">rs138988373</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.194</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000283025.2">TEKT5:ENST00000283025.2:c.781T&gt;C:p.(Cys261Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.21279746</dd>
                                    
                                    <dd>REVEL: 0.119</dd>
                                    <dd>MVP: 0.213</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.2196%</dd>
                                    <dd>TOPMed: 0.2320%</dd>
                                    <dd>UK10K: 0.4099%</dd>
                                    <dd>ESP AA: 0.0683%</dd>
                                    <dd>ESP EA: 0.3372%</dd>
                                    <dd>ESP All: 0.2463%</dd>
                                    <dd>ExAC AFR: 0.0577%</dd>
                                    <dd>ExAC AMR: 0.3196%</dd>
                                    <dd>ExAC FIN: 0.2873%</dd>
                                    <dd>ExAC NFE: 0.3432%</dd>
                                    <dd>ExAC OTH: 0.4405%</dd>
                                    <dd>ExAC SAS: 0.0303%</dd>
                                    <dd>gnomAD_E_AFR: 0.0849%</dd>
                                    <dd>gnomAD_E_AMR: 0.2651%</dd>
                                    <dd>gnomAD_E_FIN: 0.2242%</dd>
                                    <dd>gnomAD_E_NFE: 0.3447%</dd>
                                    <dd>gnomAD_E_OTH: 0.5106%</dd>
                                    <dd>gnomAD_E_SAS: 0.0227%</dd>
                                    <dd>gnomAD_G_AFR: 0.0345%</dd>
                                    <dd>gnomAD_G_AMR: 0.3580%</dd>
                                    <dd>gnomAD_G_FIN: 0.2864%</dd>
                                    <dd>gnomAD_G_NFE: 0.2205%</dd>
                                    <dd>gnomAD_G_OTH: 0.2045%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RBP5" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139194">RBP5</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.255</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.255</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>12-7280850-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1163577532">rs1163577532</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.255</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000266560.3">RBP5:ENST00000266560.3:c.238G&gt;A:p.(Gly80Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000542370.1">RBP5:ENST00000542370.1:c.238G&gt;A:p.(Gly80Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.255</dd>
                                    
                                    <dd>REVEL: 0.255</dd>
                                    <dd>MVP: 0.067</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TNRC6A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000090905">TNRC6A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.254</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/618074">OMIM:618074</a> ?Epilepsy, familial adult myoclonic, 6 (unconfirmed)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.254</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>16-24834847-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1280050363">rs1280050363</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[PM2, BP4]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.254</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000315183.7">TNRC6A:ENST00000315183.7:c.5461C&gt;A:p.(Pro1821Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000395799.3">TNRC6A:ENST00000395799.3:c.5608C&gt;A:p.(Pro1870Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000432286.2">TNRC6A:ENST00000432286.2:c.1042C&gt;A:p.(Pro348Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.25398108</dd>
                                    
                                    <dd>REVEL: 0.235</dd>
                                    <dd>MVP: 0.254</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ENGASE" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167280">ENGASE</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.228</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.228</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>17-77078115-C-A [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.228</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000539857.2">ENGASE:ENST00000539857.2:c.450C&gt;A:p.(Asn150Lys)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000579016.1">ENGASE:ENST00000579016.1:c.1008C&gt;A:p.(Asn336Lys)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.228</dd>
                                    
                                    <dd>REVEL: 0.228</dd>
                                    <dd>MVP: 0.067</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="KLF17" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000171872">KLF17</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.8E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.201</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.8E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.201</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>1-44595144-G-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs755476285">rs755476285</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.201</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000372299.3">KLF17:ENST00000372299.3:c.201G&gt;T:p.(Met67Ile)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.20120437</dd>
                                    
                                    <dd>REVEL: 0.030</dd>
                                    <dd>MVP: 0.201</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_NFE: 0.0009%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GASK1B" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000164125">GASK1B</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=8.9E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.185</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=8.9E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.185</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>4-159091777-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs146221046">rs146221046</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.185</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000296530.8">GASK1B:ENST00000296530.8:c.751G&gt;C:p.(Gly251Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000393807.5">GASK1B:ENST00000393807.5:c.751G&gt;C:p.(Gly251Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000585682.1">GASK1B:ENST00000585682.1:c.751G&gt;C:p.(Gly251Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000592057.1">GASK1B:ENST00000592057.1:c.751G&gt;C:p.(Gly251Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.18794507</dd>
                                    
                                    <dd>REVEL: 0.041</dd>
                                    <dd>MVP: 0.188</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0200%</dd>
                                    <dd>TOPMed: 0.0223%</dd>
                                    <dd>ExAC AFR: 0.0799%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC EAS: 0.0116%</dd>
                                    <dd>ExAC NFE: 0.0061%</dd>
                                    <dd>gnomAD_E_AFR: 0.0985%</dd>
                                    <dd>gnomAD_E_AMR: 0.0089%</dd>
                                    <dd>gnomAD_E_EAS: 0.0232%</dd>
                                    <dd>gnomAD_E_NFE: 0.0081%</dd>
                                    <dd>gnomAD_G_AFR: 0.0802%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="WDR49" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000174776">WDR49</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.3E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.115</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.3E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.115</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>3-167248948-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs532182042">rs532182042</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=2288713%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.115</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000308378.3">WDR49:ENST00000308378.3:c.1117A&gt;G:p.(Met373Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000453925.2">WDR49:ENST00000453925.2:c.1309A&gt;G:p.(Met437Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000476376.1">WDR49:ENST00000476376.1:c.592A&gt;G:p.(Met198Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000479765.1">WDR49:ENST00000479765.1:c.*17+35670A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.11512431</dd>
                                    
                                    <dd>REVEL: 0.024</dd>
                                    <dd>MVP: 0.115</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0008%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DYSF" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000135636">DYSF</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.103</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/253601">OMIM:253601</a> Muscular dystrophy, limb-girdle, autosomal recessive 2 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/254130">OMIM:254130</a> Miyoshi muscular dystrophy 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/606768">OMIM:606768</a> Myopathy, distal, with anterior tibial onset - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=178400">ORPHA:178400</a> Distal myopathy with anterior tibial onset - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=268">ORPHA:268</a> Dysferlin-related limb-girdle muscular dystrophy R2 - autosomal recessive</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=45448">ORPHA:45448</a> Miyoshi myopathy - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.103</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-71740897-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs34999029">rs34999029</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=100234%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.110</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258104.3">DYSF:ENST00000258104.3:c.509C&gt;A:p.(Ala170Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394120.2">DYSF:ENST00000394120.2:c.512C&gt;A:p.(Ala171Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409366.1">DYSF:ENST00000409366.1:c.512C&gt;A:p.(Ala171Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409582.3">DYSF:ENST00000409582.3:c.602C&gt;A:p.(Ala201Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409651.1">DYSF:ENST00000409651.1:c.605C&gt;A:p.(Ala202Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409744.1">DYSF:ENST00000409744.1:c.512C&gt;A:p.(Ala171Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409762.1">DYSF:ENST00000409762.1:c.602C&gt;A:p.(Ala201Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410020.3">DYSF:ENST00000410020.3:c.605C&gt;A:p.(Ala202Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410041.1">DYSF:ENST00000410041.1:c.605C&gt;A:p.(Ala202Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413539.2">DYSF:ENST00000413539.2:c.602C&gt;A:p.(Ala201Glu)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429174.2">DYSF:ENST00000429174.2:c.509C&gt;A:p.(Ala170Glu)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.217</dd>
                                    
                                    <dd>REVEL: 0.217</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4193%</dd>
                                    <dd>TOPMed: 0.9134%</dd>
                                    <dd>UK10K: 1.4943%</dd>
                                    <dd>ESP AA: 0.0681%</dd>
                                    <dd>ESP EA: 1.5349%</dd>
                                    <dd>ESP All: 1.0380%</dd>
                                    <dd>ExAC AFR: 0.1749%</dd>
                                    <dd>ExAC AMR: 0.4341%</dd>
                                    <dd>ExAC FIN: 0.9828%</dd>
                                    <dd>ExAC NFE: 1.4842%</dd>
                                    <dd>ExAC OTH: 1.0135%</dd>
                                    <dd>ExAC SAS: 0.3521%</dd>
                                    <dd>gnomAD_E_AFR: 0.1703%</dd>
                                    <dd>gnomAD_E_AMR: 0.6611%</dd>
                                    <dd>gnomAD_E_FIN: 0.8076%</dd>
                                    <dd>gnomAD_E_NFE: 1.5102%</dd>
                                    <dd>gnomAD_E_OTH: 1.1857%</dd>
                                    <dd>gnomAD_E_SAS: 0.3736%</dd>
                                    <dd>gnomAD_G_AFR: 0.2983%</dd>
                                    <dd>gnomAD_G_AMR: 0.9547%</dd>
                                    <dd>gnomAD_G_FIN: 0.9742%</dd>
                                    <dd>gnomAD_G_NFE: 1.2843%</dd>
                                    <dd>gnomAD_G_OTH: 0.7143%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-71797754-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs143475751">rs143475751</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">CONFLICTING_PATHOGENICITY_INTERPRETATIONS</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=265920%5Balleleid%5D">(criteria provided, conflicting interpretations)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.096</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000258104.3">DYSF:ENST00000258104.3:c.3057C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394120.2">DYSF:ENST00000394120.2:c.3060C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409366.1">DYSF:ENST00000409366.1:c.3060C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409582.3">DYSF:ENST00000409582.3:c.3108C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409651.1">DYSF:ENST00000409651.1:c.3153C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409744.1">DYSF:ENST00000409744.1:c.3018C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409762.1">DYSF:ENST00000409762.1:c.3108C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410020.3">DYSF:ENST00000410020.3:c.3111C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000410041.1">DYSF:ENST00000410041.1:c.3111C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000413539.2">DYSF:ENST00000413539.2:c.3150C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000429174.2">DYSF:ENST00000429174.2:c.3057C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.0661%</dd>
                                    <dd>ESP EA: 0.1047%</dd>
                                    <dd>ESP All: 0.0692%</dd>
                                    <dd>ExAC AFR: 0.0102%</dd>
                                    <dd>ExAC AMR: 0.0354%</dd>
                                    <dd>ExAC NFE: 0.1509%</dd>
                                    <dd>ExAC OTH: 0.1163%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.0179%</dd>
                                    <dd>gnomAD_E_NFE: 0.0189%</dd>
                                    <dd>gnomAD_E_OTH: 0.1645%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                    <dd>gnomAD_G_OTH: 0.1020%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="FILIP1L" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168386">FILIP1L</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-99569638-T-C [0/1]</b>
                            
                            
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000331335.5">FILIP1L:ENST00000331335.5:c.882A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000354552.3">FILIP1L:ENST00000354552.3:c.882A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000383694.2">FILIP1L:ENST00000383694.2:c.162A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000471562.1">FILIP1L:ENST00000471562.1:c.162A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000421999.2">FILIP1L:ENST00000421999.2:c.64+32751T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000487087.1">FILIP1L:ENST00000487087.1:c.-25+139A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="PYHIN1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000163564">PYHIN1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>1-158906715-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1368228534">rs1368228534</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368135.4">PYHIN1:ENST00000368135.4:c.15C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368138.3">PYHIN1:ENST00000368138.3:c.15C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000368140.1">PYHIN1:ENST00000368140.1:c.15C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392252.3">PYHIN1:ENST00000392252.3:c.15C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000392254.2">PYHIN1:ENST00000392254.2:c.15C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF222" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000159885">ZNF222</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-44536725-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs746541695">rs746541695</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000187879.8">ZNF222:ENST00000187879.8:c.898T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391960.3">ZNF222:ENST00000391960.3:c.1018T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000591793.1">ZNF222:ENST00000591793.1:c.262+5083T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    <dd>No frequency data</dd>
                                    
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="SRRM2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000167978">SRRM2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>16-2813961-A-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs756618722">rs756618722</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301740.8">SRRM2:ENST00000301740.8:c.3432A&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0011%</dd>
                                    <dd>ExAC NFE: 0.0015%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="RIOX2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000170854">RIOX2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-97680493-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs112696353">rs112696353</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000330299.2">RIOX2:ENST00000330299.2:c.492G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000333396.7">RIOX2:ENST00000333396.7:c.492G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000360258.4">RIOX2:ENST00000360258.4:c.492G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000394198.2">RIOX2:ENST00000394198.2:c.492G&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0060%</dd>
                                    <dd>ExAC AFR: 0.0096%</dd>
                                    <dd>ExAC NFE: 0.0090%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0072%</dd>
                                    <dd>gnomAD_G_NFE: 0.0067%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TTC21A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168026">TTC21A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases - observed variants incompatible with mode of inheritance:</dt>
<dd><a href="http://www.omim.org/entry/618429">OMIM:618429</a> Spermatogenic failure 37 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-39175810-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs916387868">rs916387868</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000301819.6">TTC21A:ENST00000301819.6:c.2877C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431162.2">TTC21A:ENST00000431162.2:c.2874C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000440121.1">TTC21A:ENST00000440121.1:c.2730C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>gnomAD_E_AFR: 0.0065%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="DCP1A" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000272886">DCP1A</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.100</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.100</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>3-53326567-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200307108">rs200307108</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.100</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000294241.6">DCP1A:ENST00000294241.6:c.915G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000606822.1">DCP1A:ENST00000606822.1:c.801G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000607628.1">DCP1A:ENST00000607628.1:c.915G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0026%</dd>
                                    <dd>ExAC NFE: 0.0030%</dd>
                                    <dd>gnomAD_E_AMR: 0.0030%</dd>
                                    <dd>gnomAD_E_NFE: 0.0027%</dd>
                                    <dd>gnomAD_G_NFE: 0.0133%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ZNF100" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000197020">ZNF100</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>19-21910313-C-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs751670362">rs751670362</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000305570.6">ZNF100:ENST00000305570.6:c.609G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000358296.6">ZNF100:ENST00000358296.6:c.801G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>UK10K: 0.0529%</dd>
                                    <dd>ExAC NFE: 0.0091%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_NFE: 0.0018%</dd>
                                    <dd>gnomAD_E_SAS: 0.0034%</dd>
                                    <dd>gnomAD_G_NFE: 0.0070%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CEP126" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000110318">CEP126</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=65684">ORPHA:65684</a> Monomelic amyotrophy (susceptibility)</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>11-101833008-A-G [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs373938848">rs373938848</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263468.8">CEP126:ENST00000263468.8:c.1242A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000537689.1">CEP126:ENST00000537689.1:c.645A&gt;G:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0060%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0116%</dd>
                                    <dd>ESP All: 0.0154%</dd>
                                    <dd>ExAC AFR: 0.0101%</dd>
                                    <dd>ExAC AMR: 0.0087%</dd>
                                    <dd>ExAC NFE: 0.0136%</dd>
                                    <dd>gnomAD_E_AFR: 0.0066%</dd>
                                    <dd>gnomAD_E_AMR: 0.0060%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.0126%</dd>
                                    <dd>gnomAD_E_OTH: 0.0183%</dd>
                                    <dd>gnomAD_G_NFE: 0.0667%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="CRISPLD1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000121005">CRISPLD1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.4E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.099</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.4E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.099</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>8-75929142-T-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs200626463">rs200626463</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.099</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000262207.4">CRISPLD1:ENST00000262207.4:c.895T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000517786.1">CRISPLD1:ENST00000517786.1:c.337T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000523524.1">CRISPLD1:ENST00000523524.1:c.331T&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0170%</dd>
                                    <dd>UK10K: 0.0132%</dd>
                                    <dd>ESP EA: 0.0118%</dd>
                                    <dd>ESP All: 0.0078%</dd>
                                    <dd>ExAC AMR: 0.0695%</dd>
                                    <dd>ExAC NFE: 0.0227%</dd>
                                    <dd>gnomAD_E_AMR: 0.0475%</dd>
                                    <dd>gnomAD_E_NFE: 0.0098%</dd>
                                    <dd>gnomAD_E_OTH: 0.0818%</dd>
                                    <dd>gnomAD_G_AFR: 0.0115%</dd>
                                    <dd>gnomAD_G_NFE: 0.0267%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="ITPRID2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000138434">ITPRID2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.095</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.095</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-182766624-C-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs1692696474">rs1692696474</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000320370.7">ITPRID2:ENST00000320370.7:c.844C&gt;A:p.(Pro282Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000409001.1">ITPRID2:ENST00000409001.1:c.844C&gt;A:p.(Pro282Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000428267.2">ITPRID2:ENST00000428267.2:c.385C&gt;A:p.(Pro129Thr)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000431877.2">ITPRID2:ENST00000431877.2:c.844C&gt;A:p.(Pro282Thr)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.09545038</dd>
                                    
                                    <dd>REVEL: 0.039</dd>
                                    <dd>MVP: 0.095</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>TOPMed: 0.0004%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="TNXB" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000168477">TNXB</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.5E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.082</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dl>
<dt>Known diseases:</dt>
<dd><a href="http://www.omim.org/entry/606408">OMIM:606408</a> Ehlers-Danlos syndrome, classic-like, 1 - autosomal recessive</dd>
<dd><a href="http://www.omim.org/entry/615963">OMIM:615963</a> Vesicoureteral reflux 8 - autosomal dominant</dd>
<dd><a href="http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=230839">ORPHA:230839</a> Classical-like Ehlers-Danlos syndrome type 1 - autosomal recessive</dd>
</dl>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_DOMINANT</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.5E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.082</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>6-32036743-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs776795346">rs776795346</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[BP4_Moderate]</b>
                            
                        </div>
                        <div class="col-sm-12">
                            <b>ClinVar: </b>
                            
                                    
                                
                                
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                <a href="https://www.ncbi.nlm.nih.gov/clinvar/?term=1800484%5Balleleid%5D">(criteria provided, single submitter)</a>
                                
                            </div>
                            <div class="col-sm-6">
                                <b>Variant score: 0.082</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375244.3">TNXB:ENST00000375244.3:c.5758G&gt;A:p.(Gly1920Arg)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000375247.2">TNXB:ENST00000375247.2:c.5758G&gt;A:p.(Gly1920Arg)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.08231511</dd>
                                    
                                    <dd>REVEL: 0.064</dd>
                                    <dd>MVP: 0.082</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>ExAC SAS: 0.0061%</dd>
                                    <dd>gnomAD_E_EAS: 0.0058%</dd>
                                    <dd>gnomAD_E_SAS: 0.0033%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="LILRA2" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000239998">LILRA2</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.6E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.076</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.6E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.076</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-55087468-G-A [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs151154459">rs151154459</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.089</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251376.3">LILRA2:ENST00000251376.3:c.1147G&gt;A:p.(Gly383Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251377.3">LILRA2:ENST00000251377.3:c.1147G&gt;A:p.(Gly383Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391737.1">LILRA2:ENST00000391737.1:c.1111G&gt;A:p.(Gly371Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391738.3">LILRA2:ENST00000391738.3:c.1147G&gt;A:p.(Gly383Ser)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396321.2">LILRA2:ENST00000396321.2:c.70+1890G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418536.2">LILRA2:ENST00000418536.2:c.70+1890G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448689.1">LILRA2:ENST00000448689.1:c.70+1890G&gt;A:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.09208627</dd>
                                    
                                    <dd>REVEL: 0.004</dd>
                                    <dd>MVP: 0.092</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.0399%</dd>
                                    <dd>TOPMed: 0.1054%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0581%</dd>
                                    <dd>ESP All: 0.0461%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.1274%</dd>
                                    <dd>ExAC OTH: 0.2203%</dd>
                                    <dd>ExAC SAS: 0.0424%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0596%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.1200%</dd>
                                    <dd>gnomAD_E_OTH: 0.2005%</dd>
                                    <dd>gnomAD_E_SAS: 0.0390%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.0737%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>19-55087463-G-C [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs140640200">rs140640200</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.062</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251376.3">LILRA2:ENST00000251376.3:c.1142G&gt;C:p.(Arg381Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000251377.3">LILRA2:ENST00000251377.3:c.1142G&gt;C:p.(Arg381Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391737.1">LILRA2:ENST00000391737.1:c.1106G&gt;C:p.(Arg369Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000391738.3">LILRA2:ENST00000391738.3:c.1142G&gt;C:p.(Arg381Pro)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396321.2">LILRA2:ENST00000396321.2:c.70+1885G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000418536.2">LILRA2:ENST00000418536.2:c.70+1885G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000448689.1">LILRA2:ENST00000448689.1:c.70+1885G&gt;C:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.07629995</dd>
                                    
                                    <dd>REVEL: 0.050</dd>
                                    <dd>MVP: 0.076</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.5192%</dd>
                                    <dd>ESP AA: 0.0227%</dd>
                                    <dd>ESP EA: 0.0581%</dd>
                                    <dd>ESP All: 0.0461%</dd>
                                    <dd>ExAC AFR: 0.0192%</dd>
                                    <dd>ExAC AMR: 0.0259%</dd>
                                    <dd>ExAC FIN: 0.0151%</dd>
                                    <dd>ExAC NFE: 0.1274%</dd>
                                    <dd>ExAC OTH: 0.3304%</dd>
                                    <dd>ExAC SAS: 0.1454%</dd>
                                    <dd>gnomAD_E_AFR: 0.0131%</dd>
                                    <dd>gnomAD_E_AMR: 0.0596%</dd>
                                    <dd>gnomAD_E_FIN: 0.0090%</dd>
                                    <dd>gnomAD_E_NFE: 0.1191%</dd>
                                    <dd>gnomAD_E_OTH: 0.2188%</dd>
                                    <dd>gnomAD_E_SAS: 0.0488%</dd>
                                    <dd>gnomAD_G_AFR: 0.0575%</dd>
                                    <dd>gnomAD_G_EAS: 0.8663%</dd>
                                    <dd>gnomAD_G_FIN: 0.0286%</dd>
                                    <dd>gnomAD_G_NFE: 0.1406%</dd>
                                    <dd>gnomAD_G_OTH: 0.3080%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>
    <div class="panel panel-default">
        <div id="GREB1" class="panel-heading">
            <div class="row">
                <div class="col-sm-3">
                    <h4>
                        <b></b>
                        <a href="https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000196208">GREB1</a>
                        
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Exomiser Score: <b>0.000</b>
                    <data>(p=9.7E-1)</data>
                    </h4>
                </div>
                <div class="col-sm-3">
                    <h4>Phenotype Score: <b>0.000</b></h4>
                </div>
                <div class="col-sm-3">
                    <h4>Variant Score: <b>0.058</b></h4>
                </div>
            </div>
        </div>
        <div class="panel-body">
            <div class="row">
                <div class="col-sm-12">
                    <b>Phenotype matches:</b>
                </div>
            </div>
            <div class="row">
                <div class="col-sm-12">
                    <dl><dt>No phenotype or PPI evidence</dt></dl><dt>No known disease</dt>
                </div>
            </div>

            <div class="row">
                <div class="col-sm-12">
                    <b>Gene scores under compatible inheritance modes:</b>
                </div>
            </div>

            <div class="panel panel-default">

                <!--<div th:if="${not #lists.isEmpty(geneScore.getContributingVariants())}">-->
                <div class="panel-heading">
                    <div class="row">
                        <div class="col-sm-3">
                            <h4>AUTOSOMAL_RECESSIVE</h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Exomiser Score:
                                    <b>0.000</b>
                                    <data>(p=9.7E-1)</data>
                                </h4>
                            </div>
                            <div class="col-sm-3">
                                <h4>Phenotype Score: <b>0.000</b>
                                </h4>
                            </div>
                        <div class="col-sm-3">
                            <h4>Variant Score: <b>0.058</b>
                            </h4>
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>
                                No phenotype matches to diseases with this MOI.
                            </b>
                            
                        </div>
                    </div>
                </div>
                <div class="panel-body">
                    <div class="row">
                        <div class="col-sm-12">
                            <b>Variants contributing to score:</b>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">SYNONYMOUS_VARIANT</span>
                            <b>SNV</b>
                            <b>2-11778907-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs138273238">rs138273238</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.095</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000234142.5">GREB1:ENST00000234142.5:c.5658C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381486.2">GREB1:ENST00000381486.2:c.5658C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000396123.1">GREB1:ENST00000396123.1:c.2652C&gt;T:p.(=)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    
                                    <dd>No
                                        pathogenicity data
                                    </dd>
                                    
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.1398%</dd>
                                    <dd>TOPMed: 0.1281%</dd>
                                    <dd>UK10K: 0.3042%</dd>
                                    <dd>ESP AA: 0.0526%</dd>
                                    <dd>ESP EA: 0.2549%</dd>
                                    <dd>ESP All: 0.1910%</dd>
                                    <dd>ExAC AFR: 0.0816%</dd>
                                    <dd>ExAC AMR: 0.1296%</dd>
                                    <dd>ExAC FIN: 0.1058%</dd>
                                    <dd>ExAC NFE: 0.1424%</dd>
                                    <dd>ExAC OTH: 0.2222%</dd>
                                    <dd>ExAC SAS: 0.0303%</dd>
                                    <dd>gnomAD_E_AFR: 0.0458%</dd>
                                    <dd>gnomAD_E_AMR: 0.1103%</dd>
                                    <dd>gnomAD_E_FIN: 0.0897%</dd>
                                    <dd>gnomAD_E_NFE: 0.1818%</dd>
                                    <dd>gnomAD_E_OTH: 0.1826%</dd>
                                    <dd>gnomAD_E_SAS: 0.0423%</dd>
                                    <dd>gnomAD_G_AFR: 0.0344%</dd>
                                    <dd>gnomAD_G_AMR: 0.2387%</dd>
                                    <dd>gnomAD_G_FIN: 0.0287%</dd>
                                    <dd>gnomAD_G_NFE: 0.0934%</dd>
                                </dl>
                            </div>
                        </div>
                    <div class="row">
                        <div class="col-sm-12">
                                <span class="label label-danger">MISSENSE_VARIANT</span>
                            <b>SNV</b>
                            <b>2-11706672-C-T [0/1]</b>
                            
                            
                                <a href="https://www.ncbi.nlm.nih.gov/snp/rs142882892">rs142882892</a>
                                    
                                    
                                    
                                    
                                
                        </div>
                        <div class="col-sm-12">
                            <b>Exomiser ACMG: </b>
                            
                                    
                                
                                
                                
                                <span class="label label-default">UNCERTAIN_SIGNIFICANCE</span>
                                
                                <b>[]</b>
                            
                        </div>
                        
                            <div class="col-sm-6">
                                <b>Variant score: 0.021</b>
                                <span class="label label-info">CONTRIBUTING VARIANT</span>
                                
                                <dl class="list-unstyled">
                                    <dt>Transcripts:</dt>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000234142.5">GREB1:ENST00000234142.5:c.344C&gt;T:p.(Ala115Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000263834.5">GREB1:ENST00000263834.5:c.344C&gt;T:p.(Ala115Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381483.2">GREB1:ENST00000381483.2:c.344C&gt;T:p.(Ala115Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000381486.2">GREB1:ENST00000381486.2:c.344C&gt;T:p.(Ala115Val)</a>
                                        
                                    </dd>
                                    <dd>
                                        
                                        <a href="https://grch37.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000389825.3">GREB1:ENST00000389825.3:c.14C&gt;T:p.(Ala5Val)</a>
                                        
                                    </dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Pathogenicity Score: 1.00</b>-->
                                <dl class="list-unstyled">
                                    <dt>Pathogenicity Data:</dt>
                                    <dd>Best Score: 0.111</dd>
                                    
                                    <dd>REVEL: 0.111</dd>
                                </dl>
                            </div>
                            <div class="col-sm-3">
                                <!--<b>Frequency Score: 0.95</b>-->
                                <dl>
                                    <dt>Frequency Data:</dt>
                                    
                                    <dd>1000Genomes: 0.4792%</dd>
                                    <dd>TOPMed: 1.2560%</dd>
                                    <dd>UK10K: 1.9439%</dd>
                                    <dd>ESP AA: 0.4539%</dd>
                                    <dd>ESP EA: 1.8256%</dd>
                                    <dd>ESP All: 1.3609%</dd>
                                    <dd>ExAC AFR: 0.2117%</dd>
                                    <dd>ExAC AMR: 0.9504%</dd>
                                    <dd>ExAC FIN: 1.1188%</dd>
                                    <dd>ExAC NFE: 1.6884%</dd>
                                    <dd>ExAC OTH: 1.4349%</dd>
                                    <dd>ExAC SAS: 0.6483%</dd>
                                    <dd>gnomAD_E_AFR: 0.2418%</dd>
                                    <dd>gnomAD_E_AMR: 1.0901%</dd>
                                    <dd>gnomAD_E_EAS: 0.0116%</dd>
                                    <dd>gnomAD_E_FIN: 0.8164%</dd>
                                    <dd>gnomAD_E_NFE: 1.7117%</dd>
                                    <dd>gnomAD_E_OTH: 1.2400%</dd>
                                    <dd>gnomAD_E_SAS: 0.6596%</dd>
                                    <dd>gnomAD_G_AFR: 0.3097%</dd>
                                    <dd>gnomAD_G_AMR: 0.9592%</dd>
                                    <dd>gnomAD_G_FIN: 0.6869%</dd>
                                    <dd>gnomAD_G_NFE: 1.8153%</dd>
                                    <dd>gnomAD_G_OTH: 0.8163%</dd>
                                </dl>
                            </div>
                        </div> <!-- end variant row -->
                    </div> <!-- end panel-body -->
                <!--</div> &lt;!&ndash; end gene score section&ndash;&gt;-->
            </div>

            

            

            

             <!-- end gene scores -->
            
        </div>
    </div>

    

    <div id="about">
        <h3>About</h3>
        <p>
            The Exomizer is a Java program that functionally annotates variants from whole-exome
            sequencing data starting from a VCF file (version 4). The functional annotation code is
            based on <a href="https://github.com/charite/jannovar/">Jannovar</a> and uses
            <a href="https://genome.ucsc.edu/">UCSC</a> KnownGene transcript definitions and
            hg19 genomic coordinates
        </p>
        <p>
            Variants are prioritized according to user-defined criteria on variant frequency, pathogenicity,
            quality, inheritance pattern, and model organism phenotype data. Predicted pathogenicity data
            was extracted from the <a href="https://www.ncbi.nlm.nih.gov/pubmed/21520341">dbNSFP</a> resource.
        </p>
        <p>
            Developed by the Computational Biology and Bioinformatics group at the
            <a href="https://genetik.charite.de/">Institute for Medical Genetics and Human Genetics</a> of the
            <a href="https://www.charite.de">Charit&eacute; - Universit&auml;tsmedizin Berlin</a>, the Mouse
            Informatics Group at the <a href="https://www.sanger.ac.uk/">Sanger Institute</a> and the Smedley group at
            <a href="https://qmul.ac.uk/whri">Queen Mary University of London</a>.
        </p>


        <p>Problems, suggestions, or comments? Please <a href="mailto:peter.robinson@charite.de">let us know</a></p>
    </div>
</div>
</body>
</html>
